{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/a96123155/parade/blob/master/Umol.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# **Umol** - **U**niversal **mol**ecular framework\n",
        "**Structure prediction of protein-ligand complexes from sequence information**\n",
        "\n",
        "The protein is represented with a multiple sequence alignment and the ligand as a SMILES string, allowing for unconstrained flexibility in the protein-ligand interface. At a high accuracy threshold, unseen protein-ligand complexes can be predicted more accurately than for RoseTTAFold-AA, and at medium accuracy even classical docking methods that use known protein structures as input are surpassed.\n",
        "\n",
        "For local installation, see: https://github.com/patrickbryant1/Umol\n",
        "\\\n",
        "[Read the paper here](https://www.biorxiv.org/content/10.1101/2023.11.03.565471v1)\n",
        "\n",
        "Umol has **no size limit**. This only depends on available RAM.\n",
        "\n",
        "Umol is available under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0). \\\n",
        "The Umol parameters are made available under the terms of the [CC BY 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode).\n"
      ],
      "metadata": {
        "id": "b_8OwYliheyW"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Install dependencies\n",
        "#@markdown Make sure your runtime is GPU.\n",
        "#@markdown In the menu above do: Runtime --> Change runtime type --> Hardware accelerator (set to GPU)\n",
        "\n",
        "#@markdown **Press play.**\n",
        "\n",
        "#@markdown Simply press play on each cell below and follow the instructions.\n",
        "\n",
        "#@markdown You will have to restart the runtime after this finishes to include the new packages.\n",
        "#@markdown In the menu above do: Runtime --> Restart runtime\n",
        "\n",
        "#@markdown Don't worry about all the errors that pip give below, these are resolved in the end.\n",
        "!pip install --upgrade \"jax[cuda12_pip]\" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html\n",
        "!pip install ml-collections==0.1.1\n",
        "!pip install dm-haiku==0.0.11\n",
        "!pip install numpy\n",
        "!pip install pandas\n",
        "!pip install biopython==1.81\n",
        "!pip install chex==0.1.5\n",
        "!pip install dm-tree==0.1.8\n",
        "!pip install immutabledict==2.0.0\n",
        "!pip install --upgrade numpy\n",
        "!pip install scipy==1.7.3\n",
        "!pip install tensorflow==2.11.0\n",
        "!pip install rdkit-pypi\n",
        "!pip install py3Dmol"
      ],
      "metadata": {
        "id": "iF2PqaS8cDLH",
        "outputId": "300c1f44-746d-4729-afd1-52bcad97500c",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": 1,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in links: https://storage.googleapis.com/jax-releases/jax_cuda_releases.html\n",
            "Requirement already satisfied: jax[cuda12_pip] in /usr/local/lib/python3.11/dist-packages (0.5.2)\n",
            "Collecting jax[cuda12_pip]\n",
            "  Downloading jax-0.5.3-py3-none-any.whl.metadata (22 kB)\n",
            "Collecting jaxlib<=0.5.3,>=0.5.3 (from jax[cuda12_pip])\n",
            "  Downloading jaxlib-0.5.3-cp311-cp311-manylinux2014_x86_64.whl.metadata (1.2 kB)\n",
            "Requirement already satisfied: ml_dtypes>=0.4.0 in /usr/local/lib/python3.11/dist-packages (from jax[cuda12_pip]) (0.4.1)\n",
            "Requirement already satisfied: numpy>=1.25 in /usr/local/lib/python3.11/dist-packages (from jax[cuda12_pip]) (2.0.2)\n",
            "Requirement already satisfied: opt_einsum in /usr/local/lib/python3.11/dist-packages (from jax[cuda12_pip]) (3.4.0)\n",
            "Requirement already satisfied: scipy>=1.11.1 in /usr/local/lib/python3.11/dist-packages (from jax[cuda12_pip]) (1.14.1)\n",
            "Collecting jax-cuda12-plugin<=0.5.3,>=0.5.3 (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip])\n",
            "  Downloading jax_cuda12_plugin-0.5.3-cp311-cp311-manylinux2014_x86_64.whl.metadata (1.4 kB)\n",
            "Collecting jax-cuda12-pjrt==0.5.3 (from jax-cuda12-plugin<=0.5.3,>=0.5.3->jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip])\n",
            "  Downloading jax_cuda12_pjrt-0.5.3-py3-none-manylinux2014_x86_64.whl.metadata (492 bytes)\n",
            "Requirement already satisfied: nvidia-cublas-cu12>=12.1.3.1 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (12.5.3.2)\n",
            "Requirement already satisfied: nvidia-cuda-cupti-cu12>=12.1.105 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (12.5.82)\n",
            "Collecting nvidia-cuda-nvcc-cu12>=12.6.85 (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip])\n",
            "  Downloading nvidia_cuda_nvcc_cu12-12.8.93-py3-none-manylinux2010_x86_64.manylinux_2_12_x86_64.whl.metadata (1.7 kB)\n",
            "Requirement already satisfied: nvidia-cuda-runtime-cu12>=12.1.105 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (12.5.82)\n",
            "Requirement already satisfied: nvidia-cudnn-cu12<10.0,>=9.1 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (9.3.0.75)\n",
            "Requirement already satisfied: nvidia-cufft-cu12>=11.0.2.54 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (11.2.3.61)\n",
            "Requirement already satisfied: nvidia-cusolver-cu12>=11.4.5.107 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (11.6.3.83)\n",
            "Requirement already satisfied: nvidia-cusparse-cu12>=12.1.0.106 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (12.5.1.3)\n",
            "Requirement already satisfied: nvidia-nccl-cu12>=2.18.1 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (2.21.5)\n",
            "Requirement already satisfied: nvidia-nvjitlink-cu12>=12.1.105 in /usr/local/lib/python3.11/dist-packages (from jax-cuda12-plugin[with_cuda]<=0.5.3,>=0.5.3; extra == \"cuda12-pip\"->jax[cuda12_pip]) (12.5.82)\n",
            "Downloading jaxlib-0.5.3-cp311-cp311-manylinux2014_x86_64.whl (105.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m105.1/105.1 MB\u001b[0m \u001b[31m8.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jax_cuda12_plugin-0.5.3-cp311-cp311-manylinux2014_x86_64.whl (16.7 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m16.7/16.7 MB\u001b[0m \u001b[31m8.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jax_cuda12_pjrt-0.5.3-py3-none-manylinux2014_x86_64.whl (104.7 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m104.7/104.7 MB\u001b[0m \u001b[31m7.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jax-0.5.3-py3-none-any.whl (2.4 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.4/2.4 MB\u001b[0m \u001b[31m67.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading nvidia_cuda_nvcc_cu12-12.8.93-py3-none-manylinux2010_x86_64.manylinux_2_12_x86_64.whl (40.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m40.1/40.1 MB\u001b[0m \u001b[31m16.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: jax-cuda12-pjrt, nvidia-cuda-nvcc-cu12, jax-cuda12-plugin, jaxlib, jax\n",
            "  Attempting uninstall: jax-cuda12-pjrt\n",
            "    Found existing installation: jax-cuda12-pjrt 0.5.1\n",
            "    Uninstalling jax-cuda12-pjrt-0.5.1:\n",
            "      Successfully uninstalled jax-cuda12-pjrt-0.5.1\n",
            "  Attempting uninstall: nvidia-cuda-nvcc-cu12\n",
            "    Found existing installation: nvidia-cuda-nvcc-cu12 12.5.82\n",
            "    Uninstalling nvidia-cuda-nvcc-cu12-12.5.82:\n",
            "      Successfully uninstalled nvidia-cuda-nvcc-cu12-12.5.82\n",
            "  Attempting uninstall: jax-cuda12-plugin\n",
            "    Found existing installation: jax-cuda12-plugin 0.5.1\n",
            "    Uninstalling jax-cuda12-plugin-0.5.1:\n",
            "      Successfully uninstalled jax-cuda12-plugin-0.5.1\n",
            "  Attempting uninstall: jaxlib\n",
            "    Found existing installation: jaxlib 0.5.1\n",
            "    Uninstalling jaxlib-0.5.1:\n",
            "      Successfully uninstalled jaxlib-0.5.1\n",
            "  Attempting uninstall: jax\n",
            "    Found existing installation: jax 0.5.2\n",
            "    Uninstalling jax-0.5.2:\n",
            "      Successfully uninstalled jax-0.5.2\n",
            "Successfully installed jax-0.5.3 jax-cuda12-pjrt-0.5.3 jax-cuda12-plugin-0.5.3 jaxlib-0.5.3 nvidia-cuda-nvcc-cu12-12.8.93\n",
            "Collecting ml-collections==0.1.1\n",
            "  Downloading ml_collections-0.1.1.tar.gz (77 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m77.9/77.9 kB\u001b[0m \u001b[31m8.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: absl-py in /usr/local/lib/python3.11/dist-packages (from ml-collections==0.1.1) (1.4.0)\n",
            "Requirement already satisfied: PyYAML in /usr/local/lib/python3.11/dist-packages (from ml-collections==0.1.1) (6.0.2)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.11/dist-packages (from ml-collections==0.1.1) (1.17.0)\n",
            "Collecting contextlib2 (from ml-collections==0.1.1)\n",
            "  Downloading contextlib2-21.6.0-py2.py3-none-any.whl.metadata (4.1 kB)\n",
            "Downloading contextlib2-21.6.0-py2.py3-none-any.whl (13 kB)\n",
            "Building wheels for collected packages: ml-collections\n",
            "  Building wheel for ml-collections (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for ml-collections: filename=ml_collections-0.1.1-py3-none-any.whl size=94507 sha256=6adf79b9f12ea665d114146b75655c6f64149a5d8f869834219c279a85a25815\n",
            "  Stored in directory: /root/.cache/pip/wheels/28/82/ef/a6971b09a96519d55ce6efef66f0cbcdef2ae9cc1e6b41daf7\n",
            "Successfully built ml-collections\n",
            "Installing collected packages: contextlib2, ml-collections\n",
            "Successfully installed contextlib2-21.6.0 ml-collections-0.1.1\n",
            "Collecting dm-haiku==0.0.11\n",
            "  Downloading dm_haiku-0.0.11-py3-none-any.whl.metadata (19 kB)\n",
            "Requirement already satisfied: absl-py>=0.7.1 in /usr/local/lib/python3.11/dist-packages (from dm-haiku==0.0.11) (1.4.0)\n",
            "Collecting jmp>=0.0.2 (from dm-haiku==0.0.11)\n",
            "  Downloading jmp-0.0.4-py3-none-any.whl.metadata (8.9 kB)\n",
            "Requirement already satisfied: numpy>=1.18.0 in /usr/local/lib/python3.11/dist-packages (from dm-haiku==0.0.11) (2.0.2)\n",
            "Requirement already satisfied: tabulate>=0.8.9 in /usr/local/lib/python3.11/dist-packages (from dm-haiku==0.0.11) (0.9.0)\n",
            "Requirement already satisfied: flax>=0.7.1 in /usr/local/lib/python3.11/dist-packages (from dm-haiku==0.0.11) (0.10.5)\n",
            "Requirement already satisfied: jax>=0.5.1 in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (0.5.3)\n",
            "Requirement already satisfied: msgpack in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (1.1.0)\n",
            "Requirement already satisfied: optax in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (0.2.4)\n",
            "Requirement already satisfied: orbax-checkpoint in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (0.11.10)\n",
            "Requirement already satisfied: tensorstore in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (0.1.73)\n",
            "Requirement already satisfied: rich>=11.1 in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (13.9.4)\n",
            "Requirement already satisfied: typing_extensions>=4.2 in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (4.13.1)\n",
            "Requirement already satisfied: PyYAML>=5.4.1 in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (6.0.2)\n",
            "Requirement already satisfied: treescope>=0.1.7 in /usr/local/lib/python3.11/dist-packages (from flax>=0.7.1->dm-haiku==0.0.11) (0.1.9)\n",
            "Requirement already satisfied: jaxlib<=0.5.3,>=0.5.3 in /usr/local/lib/python3.11/dist-packages (from jax>=0.5.1->flax>=0.7.1->dm-haiku==0.0.11) (0.5.3)\n",
            "Requirement already satisfied: ml_dtypes>=0.4.0 in /usr/local/lib/python3.11/dist-packages (from jax>=0.5.1->flax>=0.7.1->dm-haiku==0.0.11) (0.4.1)\n",
            "Requirement already satisfied: opt_einsum in /usr/local/lib/python3.11/dist-packages (from jax>=0.5.1->flax>=0.7.1->dm-haiku==0.0.11) (3.4.0)\n",
            "Requirement already satisfied: scipy>=1.11.1 in /usr/local/lib/python3.11/dist-packages (from jax>=0.5.1->flax>=0.7.1->dm-haiku==0.0.11) (1.14.1)\n",
            "Requirement already satisfied: markdown-it-py>=2.2.0 in /usr/local/lib/python3.11/dist-packages (from rich>=11.1->flax>=0.7.1->dm-haiku==0.0.11) (3.0.0)\n",
            "Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /usr/local/lib/python3.11/dist-packages (from rich>=11.1->flax>=0.7.1->dm-haiku==0.0.11) (2.18.0)\n",
            "Requirement already satisfied: chex>=0.1.87 in /usr/local/lib/python3.11/dist-packages (from optax->flax>=0.7.1->dm-haiku==0.0.11) (0.1.89)\n",
            "Requirement already satisfied: etils[epy] in /usr/local/lib/python3.11/dist-packages (from optax->flax>=0.7.1->dm-haiku==0.0.11) (1.12.2)\n",
            "Requirement already satisfied: nest_asyncio in /usr/local/lib/python3.11/dist-packages (from orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (1.6.0)\n",
            "Requirement already satisfied: protobuf in /usr/local/lib/python3.11/dist-packages (from orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (5.29.4)\n",
            "Requirement already satisfied: humanize in /usr/local/lib/python3.11/dist-packages (from orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (4.12.2)\n",
            "Requirement already satisfied: simplejson>=3.16.0 in /usr/local/lib/python3.11/dist-packages (from orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (3.20.1)\n",
            "Requirement already satisfied: toolz>=0.9.0 in /usr/local/lib/python3.11/dist-packages (from chex>=0.1.87->optax->flax>=0.7.1->dm-haiku==0.0.11) (0.12.1)\n",
            "Requirement already satisfied: mdurl~=0.1 in /usr/local/lib/python3.11/dist-packages (from markdown-it-py>=2.2.0->rich>=11.1->flax>=0.7.1->dm-haiku==0.0.11) (0.1.2)\n",
            "Requirement already satisfied: fsspec in /usr/local/lib/python3.11/dist-packages (from etils[epath,epy]->orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (2025.3.2)\n",
            "Requirement already satisfied: importlib_resources in /usr/local/lib/python3.11/dist-packages (from etils[epath,epy]->orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (6.5.2)\n",
            "Requirement already satisfied: zipp in /usr/local/lib/python3.11/dist-packages (from etils[epath,epy]->orbax-checkpoint->flax>=0.7.1->dm-haiku==0.0.11) (3.21.0)\n",
            "Downloading dm_haiku-0.0.11-py3-none-any.whl (370 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m371.0/371.0 kB\u001b[0m \u001b[31m32.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jmp-0.0.4-py3-none-any.whl (18 kB)\n",
            "Installing collected packages: jmp, dm-haiku\n",
            "Successfully installed dm-haiku-0.0.11 jmp-0.0.4\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.11/dist-packages (2.0.2)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.11/dist-packages (2.2.2)\n",
            "Requirement already satisfied: numpy>=1.23.2 in /usr/local/lib/python3.11/dist-packages (from pandas) (2.0.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.11/dist-packages (from pandas) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.11/dist-packages (from pandas) (2025.2)\n",
            "Requirement already satisfied: tzdata>=2022.7 in /usr/local/lib/python3.11/dist-packages (from pandas) (2025.2)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.11/dist-packages (from python-dateutil>=2.8.2->pandas) (1.17.0)\n",
            "Collecting biopython==1.81\n",
            "  Downloading biopython-1.81-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (13 kB)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.11/dist-packages (from biopython==1.81) (2.0.2)\n",
            "Downloading biopython-1.81-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m3.1/3.1 MB\u001b[0m \u001b[31m82.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: biopython\n",
            "Successfully installed biopython-1.81\n",
            "Collecting chex==0.1.5\n",
            "  Downloading chex-0.1.5-py3-none-any.whl.metadata (17 kB)\n",
            "Requirement already satisfied: absl-py>=0.9.0 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (1.4.0)\n",
            "Requirement already satisfied: dm-tree>=0.1.5 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (0.1.9)\n",
            "Requirement already satisfied: jax>=0.1.55 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (0.5.3)\n",
            "Requirement already satisfied: jaxlib>=0.1.37 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (0.5.3)\n",
            "Requirement already satisfied: numpy>=1.18.0 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (2.0.2)\n",
            "Requirement already satisfied: toolz>=0.9.0 in /usr/local/lib/python3.11/dist-packages (from chex==0.1.5) (0.12.1)\n",
            "Requirement already satisfied: attrs>=18.2.0 in /usr/local/lib/python3.11/dist-packages (from dm-tree>=0.1.5->chex==0.1.5) (25.3.0)\n",
            "Requirement already satisfied: wrapt>=1.11.2 in /usr/local/lib/python3.11/dist-packages (from dm-tree>=0.1.5->chex==0.1.5) (1.17.2)\n",
            "Requirement already satisfied: ml_dtypes>=0.4.0 in /usr/local/lib/python3.11/dist-packages (from jax>=0.1.55->chex==0.1.5) (0.4.1)\n",
            "Requirement already satisfied: opt_einsum in /usr/local/lib/python3.11/dist-packages (from jax>=0.1.55->chex==0.1.5) (3.4.0)\n",
            "Requirement already satisfied: scipy>=1.11.1 in /usr/local/lib/python3.11/dist-packages (from jax>=0.1.55->chex==0.1.5) (1.14.1)\n",
            "Downloading chex-0.1.5-py3-none-any.whl (85 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m85.3/85.3 kB\u001b[0m \u001b[31m9.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: chex\n",
            "  Attempting uninstall: chex\n",
            "    Found existing installation: chex 0.1.89\n",
            "    Uninstalling chex-0.1.89:\n",
            "      Successfully uninstalled chex-0.1.89\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "optax 0.2.4 requires chex>=0.1.87, but you have chex 0.1.5 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed chex-0.1.5\n",
            "Collecting dm-tree==0.1.8\n",
            "  Downloading dm_tree-0.1.8-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.9 kB)\n",
            "Downloading dm_tree-0.1.8-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (152 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m152.8/152.8 kB\u001b[0m \u001b[31m12.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: dm-tree\n",
            "  Attempting uninstall: dm-tree\n",
            "    Found existing installation: dm-tree 0.1.9\n",
            "    Uninstalling dm-tree-0.1.9:\n",
            "      Successfully uninstalled dm-tree-0.1.9\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "optax 0.2.4 requires chex>=0.1.87, but you have chex 0.1.5 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed dm-tree-0.1.8\n",
            "Collecting immutabledict==2.0.0\n",
            "  Downloading immutabledict-2.0.0-py3-none-any.whl.metadata (2.7 kB)\n",
            "Downloading immutabledict-2.0.0-py3-none-any.whl (4.0 kB)\n",
            "Installing collected packages: immutabledict\n",
            "  Attempting uninstall: immutabledict\n",
            "    Found existing installation: immutabledict 4.2.1\n",
            "    Uninstalling immutabledict-4.2.1:\n",
            "      Successfully uninstalled immutabledict-4.2.1\n",
            "Successfully installed immutabledict-2.0.0\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.11/dist-packages (2.0.2)\n",
            "Collecting numpy\n",
            "  Downloading numpy-2.2.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (62 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m62.0/62.0 kB\u001b[0m \u001b[31m5.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading numpy-2.2.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (16.4 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m16.4/16.4 MB\u001b[0m \u001b[31m86.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: numpy\n",
            "  Attempting uninstall: numpy\n",
            "    Found existing installation: numpy 2.0.2\n",
            "    Uninstalling numpy-2.0.2:\n",
            "      Successfully uninstalled numpy-2.0.2\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "numba 0.60.0 requires numpy<2.1,>=1.22, but you have numpy 2.2.4 which is incompatible.\n",
            "tensorflow 2.18.0 requires numpy<2.1.0,>=1.26.0, but you have numpy 2.2.4 which is incompatible.\n",
            "optax 0.2.4 requires chex>=0.1.87, but you have chex 0.1.5 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed numpy-2.2.4\n",
            "\u001b[31mERROR: Ignored the following yanked versions: 1.11.0, 1.14.0rc1\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[31mERROR: Ignored the following versions that require a different python version: 1.6.2 Requires-Python >=3.7,<3.10; 1.6.3 Requires-Python >=3.7,<3.10; 1.7.0 Requires-Python >=3.7,<3.10; 1.7.1 Requires-Python >=3.7,<3.10; 1.7.2 Requires-Python >=3.7,<3.11; 1.7.3 Requires-Python >=3.7,<3.11; 1.8.0 Requires-Python >=3.8,<3.11; 1.8.0rc1 Requires-Python >=3.8,<3.11; 1.8.0rc2 Requires-Python >=3.8,<3.11; 1.8.0rc3 Requires-Python >=3.8,<3.11; 1.8.0rc4 Requires-Python >=3.8,<3.11; 1.8.1 Requires-Python >=3.8,<3.11\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[31mERROR: Could not find a version that satisfies the requirement scipy==1.7.3 (from versions: 0.8.0, 0.9.0, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.12.1, 0.13.0, 0.13.1, 0.13.2, 0.13.3, 0.14.0, 0.14.1, 0.15.0, 0.15.1, 0.16.0, 0.16.1, 0.17.0, 0.17.1, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 1.0.0, 1.0.1, 1.1.0, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.6.0, 1.6.1, 1.9.0rc1, 1.9.0rc2, 1.9.0rc3, 1.9.0, 1.9.1, 1.9.2, 1.9.3, 1.10.0rc1, 1.10.0rc2, 1.10.0, 1.10.1, 1.11.0rc1, 1.11.0rc2, 1.11.1, 1.11.2, 1.11.3, 1.11.4, 1.12.0rc1, 1.12.0rc2, 1.12.0, 1.13.0rc1, 1.13.0, 1.13.1, 1.14.0rc2, 1.14.0, 1.14.1, 1.15.0rc1, 1.15.0rc2, 1.15.0, 1.15.1, 1.15.2)\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[31mERROR: No matching distribution found for scipy==1.7.3\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[31mERROR: Could not find a version that satisfies the requirement tensorflow==2.11.0 (from versions: 2.12.0rc0, 2.12.0rc1, 2.12.0, 2.12.1, 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0, 2.13.1, 2.14.0rc0, 2.14.0rc1, 2.14.0, 2.14.1, 2.15.0rc0, 2.15.0rc1, 2.15.0, 2.15.0.post1, 2.15.1, 2.16.0rc0, 2.16.1, 2.16.2, 2.17.0rc0, 2.17.0rc1, 2.17.0, 2.17.1, 2.18.0rc0, 2.18.0rc1, 2.18.0rc2, 2.18.0, 2.18.1, 2.19.0rc0, 2.19.0)\u001b[0m\u001b[31m\n",
            "\u001b[0m\u001b[31mERROR: No matching distribution found for tensorflow==2.11.0\u001b[0m\u001b[31m\n",
            "\u001b[0mCollecting rdkit-pypi\n",
            "  Downloading rdkit_pypi-2022.9.5-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.9 kB)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.11/dist-packages (from rdkit-pypi) (2.2.4)\n",
            "Requirement already satisfied: Pillow in /usr/local/lib/python3.11/dist-packages (from rdkit-pypi) (11.1.0)\n",
            "Downloading rdkit_pypi-2022.9.5-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (29.4 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m29.4/29.4 MB\u001b[0m \u001b[31m47.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hInstalling collected packages: rdkit-pypi\n",
            "Successfully installed rdkit-pypi-2022.9.5\n",
            "Collecting py3Dmol\n",
            "  Downloading py3Dmol-2.4.2-py2.py3-none-any.whl.metadata (1.9 kB)\n",
            "Downloading py3Dmol-2.4.2-py2.py3-none-any.whl (7.0 kB)\n",
            "Installing collected packages: py3Dmol\n",
            "Successfully installed py3Dmol-2.4.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Check that the GPU is accessible. The response from this cell should be \"gpu\".\n",
        "import jax\n",
        "from jax.lib import xla_bridge\n",
        "print(xla_bridge.get_backend().platform)"
      ],
      "metadata": {
        "cellView": "form",
        "id": "cRl_0898E2Kk",
        "outputId": "c666b16d-e498-4885-b7b2-f04599971f1e",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "<ipython-input-2-7c59e3a01619>:4: DeprecationWarning: jax.lib.xla_bridge.get_backend is deprecated; use jax.extend.backend.get_backend.\n",
            "  print(xla_bridge.get_backend().platform)\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "gpu\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Clone the Umol github repo\n",
        "import shutil\n",
        "try:\n",
        "  shutil.rmtree('/content/Umol', ignore_errors=True)\n",
        "except:\n",
        "  print('')\n",
        "\n",
        "!git clone https://github.com/patrickbryant1/Umol.git"
      ],
      "metadata": {
        "cellView": "form",
        "id": "AY2dLdvxhDV1",
        "outputId": "63d23495-badd-44c0-93f3-917be89f5b18",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Cloning into 'Umol'...\n",
            "remote: Enumerating objects: 529, done.\u001b[K\n",
            "remote: Counting objects: 100% (18/18), done.\u001b[K\n",
            "remote: Compressing objects: 100% (15/15), done.\u001b[K\n",
            "remote: Total 529 (delta 5), reused 9 (delta 3), pack-reused 511 (from 1)\u001b[K\n",
            "Receiving objects: 100% (529/529), 2.08 MiB | 20.69 MiB/s, done.\n",
            "Resolving deltas: 100% (259/259), done.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@title #Follow all steps outlined below to run Umol\n",
        "#@markdown To try the **test case** 7NB4, click the box \"test_case\". Then press the play button to the left.\n",
        "\\\n",
        "#@markdown If you don't want to run the test case, **leave the box blank**.\n",
        "\\\n",
        "#@markdown The target positions are visualised on a structure generated from [ESMFold](https://www.science.org/doi/10.1126/science.ade2574).\n",
        "#@markdown You can opt out of adding target positions, but the accuracy is increased if you do.\n",
        "\n",
        "#@markdown #Settings\n",
        "#@markdown - *ID* - name \\\n",
        "#@markdown - **MSA** - currently no MSA search is available directly in the browser, therefore you have to provide your own MSAs in a3m format and upload them here. \\\n",
        "#@markdown - Generating an MSA takes a few minutes: \\\n",
        "#@markdown Go to https://toolkit.tuebingen.mpg.de/tools/hhblits \\\n",
        "#@markdown Paste your protein sequence in the search field in fasta format --> Submit. \\\n",
        "#@markdown When the search is finished, go to the tab \"Query MSA\" and \"Download Full A3M\" \\\n",
        "#@markdown Upload the MSAs here: \\\n",
        "#@markdown Click the folder icon (Files) to the left and select the upload file icon. Upload your files.\n",
        "#@markdown Make sure to name your MSA **\"ID\".a3m**\n",
        "\n",
        "#@markdown - LIGAND - smiles string of your ligand. Make sure these are canonical (e.g. as generated by RDKit)\n",
        "#@markdown - SEQUENCE - protein sequence (same as used for the MSA). **Limit=400 amino acids**.\n",
        "#@markdown You can crop the structure around the target region if it is too big - Umol will handle it.\n",
        "#@markdown - TARGET_POS - **OPTIONAL** (leave blank if none). What positions to target in the protein sequence (the binding pocket, starting at 1). Note that these have to be all CBs within 10 Å from the putative ligand position.\n",
        "#@markdown e.g. \"50,51,53,54,55,56,57,58,59,60,61,62,64,65,76,77,78,79,80,81,82,83,84,85,86,87,88,89,90,92,93,94,95,96,97,98,99,100,101,103,104,124,127,128\"\n",
        "#@markdown - NUM_RECYCLES - how many recycles to use in the network (increase for harder targets)\n",
        "\n",
        "\n",
        "import sys, os\n",
        "from google.colab import files\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "import py3Dmol\n",
        "\n",
        "sys.path.insert(0,'/content/Umol/src')\n",
        "test_case = True #@param {type:\"boolean\"}\n",
        "ID = \"Hit-1\" #@param {type:\"string\"}\n",
        "LIGAND = \"c1ccc2c(c1)OCC(c1nc(SSc3n[nH]c(C4COc5ccccc5O4)n3)n[nH]1)O2\" # @param {type:\"string\"}\n",
        "SEQUENCE = \"VLSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHGKKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPAEFTPAVHASLDKFLASVSTVLTSKYR\" #@param {type:\"string\"}\n",
        "TARGET_POSITIONS = \"\" #@param {type:\"string\"}\n",
        "if len(TARGET_POSITIONS)>0:\n",
        "  TARGET_POSITIONS = np.array([int(x) for x in TARGET_POSITIONS.split(',')])\n",
        "else:\n",
        "  TARGET_POSITIONS = []\n",
        "NUM_RECYCLES = 3 # @param {type:\"integer\"}\n",
        "OUTDIR=\"/content/\"+ID+'/'\n",
        "if not os.path.exists(OUTDIR):\n",
        "  os.mkdir(OUTDIR)\n",
        "\n",
        " #Write fasta\n",
        "with open('/content/'+ID+'.fasta', 'w') as file:\n",
        "  file.write('>'+ID+'\\n')\n",
        "  file.write(SEQUENCE)\n",
        "FASTA_FILE='/content/'+ID+'.fasta'\n",
        "\n",
        "#Check MSA\n",
        "if test_case!=True:\n",
        "  from check_msa_colab import process_a3m\n",
        "  MSA='/content/'+ID+'.a3m'\n",
        "  PROCESSED_MSA=MSA.split('.')[0]+'_processed.a3m'\n",
        "  process_a3m(MSA, SEQUENCE, PROCESSED_MSA)\n",
        "  MSA=PROCESSED_MSA\n",
        "else:\n",
        "  MSA='/content/Umol/data/test_case/'+ID+'/'+ID+'.a3m'\n",
        "\n",
        "\n",
        "print('Using ligand:', LIGAND)\n",
        "#print('Using MSA:' ,MSA)\n",
        "print('Using',NUM_RECYCLES,'recycles')\n",
        "\n",
        "#Visualise with ESMFold\n",
        "if not os.path.exists(OUTDIR+'/'+ID+'_esmfold.pdb'):\n",
        "  !curl -k -X POST --data $SEQUENCE  https://api.esmatlas.com/foldSequence/v1/pdb/ >> $OUTDIR/$ID'_esmfold.pdb'\n",
        "\n",
        "\n",
        "view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "view.addModel(open(OUTDIR+ID+\"_esmfold.pdb\",'r').read(),'pdb')\n",
        "view.setStyle({'chain':'A'},{'cartoon': {'color':'green'}})\n",
        "\n",
        "#Highlight\n",
        "with open(OUTDIR+ID+\"_esmfold.pdb\",\"r\") as ifile:\n",
        "    system = \"\".join([x for x in ifile])\n",
        "i = 0\n",
        "for line in system.split(\"\\n\"):\n",
        "    split = line.split()\n",
        "    if len(split) == 0 or split[0] != \"ATOM\":\n",
        "        continue\n",
        "    idx = int(split[5])\n",
        "    if idx in TARGET_POSITIONS:\n",
        "        color = \"green\"\n",
        "        view.setStyle({'model': -1, 'serial': i+1}, {\"stick\": {'color': color}})\n",
        "    i += 1\n",
        "\n",
        "view.zoomTo()\n",
        "view.show()\n",
        "\n",
        "print('The target structure is in cartoon and the target positions (pocket) in stick format.')\n",
        "\n",
        "#Subtract 1 from the target positions to make them zero indexed\n",
        "if len(TARGET_POSITIONS)>0:\n",
        "  TARGET_POSITIONS -= 1"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 601
        },
        "cellView": "form",
        "id": "GcurRs4ResXi",
        "outputId": "92c72774-1453-423a-f5f5-9a0c9364ec44"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using ligand: c1ccc2c(c1)OCC(c1nc(SSc3n[nH]c(C4COc5ccccc5O4)n3)n[nH]1)O2\n",
            "Using 3 recycles\n",
            "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
            "                                 Dload  Upload   Total   Spent    Left  Speed\n",
            "100 88149  100 88008  100   141  73961    118  0:00:01  0:00:01 --:--:-- 74137\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1744327365365795\"  style=\"position: relative; width: 640px; height: 480px;\">\n        <p id=\"3dmolwarning_1744327365365795\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_1744327365365795 = null;\nvar warn = document.getElementById(\"3dmolwarning_1744327365365795\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1744327365365795 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1744327365365795\"),{backgroundColor:\"white\"});\nviewer_1744327365365795.zoomTo();\n\tviewer_1744327365365795.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   VAL A   1       9.646   6.431 -12.355  1.00  0.83           N  \\nATOM      2  CA  VAL A   1       8.465   6.885 -13.083  1.00  0.85           C  \\nATOM      3  C   VAL A   1       7.678   5.680 -13.595  1.00  0.90           C  \\nATOM      4  CB  VAL A   1       8.845   7.812 -14.259  1.00  0.79           C  \\nATOM      5  O   VAL A   1       8.263   4.713 -14.088  1.00  0.89           O  \\nATOM      6  CG1 VAL A   1       7.602   8.222 -15.046  1.00  0.65           C  \\nATOM      7  CG2 VAL A   1       9.589   9.044 -13.747  1.00  0.65           C  \\nATOM      8  N   LEU A   2       6.407   5.746 -13.541  1.00  0.95           N  \\nATOM      9  CA  LEU A   2       5.512   4.684 -13.986  1.00  0.95           C  \\nATOM     10  C   LEU A   2       5.221   4.807 -15.477  1.00  0.94           C  \\nATOM     11  CB  LEU A   2       4.202   4.720 -13.193  1.00  0.93           C  \\nATOM     12  O   LEU A   2       4.777   5.859 -15.943  1.00  0.93           O  \\nATOM     13  CG  LEU A   2       4.310   4.439 -11.694  1.00  0.88           C  \\nATOM     14  CD1 LEU A   2       2.998   4.784 -10.995  1.00  0.81           C  \\nATOM     15  CD2 LEU A   2       4.686   2.982 -11.447  1.00  0.81           C  \\nATOM     16  N   SER A   3       5.533   3.858 -16.212  1.00  0.96           N  \\nATOM     17  CA  SER A   3       5.147   3.805 -17.619  1.00  0.96           C  \\nATOM     18  C   SER A   3       3.643   3.604 -17.773  1.00  0.96           C  \\nATOM     19  CB  SER A   3       5.896   2.682 -18.338  1.00  0.95           C  \\nATOM     20  O   SER A   3       2.957   3.257 -16.809  1.00  0.96           O  \\nATOM     21  OG  SER A   3       5.422   1.413 -17.921  1.00  0.86           O  \\nATOM     22  N   PRO A   4       3.134   3.858 -18.947  1.00  0.96           N  \\nATOM     23  CA  PRO A   4       1.715   3.585 -19.183  1.00  0.96           C  \\nATOM     24  C   PRO A   4       1.333   2.139 -18.872  1.00  0.96           C  \\nATOM     25  CB  PRO A   4       1.540   3.887 -20.673  1.00  0.95           C  \\nATOM     26  O   PRO A   4       0.262   1.886 -18.314  1.00  0.96           O  \\nATOM     27  CG  PRO A   4       2.577   4.919 -20.978  1.00  0.92           C  \\nATOM     28  CD  PRO A   4       3.808   4.609 -20.175  1.00  0.90           C  \\nATOM     29  N   ALA A   5       2.245   1.221 -19.202  1.00  0.96           N  \\nATOM     30  CA  ALA A   5       1.996  -0.186 -18.898  1.00  0.96           C  \\nATOM     31  C   ALA A   5       1.939  -0.419 -17.391  1.00  0.97           C  \\nATOM     32  CB  ALA A   5       3.072  -1.066 -19.529  1.00  0.96           C  \\nATOM     33  O   ALA A   5       1.097  -1.178 -16.905  1.00  0.97           O  \\nATOM     34  N   ASP A   6       2.814   0.196 -16.642  1.00  0.97           N  \\nATOM     35  CA  ASP A   6       2.795   0.095 -15.186  1.00  0.97           C  \\nATOM     36  C   ASP A   6       1.472   0.605 -14.617  1.00  0.97           C  \\nATOM     37  CB  ASP A   6       3.963   0.874 -14.579  1.00  0.97           C  \\nATOM     38  O   ASP A   6       0.876  -0.036 -13.749  1.00  0.97           O  \\nATOM     39  CG  ASP A   6       5.314   0.250 -14.884  1.00  0.95           C  \\nATOM     40  OD1 ASP A   6       5.405  -0.993 -14.978  1.00  0.94           O  \\nATOM     41  OD2 ASP A   6       6.297   1.009 -15.029  1.00  0.93           O  \\nATOM     42  N   LYS A   7       1.050   1.711 -15.118  1.00  0.97           N  \\nATOM     43  CA  LYS A   7      -0.194   2.296 -14.626  1.00  0.97           C  \\nATOM     44  C   LYS A   7      -1.379   1.373 -14.894  1.00  0.97           C  \\nATOM     45  CB  LYS A   7      -0.437   3.662 -15.271  1.00  0.97           C  \\nATOM     46  O   LYS A   7      -2.240   1.193 -14.030  1.00  0.97           O  \\nATOM     47  CG  LYS A   7       0.608   4.709 -14.916  1.00  0.94           C  \\nATOM     48  CD  LYS A   7       0.359   6.019 -15.652  1.00  0.92           C  \\nATOM     49  CE  LYS A   7       1.456   7.037 -15.370  1.00  0.89           C  \\nATOM     50  NZ  LYS A   7       1.208   8.329 -16.077  1.00  0.85           N  \\nATOM     51  N   THR A   8      -1.411   0.813 -16.028  1.00  0.97           N  \\nATOM     52  CA  THR A   8      -2.460  -0.131 -16.394  1.00  0.97           C  \\nATOM     53  C   THR A   8      -2.438  -1.348 -15.474  1.00  0.97           C  \\nATOM     54  CB  THR A   8      -2.316  -0.588 -17.858  1.00  0.96           C  \\nATOM     55  O   THR A   8      -3.481  -1.773 -14.972  1.00  0.96           O  \\nATOM     56  CG2 THR A   8      -3.359  -1.643 -18.210  1.00  0.90           C  \\nATOM     57  OG1 THR A   8      -2.483   0.541 -18.724  1.00  0.90           O  \\nATOM     58  N   ASN A   9      -1.287  -1.887 -15.265  1.00  0.97           N  \\nATOM     59  CA  ASN A   9      -1.134  -3.041 -14.386  1.00  0.97           C  \\nATOM     60  C   ASN A   9      -1.584  -2.724 -12.962  1.00  0.97           C  \\nATOM     61  CB  ASN A   9       0.316  -3.530 -14.388  1.00  0.96           C  \\nATOM     62  O   ASN A   9      -2.280  -3.524 -12.334  1.00  0.97           O  \\nATOM     63  CG  ASN A   9       0.669  -4.305 -15.642  1.00  0.93           C  \\nATOM     64  ND2 ASN A   9       1.960  -4.395 -15.940  1.00  0.89           N  \\nATOM     65  OD1 ASN A   9      -0.211  -4.818 -16.338  1.00  0.88           O  \\nATOM     66  N   VAL A  10      -1.174  -1.563 -12.462  1.00  0.97           N  \\nATOM     67  CA  VAL A  10      -1.507  -1.159 -11.100  1.00  0.97           C  \\nATOM     68  C   VAL A  10      -3.019  -0.995 -10.965  1.00  0.97           C  \\nATOM     69  CB  VAL A  10      -0.791   0.152 -10.705  1.00  0.97           C  \\nATOM     70  O   VAL A  10      -3.623  -1.501 -10.016  1.00  0.97           O  \\nATOM     71  CG1 VAL A  10      -1.389   0.729  -9.423  1.00  0.94           C  \\nATOM     72  CG2 VAL A  10       0.708  -0.089 -10.538  1.00  0.94           C  \\nATOM     73  N   LYS A  11      -3.631  -0.324 -11.906  1.00  0.97           N  \\nATOM     74  CA  LYS A  11      -5.075  -0.113 -11.853  1.00  0.97           C  \\nATOM     75  C   LYS A  11      -5.828  -1.437 -11.945  1.00  0.97           C  \\nATOM     76  CB  LYS A  11      -5.522   0.824 -12.976  1.00  0.96           C  \\nATOM     77  O   LYS A  11      -6.808  -1.651 -11.229  1.00  0.96           O  \\nATOM     78  CG  LYS A  11      -5.118   2.276 -12.768  1.00  0.92           C  \\nATOM     79  CD  LYS A  11      -5.546   3.149 -13.942  1.00  0.89           C  \\nATOM     80  CE  LYS A  11      -5.094   4.591 -13.761  1.00  0.83           C  \\nATOM     81  NZ  LYS A  11      -5.461   5.438 -14.935  1.00  0.79           N  \\nATOM     82  N   ALA A  12      -5.348  -2.276 -12.806  1.00  0.97           N  \\nATOM     83  CA  ALA A  12      -5.997  -3.574 -12.971  1.00  0.97           C  \\nATOM     84  C   ALA A  12      -5.892  -4.406 -11.696  1.00  0.97           C  \\nATOM     85  CB  ALA A  12      -5.384  -4.330 -14.148  1.00  0.96           C  \\nATOM     86  O   ALA A  12      -6.889  -4.956 -11.223  1.00  0.96           O  \\nATOM     87  N   ALA A  13      -4.705  -4.494 -11.137  1.00  0.97           N  \\nATOM     88  CA  ALA A  13      -4.499  -5.267  -9.915  1.00  0.97           C  \\nATOM     89  C   ALA A  13      -5.304  -4.686  -8.756  1.00  0.97           C  \\nATOM     90  CB  ALA A  13      -3.015  -5.313  -9.559  1.00  0.96           C  \\nATOM     91  O   ALA A  13      -5.935  -5.427  -7.998  1.00  0.96           O  \\nATOM     92  N   TRP A  14      -5.271  -3.390  -8.654  1.00  0.97           N  \\nATOM     93  CA  TRP A  14      -5.993  -2.749  -7.560  1.00  0.97           C  \\nATOM     94  C   TRP A  14      -7.498  -2.949  -7.711  1.00  0.96           C  \\nATOM     95  CB  TRP A  14      -5.667  -1.255  -7.503  1.00  0.96           C  \\nATOM     96  O   TRP A  14      -8.219  -3.054  -6.716  1.00  0.95           O  \\nATOM     97  CG  TRP A  14      -6.024  -0.603  -6.201  1.00  0.89           C  \\nATOM     98  CD1 TRP A  14      -7.078   0.233  -5.960  1.00  0.79           C  \\nATOM     99  CD2 TRP A  14      -5.322  -0.731  -4.961  1.00  0.80           C  \\nATOM    100  CE2 TRP A  14      -6.007   0.056  -4.009  1.00  0.79           C  \\nATOM    101  CE3 TRP A  14      -4.178  -1.436  -4.562  1.00  0.74           C  \\nATOM    102  NE1 TRP A  14      -7.074   0.633  -4.643  1.00  0.84           N  \\nATOM    103  CH2 TRP A  14      -4.464  -0.543  -2.317  1.00  0.81           C  \\nATOM    104  CZ2 TRP A  14      -5.586   0.157  -2.681  1.00  0.83           C  \\nATOM    105  CZ3 TRP A  14      -3.760  -1.334  -3.240  1.00  0.80           C  \\nATOM    106  N   GLY A  15      -7.971  -2.975  -8.950  1.00  0.96           N  \\nATOM    107  CA  GLY A  15      -9.366  -3.298  -9.207  1.00  0.96           C  \\nATOM    108  C   GLY A  15      -9.775  -4.652  -8.660  1.00  0.95           C  \\nATOM    109  O   GLY A  15     -10.908  -4.827  -8.205  1.00  0.94           O  \\nATOM    110  N   LYS A  16      -8.793  -5.598  -8.617  1.00  0.96           N  \\nATOM    111  CA  LYS A  16      -9.058  -6.939  -8.103  1.00  0.96           C  \\nATOM    112  C   LYS A  16      -9.075  -6.950  -6.577  1.00  0.95           C  \\nATOM    113  CB  LYS A  16      -8.015  -7.929  -8.623  1.00  0.94           C  \\nATOM    114  O   LYS A  16      -9.733  -7.794  -5.964  1.00  0.95           O  \\nATOM    115  CG  LYS A  16      -8.019  -8.096 -10.135  1.00  0.87           C  \\nATOM    116  CD  LYS A  16      -9.309  -8.743 -10.623  1.00  0.83           C  \\nATOM    117  CE  LYS A  16      -9.282  -8.981 -12.127  1.00  0.77           C  \\nATOM    118  NZ  LYS A  16     -10.562  -9.573 -12.617  1.00  0.74           N  \\nATOM    119  N   VAL A  17      -8.330  -6.052  -5.973  1.00  0.95           N  \\nATOM    120  CA  VAL A  17      -8.340  -5.913  -4.521  1.00  0.95           C  \\nATOM    121  C   VAL A  17      -9.734  -5.502  -4.051  1.00  0.93           C  \\nATOM    122  CB  VAL A  17      -7.291  -4.885  -4.041  1.00  0.93           C  \\nATOM    123  O   VAL A  17     -10.261  -6.061  -3.087  1.00  0.91           O  \\nATOM    124  CG1 VAL A  17      -7.479  -4.574  -2.557  1.00  0.88           C  \\nATOM    125  CG2 VAL A  17      -5.879  -5.402  -4.307  1.00  0.88           C  \\nATOM    126  N   GLY A  18     -10.306  -4.545  -4.797  1.00  0.93           N  \\nATOM    127  CA  GLY A  18     -11.685  -4.146  -4.567  1.00  0.93           C  \\nATOM    128  C   GLY A  18     -11.990  -3.866  -3.107  1.00  0.93           C  \\nATOM    129  O   GLY A  18     -11.256  -3.128  -2.445  1.00  0.89           O  \\nATOM    130  N   ALA A  19     -13.072  -4.451  -2.666  1.00  0.92           N  \\nATOM    131  CA  ALA A  19     -13.642  -4.215  -1.342  1.00  0.91           C  \\nATOM    132  C   ALA A  19     -12.757  -4.807  -0.249  1.00  0.92           C  \\nATOM    133  CB  ALA A  19     -15.050  -4.800  -1.256  1.00  0.88           C  \\nATOM    134  O   ALA A  19     -12.992  -4.580   0.940  1.00  0.90           O  \\nATOM    135  N   HIS A  20     -11.810  -5.596  -0.593  1.00  0.95           N  \\nATOM    136  CA  HIS A  20     -10.943  -6.252   0.379  1.00  0.95           C  \\nATOM    137  C   HIS A  20      -9.755  -5.368   0.743  1.00  0.95           C  \\nATOM    138  CB  HIS A  20     -10.451  -7.595  -0.164  1.00  0.93           C  \\nATOM    139  O   HIS A  20      -8.888  -5.776   1.519  1.00  0.95           O  \\nATOM    140  CG  HIS A  20     -11.552  -8.567  -0.449  1.00  0.87           C  \\nATOM    141  CD2 HIS A  20     -12.054  -9.022  -1.621  1.00  0.79           C  \\nATOM    142  ND1 HIS A  20     -12.274  -9.186   0.547  1.00  0.81           N  \\nATOM    143  CE1 HIS A  20     -13.175  -9.984  -0.001  1.00  0.78           C  \\nATOM    144  NE2 HIS A  20     -13.063  -9.903  -1.316  1.00  0.80           N  \\nATOM    145  N   ALA A  21      -9.733  -4.179   0.206  1.00  0.95           N  \\nATOM    146  CA  ALA A  21      -8.559  -3.325   0.366  1.00  0.95           C  \\nATOM    147  C   ALA A  21      -8.251  -3.088   1.842  1.00  0.96           C  \\nATOM    148  CB  ALA A  21      -8.768  -1.993  -0.351  1.00  0.94           C  \\nATOM    149  O   ALA A  21      -7.095  -3.170   2.262  1.00  0.95           O  \\nATOM    150  N   GLY A  22      -9.287  -2.789   2.643  1.00  0.95           N  \\nATOM    151  CA  GLY A  22      -9.083  -2.554   4.064  1.00  0.95           C  \\nATOM    152  C   GLY A  22      -8.518  -3.759   4.793  1.00  0.95           C  \\nATOM    153  O   GLY A  22      -7.618  -3.622   5.624  1.00  0.95           O  \\nATOM    154  N   GLU A  23      -9.010  -4.925   4.464  1.00  0.96           N  \\nATOM    155  CA  GLU A  23      -8.541  -6.173   5.060  1.00  0.96           C  \\nATOM    156  C   GLU A  23      -7.091  -6.457   4.676  1.00  0.96           C  \\nATOM    157  CB  GLU A  23      -9.436  -7.340   4.636  1.00  0.93           C  \\nATOM    158  O   GLU A  23      -6.295  -6.888   5.513  1.00  0.95           O  \\nATOM    159  CG  GLU A  23      -9.026  -8.678   5.237  1.00  0.76           C  \\nATOM    160  CD  GLU A  23      -9.837  -9.848   4.704  1.00  0.70           C  \\nATOM    161  OE1 GLU A  23     -10.789  -9.621   3.922  1.00  0.66           O  \\nATOM    162  OE2 GLU A  23      -9.519 -11.001   5.071  1.00  0.64           O  \\nATOM    163  N   TYR A  24      -6.771  -6.225   3.410  1.00  0.96           N  \\nATOM    164  CA  TYR A  24      -5.407  -6.442   2.941  1.00  0.96           C  \\nATOM    165  C   TYR A  24      -4.447  -5.444   3.577  1.00  0.96           C  \\nATOM    166  CB  TYR A  24      -5.340  -6.331   1.414  1.00  0.96           C  \\nATOM    167  O   TYR A  24      -3.313  -5.792   3.915  1.00  0.96           O  \\nATOM    168  CG  TYR A  24      -6.081  -7.430   0.693  1.00  0.94           C  \\nATOM    169  CD1 TYR A  24      -6.660  -8.484   1.396  1.00  0.91           C  \\nATOM    170  CD2 TYR A  24      -6.203  -7.418  -0.692  1.00  0.91           C  \\nATOM    171  CE1 TYR A  24      -7.342  -9.501   0.735  1.00  0.90           C  \\nATOM    172  CE2 TYR A  24      -6.883  -8.429  -1.364  1.00  0.90           C  \\nATOM    173  OH  TYR A  24      -8.123 -10.467  -1.302  1.00  0.86           O  \\nATOM    174  CZ  TYR A  24      -7.449  -9.464  -0.642  1.00  0.89           C  \\nATOM    175  N   GLY A  25      -4.934  -4.213   3.746  1.00  0.96           N  \\nATOM    176  CA  GLY A  25      -4.145  -3.230   4.472  1.00  0.96           C  \\nATOM    177  C   GLY A  25      -3.849  -3.641   5.901  1.00  0.97           C  \\nATOM    178  O   GLY A  25      -2.713  -3.518   6.365  1.00  0.96           O  \\nATOM    179  N   ALA A  26      -4.842  -4.125   6.589  1.00  0.96           N  \\nATOM    180  CA  ALA A  26      -4.669  -4.571   7.969  1.00  0.96           C  \\nATOM    181  C   ALA A  26      -3.692  -5.741   8.049  1.00  0.96           C  \\nATOM    182  CB  ALA A  26      -6.014  -4.963   8.575  1.00  0.96           C  \\nATOM    183  O   ALA A  26      -2.831  -5.780   8.931  1.00  0.96           O  \\nATOM    184  N   GLU A  27      -3.808  -6.672   7.135  1.00  0.96           N  \\nATOM    185  CA  GLU A  27      -2.908  -7.821   7.118  1.00  0.96           C  \\nATOM    186  C   GLU A  27      -1.469  -7.393   6.845  1.00  0.96           C  \\nATOM    187  CB  GLU A  27      -3.360  -8.842   6.070  1.00  0.96           C  \\nATOM    188  O   GLU A  27      -0.535  -7.905   7.465  1.00  0.96           O  \\nATOM    189  CG  GLU A  27      -2.507 -10.101   6.030  1.00  0.93           C  \\nATOM    190  CD  GLU A  27      -3.001 -11.129   5.025  1.00  0.90           C  \\nATOM    191  OE1 GLU A  27      -4.206 -11.117   4.686  1.00  0.88           O  \\nATOM    192  OE2 GLU A  27      -2.176 -11.954   4.572  1.00  0.88           O  \\nATOM    193  N   ALA A  28      -1.278  -6.482   5.928  1.00  0.97           N  \\nATOM    194  CA  ALA A  28       0.063  -5.984   5.633  1.00  0.97           C  \\nATOM    195  C   ALA A  28       0.702  -5.363   6.872  1.00  0.97           C  \\nATOM    196  CB  ALA A  28       0.014  -4.966   4.496  1.00  0.96           C  \\nATOM    197  O   ALA A  28       1.887  -5.578   7.141  1.00  0.96           O  \\nATOM    198  N   LEU A  29      -0.067  -4.567   7.648  1.00  0.97           N  \\nATOM    199  CA  LEU A  29       0.445  -3.977   8.881  1.00  0.97           C  \\nATOM    200  C   LEU A  29       0.800  -5.060   9.894  1.00  0.96           C  \\nATOM    201  CB  LEU A  29      -0.584  -3.017   9.482  1.00  0.96           C  \\nATOM    202  O   LEU A  29       1.838  -4.980  10.555  1.00  0.96           O  \\nATOM    203  CG  LEU A  29      -0.788  -1.692   8.744  1.00  0.94           C  \\nATOM    204  CD1 LEU A  29      -2.007  -0.961   9.296  1.00  0.90           C  \\nATOM    205  CD2 LEU A  29       0.459  -0.822   8.854  1.00  0.90           C  \\nATOM    206  N   GLU A  30      -0.045  -6.019  10.005  1.00  0.96           N  \\nATOM    207  CA  GLU A  30       0.246  -7.104  10.936  1.00  0.96           C  \\nATOM    208  C   GLU A  30       1.520  -7.845  10.540  1.00  0.96           C  \\nATOM    209  CB  GLU A  30      -0.930  -8.082  11.008  1.00  0.96           C  \\nATOM    210  O   GLU A  30       2.349  -8.166  11.394  1.00  0.96           O  \\nATOM    211  CG  GLU A  30      -2.146  -7.529  11.737  1.00  0.92           C  \\nATOM    212  CD  GLU A  30      -3.287  -8.528  11.844  1.00  0.89           C  \\nATOM    213  OE1 GLU A  30      -3.140  -9.672  11.356  1.00  0.87           O  \\nATOM    214  OE2 GLU A  30      -4.336  -8.164  12.420  1.00  0.87           O  \\nATOM    215  N   ARG A  31       1.720  -8.158   9.264  1.00  0.96           N  \\nATOM    216  CA  ARG A  31       2.954  -8.771   8.781  1.00  0.96           C  \\nATOM    217  C   ARG A  31       4.162  -7.901   9.110  1.00  0.96           C  \\nATOM    218  CB  ARG A  31       2.879  -9.015   7.272  1.00  0.96           C  \\nATOM    219  O   ARG A  31       5.213  -8.412   9.503  1.00  0.96           O  \\nATOM    220  CG  ARG A  31       1.935 -10.139   6.876  1.00  0.93           C  \\nATOM    221  CD  ARG A  31       1.893 -10.334   5.367  1.00  0.91           C  \\nATOM    222  NE  ARG A  31       0.905 -11.338   4.981  1.00  0.90           N  \\nATOM    223  NH1 ARG A  31       2.286 -13.148   5.370  1.00  0.83           N  \\nATOM    224  NH2 ARG A  31       0.145 -13.475   4.620  1.00  0.82           N  \\nATOM    225  CZ  ARG A  31       1.114 -12.651   4.991  1.00  0.87           C  \\nATOM    226  N   MET A  32       3.987  -6.637   8.929  1.00  0.96           N  \\nATOM    227  CA  MET A  32       5.097  -5.726   9.191  1.00  0.96           C  \\nATOM    228  C   MET A  32       5.483  -5.747  10.666  1.00  0.96           C  \\nATOM    229  CB  MET A  32       4.736  -4.301   8.765  1.00  0.96           C  \\nATOM    230  O   MET A  32       6.667  -5.821  11.001  1.00  0.95           O  \\nATOM    231  CG  MET A  32       5.866  -3.301   8.949  1.00  0.94           C  \\nATOM    232  SD  MET A  32       5.343  -1.578   8.598  1.00  0.94           S  \\nATOM    233  CE  MET A  32       4.260  -1.275  10.022  1.00  0.88           C  \\nATOM    234  N   PHE A  33       4.507  -5.705  11.540  1.00  0.96           N  \\nATOM    235  CA  PHE A  33       4.784  -5.706  12.972  1.00  0.96           C  \\nATOM    236  C   PHE A  33       5.475  -6.999  13.388  1.00  0.95           C  \\nATOM    237  CB  PHE A  33       3.490  -5.522  13.771  1.00  0.95           C  \\nATOM    238  O   PHE A  33       6.334  -6.992  14.273  1.00  0.94           O  \\nATOM    239  CG  PHE A  33       2.852  -4.171  13.591  1.00  0.95           C  \\nATOM    240  CD1 PHE A  33       3.632  -3.023  13.516  1.00  0.93           C  \\nATOM    241  CD2 PHE A  33       1.472  -4.049  13.495  1.00  0.92           C  \\nATOM    242  CE1 PHE A  33       3.044  -1.771  13.349  1.00  0.92           C  \\nATOM    243  CE2 PHE A  33       0.878  -2.801  13.329  1.00  0.92           C  \\nATOM    244  CZ  PHE A  33       1.665  -1.664  13.257  1.00  0.91           C  \\nATOM    245  N   LEU A  34       5.068  -8.096  12.787  1.00  0.96           N  \\nATOM    246  CA  LEU A  34       5.637  -9.398  13.117  1.00  0.95           C  \\nATOM    247  C   LEU A  34       7.032  -9.549  12.521  1.00  0.95           C  \\nATOM    248  CB  LEU A  34       4.730 -10.523  12.612  1.00  0.94           C  \\nATOM    249  O   LEU A  34       7.944 -10.050  13.183  1.00  0.95           O  \\nATOM    250  CG  LEU A  34       3.410 -10.718  13.360  1.00  0.86           C  \\nATOM    251  CD1 LEU A  34       2.531 -11.728  12.631  1.00  0.77           C  \\nATOM    252  CD2 LEU A  34       3.669 -11.165  14.795  1.00  0.76           C  \\nATOM    253  N   SER A  35       7.230  -9.071  11.302  1.00  0.96           N  \\nATOM    254  CA  SER A  35       8.483  -9.287  10.586  1.00  0.96           C  \\nATOM    255  C   SER A  35       9.517  -8.226  10.948  1.00  0.96           C  \\nATOM    256  CB  SER A  35       8.245  -9.283   9.075  1.00  0.95           C  \\nATOM    257  O   SER A  35      10.722  -8.478  10.882  1.00  0.95           O  \\nATOM    258  OG  SER A  35       7.385 -10.345   8.698  1.00  0.82           O  \\nATOM    259  N   PHE A  36       9.007  -7.040  11.259  1.00  0.95           N  \\nATOM    260  CA  PHE A  36       9.873  -5.909  11.569  1.00  0.96           C  \\nATOM    261  C   PHE A  36       9.412  -5.208  12.842  1.00  0.96           C  \\nATOM    262  CB  PHE A  36       9.899  -4.915  10.403  1.00  0.95           C  \\nATOM    263  O   PHE A  36       8.856  -4.109  12.785  1.00  0.95           O  \\nATOM    264  CG  PHE A  36      10.149  -5.557   9.065  1.00  0.93           C  \\nATOM    265  CD1 PHE A  36      11.403  -6.061   8.743  1.00  0.89           C  \\nATOM    266  CD2 PHE A  36       9.128  -5.656   8.128  1.00  0.89           C  \\nATOM    267  CE1 PHE A  36      11.637  -6.656   7.505  1.00  0.89           C  \\nATOM    268  CE2 PHE A  36       9.355  -6.248   6.889  1.00  0.89           C  \\nATOM    269  CZ  PHE A  36      10.610  -6.747   6.579  1.00  0.87           C  \\nATOM    270  N   PRO A  37       9.723  -5.759  13.973  1.00  0.95           N  \\nATOM    271  CA  PRO A  37       9.161  -5.268  15.233  1.00  0.95           C  \\nATOM    272  C   PRO A  37       9.576  -3.832  15.544  1.00  0.95           C  \\nATOM    273  CB  PRO A  37       9.727  -6.237  16.273  1.00  0.94           C  \\nATOM    274  O   PRO A  37       8.892  -3.138  16.302  1.00  0.94           O  \\nATOM    275  CG  PRO A  37       9.966  -7.507  15.522  1.00  0.91           C  \\nATOM    276  CD  PRO A  37      10.349  -7.162  14.112  1.00  0.90           C  \\nATOM    277  N   THR A  38      10.682  -3.379  15.001  1.00  0.95           N  \\nATOM    278  CA  THR A  38      11.122  -2.008  15.235  1.00  0.94           C  \\nATOM    279  C   THR A  38      10.071  -1.013  14.753  1.00  0.94           C  \\nATOM    280  CB  THR A  38      12.463  -1.725  14.533  1.00  0.93           C  \\nATOM    281  O   THR A  38      10.000   0.113  15.251  1.00  0.94           O  \\nATOM    282  CG2 THR A  38      13.596  -2.524  15.168  1.00  0.80           C  \\nATOM    283  OG1 THR A  38      12.356  -2.084  13.150  1.00  0.81           O  \\nATOM    284  N   THR A  39       9.270  -1.412  13.855  1.00  0.95           N  \\nATOM    285  CA  THR A  39       8.261  -0.507  13.316  1.00  0.96           C  \\nATOM    286  C   THR A  39       7.176  -0.228  14.352  1.00  0.96           C  \\nATOM    287  CB  THR A  39       7.620  -1.082  12.039  1.00  0.95           C  \\nATOM    288  O   THR A  39       6.422   0.738  14.224  1.00  0.95           O  \\nATOM    289  CG2 THR A  39       8.653  -1.242  10.929  1.00  0.92           C  \\nATOM    290  OG1 THR A  39       7.049  -2.363  12.333  1.00  0.92           O  \\nATOM    291  N   LYS A  40       7.037  -1.036  15.335  1.00  0.95           N  \\nATOM    292  CA  LYS A  40       6.016  -0.870  16.365  1.00  0.95           C  \\nATOM    293  C   LYS A  40       6.226   0.425  17.144  1.00  0.95           C  \\nATOM    294  CB  LYS A  40       6.020  -2.064  17.322  1.00  0.95           C  \\nATOM    295  O   LYS A  40       5.293   0.942  17.761  1.00  0.94           O  \\nATOM    296  CG  LYS A  40       5.542  -3.364  16.692  1.00  0.93           C  \\nATOM    297  CD  LYS A  40       5.507  -4.498  17.709  1.00  0.91           C  \\nATOM    298  CE  LYS A  40       4.987  -5.788  17.091  1.00  0.88           C  \\nATOM    299  NZ  LYS A  40       4.901  -6.889  18.096  1.00  0.86           N  \\nATOM    300  N   THR A  41       7.435   0.940  17.200  1.00  0.95           N  \\nATOM    301  CA  THR A  41       7.766   2.148  17.947  1.00  0.94           C  \\nATOM    302  C   THR A  41       6.949   3.335  17.446  1.00  0.94           C  \\nATOM    303  CB  THR A  41       9.268   2.475  17.845  1.00  0.93           C  \\nATOM    304  O   THR A  41       6.727   4.298  18.183  1.00  0.93           O  \\nATOM    305  CG2 THR A  41      10.118   1.330  18.387  1.00  0.77           C  \\nATOM    306  OG1 THR A  41       9.609   2.699  16.472  1.00  0.79           O  \\nATOM    307  N   TYR A  42       6.403   3.249  16.213  1.00  0.94           N  \\nATOM    308  CA  TYR A  42       5.636   4.353  15.647  1.00  0.94           C  \\nATOM    309  C   TYR A  42       4.178   4.285  16.085  1.00  0.94           C  \\nATOM    310  CB  TYR A  42       5.723   4.340  14.118  1.00  0.94           C  \\nATOM    311  O   TYR A  42       3.385   5.173  15.762  1.00  0.93           O  \\nATOM    312  CG  TYR A  42       7.093   4.682  13.585  1.00  0.93           C  \\nATOM    313  CD1 TYR A  42       7.557   5.996  13.593  1.00  0.91           C  \\nATOM    314  CD2 TYR A  42       7.926   3.694  13.071  1.00  0.91           C  \\nATOM    315  CE1 TYR A  42       8.818   6.317  13.101  1.00  0.90           C  \\nATOM    316  CE2 TYR A  42       9.189   4.003  12.577  1.00  0.90           C  \\nATOM    317  OH  TYR A  42      10.875   5.627  12.109  1.00  0.88           O  \\nATOM    318  CZ  TYR A  42       9.626   5.315  12.597  1.00  0.90           C  \\nATOM    319  N   PHE A  43       3.764   3.287  16.751  1.00  0.94           N  \\nATOM    320  CA  PHE A  43       2.390   3.067  17.184  1.00  0.94           C  \\nATOM    321  C   PHE A  43       2.329   2.812  18.686  1.00  0.93           C  \\nATOM    322  CB  PHE A  43       1.768   1.889  16.427  1.00  0.93           C  \\nATOM    323  O   PHE A  43       1.762   1.812  19.129  1.00  0.91           O  \\nATOM    324  CG  PHE A  43       1.790   2.049  14.931  1.00  0.93           C  \\nATOM    325  CD1 PHE A  43       0.700   2.590  14.260  1.00  0.90           C  \\nATOM    326  CD2 PHE A  43       2.902   1.660  14.195  1.00  0.90           C  \\nATOM    327  CE1 PHE A  43       0.717   2.740  12.875  1.00  0.90           C  \\nATOM    328  CE2 PHE A  43       2.926   1.807  12.811  1.00  0.90           C  \\nATOM    329  CZ  PHE A  43       1.833   2.346  12.153  1.00  0.89           C  \\nATOM    330  N   PRO A  44       2.887   3.714  19.463  1.00  0.92           N  \\nATOM    331  CA  PRO A  44       3.074   3.479  20.897  1.00  0.92           C  \\nATOM    332  C   PRO A  44       1.753   3.316  21.646  1.00  0.91           C  \\nATOM    333  CB  PRO A  44       3.817   4.732  21.366  1.00  0.91           C  \\nATOM    334  O   PRO A  44       1.711   2.677  22.700  1.00  0.88           O  \\nATOM    335  CG  PRO A  44       3.470   5.781  20.361  1.00  0.88           C  \\nATOM    336  CD  PRO A  44       3.337   5.128  19.015  1.00  0.87           C  \\nATOM    337  N   HIS A  45       0.632   3.802  21.087  1.00  0.93           N  \\nATOM    338  CA  HIS A  45      -0.645   3.773  21.791  1.00  0.94           C  \\nATOM    339  C   HIS A  45      -1.604   2.774  21.153  1.00  0.94           C  \\nATOM    340  CB  HIS A  45      -1.276   5.167  21.812  1.00  0.91           C  \\nATOM    341  O   HIS A  45      -2.790   2.745  21.488  1.00  0.93           O  \\nATOM    342  CG  HIS A  45      -0.388   6.219  22.396  1.00  0.82           C  \\nATOM    343  CD2 HIS A  45       0.142   7.339  21.849  1.00  0.75           C  \\nATOM    344  ND1 HIS A  45       0.054   6.180  23.700  1.00  0.72           N  \\nATOM    345  CE1 HIS A  45       0.819   7.233  23.932  1.00  0.73           C  \\nATOM    346  NE2 HIS A  45       0.889   7.952  22.824  1.00  0.71           N  \\nATOM    347  N   PHE A  46      -1.096   2.003  20.265  1.00  0.93           N  \\nATOM    348  CA  PHE A  46      -1.936   1.036  19.568  1.00  0.94           C  \\nATOM    349  C   PHE A  46      -1.901  -0.317  20.269  1.00  0.93           C  \\nATOM    350  CB  PHE A  46      -1.487   0.884  18.111  1.00  0.92           C  \\nATOM    351  O   PHE A  46      -0.850  -0.744  20.752  1.00  0.92           O  \\nATOM    352  CG  PHE A  46      -2.090   1.902  17.181  1.00  0.89           C  \\nATOM    353  CD1 PHE A  46      -2.151   3.243  17.542  1.00  0.83           C  \\nATOM    354  CD2 PHE A  46      -2.594   1.519  15.945  1.00  0.83           C  \\nATOM    355  CE1 PHE A  46      -2.708   4.188  16.683  1.00  0.84           C  \\nATOM    356  CE2 PHE A  46      -3.152   2.457  15.082  1.00  0.81           C  \\nATOM    357  CZ  PHE A  46      -3.209   3.791  15.453  1.00  0.82           C  \\nATOM    358  N   ASP A  47      -3.019  -0.880  20.473  1.00  0.95           N  \\nATOM    359  CA  ASP A  47      -3.085  -2.330  20.623  1.00  0.95           C  \\nATOM    360  C   ASP A  47      -2.726  -3.035  19.317  1.00  0.95           C  \\nATOM    361  CB  ASP A  47      -4.479  -2.760  21.087  1.00  0.94           C  \\nATOM    362  O   ASP A  47      -3.495  -2.998  18.354  1.00  0.94           O  \\nATOM    363  CG  ASP A  47      -4.616  -4.264  21.239  1.00  0.89           C  \\nATOM    364  OD1 ASP A  47      -3.602  -4.985  21.115  1.00  0.83           O  \\nATOM    365  OD2 ASP A  47      -5.749  -4.734  21.481  1.00  0.86           O  \\nATOM    366  N   LEU A  48      -1.587  -3.738  19.331  1.00  0.93           N  \\nATOM    367  CA  LEU A  48      -1.062  -4.282  18.083  1.00  0.94           C  \\nATOM    368  C   LEU A  48      -1.283  -5.789  18.011  1.00  0.92           C  \\nATOM    369  CB  LEU A  48       0.429  -3.965  17.946  1.00  0.92           C  \\nATOM    370  O   LEU A  48      -0.667  -6.472  17.190  1.00  0.88           O  \\nATOM    371  CG  LEU A  48       0.813  -2.485  17.961  1.00  0.90           C  \\nATOM    372  CD1 LEU A  48       2.326  -2.328  17.860  1.00  0.85           C  \\nATOM    373  CD2 LEU A  48       0.116  -1.740  16.828  1.00  0.85           C  \\nATOM    374  N   SER A  49      -2.119  -6.347  18.814  1.00  0.93           N  \\nATOM    375  CA  SER A  49      -2.439  -7.771  18.788  1.00  0.92           C  \\nATOM    376  C   SER A  49      -3.190  -8.145  17.514  1.00  0.92           C  \\nATOM    377  CB  SER A  49      -3.270  -8.157  20.012  1.00  0.90           C  \\nATOM    378  O   SER A  49      -3.760  -7.279  16.846  1.00  0.92           O  \\nATOM    379  OG  SER A  49      -4.534  -7.517  19.984  1.00  0.81           O  \\nATOM    380  N   HIS A  50      -3.081  -9.380  17.246  1.00  0.90           N  \\nATOM    381  CA  HIS A  50      -3.778  -9.888  16.070  1.00  0.91           C  \\nATOM    382  C   HIS A  50      -5.273  -9.598  16.148  1.00  0.90           C  \\nATOM    383  CB  HIS A  50      -3.543 -11.392  15.915  1.00  0.87           C  \\nATOM    384  O   HIS A  50      -5.911  -9.873  17.168  1.00  0.87           O  \\nATOM    385  CG  HIS A  50      -4.205 -11.982  14.711  1.00  0.78           C  \\nATOM    386  CD2 HIS A  50      -5.240 -12.848  14.601  1.00  0.72           C  \\nATOM    387  ND1 HIS A  50      -3.805 -11.694  13.425  1.00  0.72           N  \\nATOM    388  CE1 HIS A  50      -4.568 -12.358  12.573  1.00  0.70           C  \\nATOM    389  NE2 HIS A  50      -5.447 -13.066  13.261  1.00  0.70           N  \\nATOM    390  N   GLY A  51      -5.782  -8.921  15.098  1.00  0.90           N  \\nATOM    391  CA  GLY A  51      -7.209  -8.656  15.009  1.00  0.90           C  \\nATOM    392  C   GLY A  51      -7.625  -7.384  15.723  1.00  0.90           C  \\nATOM    393  O   GLY A  51      -8.818  -7.127  15.900  1.00  0.89           O  \\nATOM    394  N   SER A  52      -6.672  -6.631  16.169  1.00  0.93           N  \\nATOM    395  CA  SER A  52      -6.923  -5.376  16.871  1.00  0.93           C  \\nATOM    396  C   SER A  52      -7.752  -4.422  16.018  1.00  0.94           C  \\nATOM    397  CB  SER A  52      -5.605  -4.708  17.264  1.00  0.92           C  \\nATOM    398  O   SER A  52      -7.521  -4.298  14.813  1.00  0.93           O  \\nATOM    399  OG  SER A  52      -5.784  -3.314  17.451  1.00  0.84           O  \\nATOM    400  N   ALA A  53      -8.736  -3.777  16.568  1.00  0.95           N  \\nATOM    401  CA  ALA A  53      -9.566  -2.782  15.894  1.00  0.95           C  \\nATOM    402  C   ALA A  53      -8.728  -1.596  15.424  1.00  0.95           C  \\nATOM    403  CB  ALA A  53     -10.683  -2.306  16.820  1.00  0.94           C  \\nATOM    404  O   ALA A  53      -8.997  -1.017  14.369  1.00  0.95           O  \\nATOM    405  N   GLN A  54      -7.734  -1.225  16.209  1.00  0.95           N  \\nATOM    406  CA  GLN A  54      -6.867  -0.104  15.862  1.00  0.96           C  \\nATOM    407  C   GLN A  54      -6.048  -0.409  14.611  1.00  0.96           C  \\nATOM    408  CB  GLN A  54      -5.938   0.237  17.028  1.00  0.95           C  \\nATOM    409  O   GLN A  54      -5.913   0.441  13.728  1.00  0.95           O  \\nATOM    410  CG  GLN A  54      -6.659   0.805  18.243  1.00  0.91           C  \\nATOM    411  CD  GLN A  54      -5.743   0.981  19.440  1.00  0.87           C  \\nATOM    412  NE2 GLN A  54      -5.650   2.209  19.940  1.00  0.86           N  \\nATOM    413  OE1 GLN A  54      -5.125   0.021  19.912  1.00  0.87           O  \\nATOM    414  N   VAL A  55      -5.518  -1.666  14.550  1.00  0.96           N  \\nATOM    415  CA  VAL A  55      -4.739  -2.049  13.377  1.00  0.96           C  \\nATOM    416  C   VAL A  55      -5.658  -2.171  12.164  1.00  0.96           C  \\nATOM    417  CB  VAL A  55      -3.981  -3.375  13.607  1.00  0.95           C  \\nATOM    418  O   VAL A  55      -5.316  -1.717  11.069  1.00  0.96           O  \\nATOM    419  CG1 VAL A  55      -3.321  -3.851  12.315  1.00  0.92           C  \\nATOM    420  CG2 VAL A  55      -2.939  -3.208  14.712  1.00  0.92           C  \\nATOM    421  N   LYS A  56      -6.816  -2.680  12.354  1.00  0.96           N  \\nATOM    422  CA  LYS A  56      -7.795  -2.797  11.277  1.00  0.96           C  \\nATOM    423  C   LYS A  56      -8.205  -1.424  10.754  1.00  0.96           C  \\nATOM    424  CB  LYS A  56      -9.028  -3.565  11.755  1.00  0.95           C  \\nATOM    425  O   LYS A  56      -8.253  -1.205   9.542  1.00  0.95           O  \\nATOM    426  CG  LYS A  56      -8.806  -5.063  11.902  1.00  0.87           C  \\nATOM    427  CD  LYS A  56     -10.075  -5.775  12.354  1.00  0.82           C  \\nATOM    428  CE  LYS A  56      -9.852  -7.273  12.509  1.00  0.77           C  \\nATOM    429  NZ  LYS A  56     -11.071  -7.964  13.025  1.00  0.71           N  \\nATOM    430  N   GLY A  57      -8.476  -0.553  11.663  1.00  0.96           N  \\nATOM    431  CA  GLY A  57      -8.820   0.802  11.265  1.00  0.95           C  \\nATOM    432  C   GLY A  57      -7.697   1.513  10.535  1.00  0.96           C  \\nATOM    433  O   GLY A  57      -7.933   2.197   9.536  1.00  0.95           O  \\nATOM    434  N   HIS A  58      -6.488   1.389  11.001  1.00  0.95           N  \\nATOM    435  CA  HIS A  58      -5.339   1.998  10.341  1.00  0.96           C  \\nATOM    436  C   HIS A  58      -5.075   1.353   8.985  1.00  0.96           C  \\nATOM    437  CB  HIS A  58      -4.095   1.890  11.225  1.00  0.95           C  \\nATOM    438  O   HIS A  58      -4.689   2.035   8.033  1.00  0.95           O  \\nATOM    439  CG  HIS A  58      -2.973   2.780  10.794  1.00  0.90           C  \\nATOM    440  CD2 HIS A  58      -1.790   2.495  10.199  1.00  0.83           C  \\nATOM    441  ND1 HIS A  58      -2.999   4.147  10.966  1.00  0.83           N  \\nATOM    442  CE1 HIS A  58      -1.878   4.665  10.494  1.00  0.84           C  \\nATOM    443  NE2 HIS A  58      -1.127   3.684  10.023  1.00  0.85           N  \\nATOM    444  N   GLY A  59      -5.275   0.006   8.901  1.00  0.96           N  \\nATOM    445  CA  GLY A  59      -5.169  -0.677   7.622  1.00  0.96           C  \\nATOM    446  C   GLY A  59      -6.124  -0.136   6.575  1.00  0.96           C  \\nATOM    447  O   GLY A  59      -5.752   0.022   5.410  1.00  0.96           O  \\nATOM    448  N   LYS A  60      -7.290   0.163   7.018  1.00  0.96           N  \\nATOM    449  CA  LYS A  60      -8.264   0.776   6.120  1.00  0.96           C  \\nATOM    450  C   LYS A  60      -7.790   2.148   5.648  1.00  0.96           C  \\nATOM    451  CB  LYS A  60      -9.624   0.901   6.809  1.00  0.95           C  \\nATOM    452  O   LYS A  60      -7.938   2.491   4.473  1.00  0.96           O  \\nATOM    453  CG  LYS A  60     -10.704   1.528   5.939  1.00  0.80           C  \\nATOM    454  CD  LYS A  60     -12.071   1.459   6.607  1.00  0.73           C  \\nATOM    455  CE  LYS A  60     -13.148   2.107   5.748  1.00  0.66           C  \\nATOM    456  NZ  LYS A  60     -12.861   1.952   4.290  1.00  0.61           N  \\nATOM    457  N   LYS A  61      -7.188   2.908   6.496  1.00  0.96           N  \\nATOM    458  CA  LYS A  61      -6.665   4.218   6.119  1.00  0.96           C  \\nATOM    459  C   LYS A  61      -5.542   4.088   5.094  1.00  0.96           C  \\nATOM    460  CB  LYS A  61      -6.164   4.971   7.352  1.00  0.95           C  \\nATOM    461  O   LYS A  61      -5.468   4.869   4.143  1.00  0.96           O  \\nATOM    462  CG  LYS A  61      -7.271   5.425   8.293  1.00  0.90           C  \\nATOM    463  CD  LYS A  61      -6.711   6.175   9.495  1.00  0.86           C  \\nATOM    464  CE  LYS A  61      -7.805   6.524  10.496  1.00  0.81           C  \\nATOM    465  NZ  LYS A  61      -7.245   7.145  11.733  1.00  0.76           N  \\nATOM    466  N   VAL A  62      -4.675   3.097   5.340  1.00  0.97           N  \\nATOM    467  CA  VAL A  62      -3.597   2.840   4.391  1.00  0.97           C  \\nATOM    468  C   VAL A  62      -4.185   2.459   3.034  1.00  0.97           C  \\nATOM    469  CB  VAL A  62      -2.650   1.728   4.894  1.00  0.96           C  \\nATOM    470  O   VAL A  62      -3.782   3.000   2.001  1.00  0.97           O  \\nATOM    471  CG1 VAL A  62      -1.669   1.318   3.797  1.00  0.92           C  \\nATOM    472  CG2 VAL A  62      -1.900   2.191   6.141  1.00  0.92           C  \\nATOM    473  N   ALA A  63      -5.181   1.581   3.020  1.00  0.97           N  \\nATOM    474  CA  ALA A  63      -5.821   1.133   1.786  1.00  0.97           C  \\nATOM    475  C   ALA A  63      -6.506   2.293   1.070  1.00  0.97           C  \\nATOM    476  CB  ALA A  63      -6.828   0.024   2.082  1.00  0.96           C  \\nATOM    477  O   ALA A  63      -6.430   2.407  -0.155  1.00  0.96           O  \\nATOM    478  N   ASP A  64      -7.160   3.151   1.835  1.00  0.96           N  \\nATOM    479  CA  ASP A  64      -7.818   4.318   1.257  1.00  0.96           C  \\nATOM    480  C   ASP A  64      -6.804   5.252   0.601  1.00  0.96           C  \\nATOM    481  CB  ASP A  64      -8.612   5.072   2.327  1.00  0.96           C  \\nATOM    482  O   ASP A  64      -7.052   5.783  -0.483  1.00  0.96           O  \\nATOM    483  CG  ASP A  64      -9.869   4.337   2.760  1.00  0.91           C  \\nATOM    484  OD1 ASP A  64     -10.299   3.397   2.057  1.00  0.86           O  \\nATOM    485  OD2 ASP A  64     -10.436   4.704   3.812  1.00  0.87           O  \\nATOM    486  N   ALA A  65      -5.701   5.440   1.256  1.00  0.97           N  \\nATOM    487  CA  ALA A  65      -4.641   6.284   0.711  1.00  0.97           C  \\nATOM    488  C   ALA A  65      -4.103   5.712  -0.597  1.00  0.97           C  \\nATOM    489  CB  ALA A  65      -3.511   6.441   1.726  1.00  0.96           C  \\nATOM    490  O   ALA A  65      -3.892   6.448  -1.564  1.00  0.97           O  \\nATOM    491  N   LEU A  66      -3.890   4.393  -0.631  1.00  0.97           N  \\nATOM    492  CA  LEU A  66      -3.388   3.754  -1.842  1.00  0.97           C  \\nATOM    493  C   LEU A  66      -4.426   3.812  -2.958  1.00  0.97           C  \\nATOM    494  CB  LEU A  66      -3.005   2.298  -1.561  1.00  0.97           C  \\nATOM    495  O   LEU A  66      -4.081   4.017  -4.124  1.00  0.97           O  \\nATOM    496  CG  LEU A  66      -1.782   2.079  -0.670  1.00  0.96           C  \\nATOM    497  CD1 LEU A  66      -1.620   0.598  -0.346  1.00  0.93           C  \\nATOM    498  CD2 LEU A  66      -0.526   2.626  -1.340  1.00  0.92           C  \\nATOM    499  N   THR A  67      -5.673   3.631  -2.600  1.00  0.97           N  \\nATOM    500  CA  THR A  67      -6.747   3.771  -3.578  1.00  0.97           C  \\nATOM    501  C   THR A  67      -6.742   5.168  -4.190  1.00  0.97           C  \\nATOM    502  CB  THR A  67      -8.121   3.491  -2.942  1.00  0.96           C  \\nATOM    503  O   THR A  67      -6.863   5.319  -5.408  1.00  0.96           O  \\nATOM    504  CG2 THR A  67      -9.245   3.686  -3.954  1.00  0.94           C  \\nATOM    505  OG1 THR A  67      -8.153   2.141  -2.462  1.00  0.94           O  \\nATOM    506  N   ASN A  68      -6.595   6.123  -3.329  1.00  0.97           N  \\nATOM    507  CA  ASN A  68      -6.501   7.501  -3.801  1.00  0.97           C  \\nATOM    508  C   ASN A  68      -5.282   7.705  -4.696  1.00  0.97           C  \\nATOM    509  CB  ASN A  68      -6.460   8.471  -2.618  1.00  0.96           C  \\nATOM    510  O   ASN A  68      -5.366   8.382  -5.722  1.00  0.96           O  \\nATOM    511  CG  ASN A  68      -6.597   9.919  -3.046  1.00  0.91           C  \\nATOM    512  ND2 ASN A  68      -5.770  10.787  -2.476  1.00  0.84           N  \\nATOM    513  OD1 ASN A  68      -7.441  10.255  -3.882  1.00  0.82           O  \\nATOM    514  N   ALA A  69      -4.179   7.127  -4.352  1.00  0.97           N  \\nATOM    515  CA  ALA A  69      -2.964   7.205  -5.158  1.00  0.97           C  \\nATOM    516  C   ALA A  69      -3.171   6.562  -6.526  1.00  0.97           C  \\nATOM    517  CB  ALA A  69      -1.799   6.538  -4.431  1.00  0.97           C  \\nATOM    518  O   ALA A  69      -2.761   7.116  -7.549  1.00  0.97           O  \\nATOM    519  N   VAL A  70      -3.835   5.399  -6.558  1.00  0.97           N  \\nATOM    520  CA  VAL A  70      -4.101   4.699  -7.810  1.00  0.97           C  \\nATOM    521  C   VAL A  70      -5.022   5.543  -8.688  1.00  0.97           C  \\nATOM    522  CB  VAL A  70      -4.727   3.309  -7.561  1.00  0.97           C  \\nATOM    523  O   VAL A  70      -4.806   5.655  -9.897  1.00  0.96           O  \\nATOM    524  CG1 VAL A  70      -5.225   2.699  -8.870  1.00  0.92           C  \\nATOM    525  CG2 VAL A  70      -3.718   2.382  -6.886  1.00  0.93           C  \\nATOM    526  N   ALA A  71      -6.028   6.193  -8.059  1.00  0.97           N  \\nATOM    527  CA  ALA A  71      -6.995   7.014  -8.783  1.00  0.96           C  \\nATOM    528  C   ALA A  71      -6.331   8.260  -9.363  1.00  0.96           C  \\nATOM    529  CB  ALA A  71      -8.150   7.408  -7.866  1.00  0.96           C  \\nATOM    530  O   ALA A  71      -6.775   8.789 -10.384  1.00  0.95           O  \\nATOM    531  N   HIS A  72      -5.292   8.731  -8.745  1.00  0.97           N  \\nATOM    532  CA  HIS A  72      -4.610   9.949  -9.167  1.00  0.97           C  \\nATOM    533  C   HIS A  72      -3.140   9.679  -9.473  1.00  0.96           C  \\nATOM    534  CB  HIS A  72      -4.732  11.031  -8.093  1.00  0.96           C  \\nATOM    535  O   HIS A  72      -2.270  10.474  -9.110  1.00  0.95           O  \\nATOM    536  CG  HIS A  72      -6.145  11.408  -7.780  1.00  0.92           C  \\nATOM    537  CD2 HIS A  72      -6.940  12.379  -8.289  1.00  0.83           C  \\nATOM    538  ND1 HIS A  72      -6.899  10.750  -6.832  1.00  0.81           N  \\nATOM    539  CE1 HIS A  72      -8.099  11.302  -6.772  1.00  0.82           C  \\nATOM    540  NE2 HIS A  72      -8.150  12.292  -7.647  1.00  0.84           N  \\nATOM    541  N   VAL A  73      -2.864   8.544 -10.158  1.00  0.96           N  \\nATOM    542  CA  VAL A  73      -1.518   8.016 -10.361  1.00  0.96           C  \\nATOM    543  C   VAL A  73      -0.661   9.051 -11.085  1.00  0.95           C  \\nATOM    544  CB  VAL A  73      -1.542   6.692 -11.157  1.00  0.95           C  \\nATOM    545  O   VAL A  73       0.552   9.119 -10.874  1.00  0.94           O  \\nATOM    546  CG1 VAL A  73      -2.044   6.927 -12.581  1.00  0.82           C  \\nATOM    547  CG2 VAL A  73      -0.153   6.056 -11.175  1.00  0.82           C  \\nATOM    548  N   ASP A  74      -1.252   9.948 -11.861  1.00  0.95           N  \\nATOM    549  CA  ASP A  74      -0.519  10.960 -12.617  1.00  0.95           C  \\nATOM    550  C   ASP A  74      -0.307  12.222 -11.784  1.00  0.95           C  \\nATOM    551  CB  ASP A  74      -1.258  11.302 -13.912  1.00  0.94           C  \\nATOM    552  O   ASP A  74       0.398  13.140 -12.207  1.00  0.94           O  \\nATOM    553  CG  ASP A  74      -1.211  10.182 -14.936  1.00  0.90           C  \\nATOM    554  OD1 ASP A  74      -0.120   9.621 -15.177  1.00  0.86           O  \\nATOM    555  OD2 ASP A  74      -2.274   9.860 -15.510  1.00  0.86           O  \\nATOM    556  N   ASP A  75      -0.930  12.292 -10.662  1.00  0.96           N  \\nATOM    557  CA  ASP A  75      -0.791  13.468  -9.810  1.00  0.96           C  \\nATOM    558  C   ASP A  75      -0.704  13.073  -8.338  1.00  0.96           C  \\nATOM    559  CB  ASP A  75      -1.962  14.430 -10.028  1.00  0.95           C  \\nATOM    560  O   ASP A  75      -1.386  13.656  -7.492  1.00  0.95           O  \\nATOM    561  CG  ASP A  75      -1.733  15.794  -9.400  1.00  0.90           C  \\nATOM    562  OD1 ASP A  75      -0.569  16.148  -9.117  1.00  0.83           O  \\nATOM    563  OD2 ASP A  75      -2.728  16.520  -9.184  1.00  0.86           O  \\nATOM    564  N   MET A  76       0.091  12.121  -7.998  1.00  0.96           N  \\nATOM    565  CA  MET A  76       0.188  11.578  -6.646  1.00  0.96           C  \\nATOM    566  C   MET A  76       0.702  12.633  -5.671  1.00  0.96           C  \\nATOM    567  CB  MET A  76       1.103  10.353  -6.622  1.00  0.95           C  \\nATOM    568  O   MET A  76       0.236  12.712  -4.534  1.00  0.95           O  \\nATOM    569  CG  MET A  76       0.459   9.096  -7.184  1.00  0.90           C  \\nATOM    570  SD  MET A  76       1.449   7.588  -6.847  1.00  0.91           S  \\nATOM    571  CE  MET A  76       2.791   7.812  -8.048  1.00  0.83           C  \\nATOM    572  N   PRO A  77       1.676  13.502  -6.114  1.00  0.95           N  \\nATOM    573  CA  PRO A  77       2.143  14.514  -5.163  1.00  0.95           C  \\nATOM    574  C   PRO A  77       1.009  15.381  -4.621  1.00  0.95           C  \\nATOM    575  CB  PRO A  77       3.121  15.351  -5.991  1.00  0.94           C  \\nATOM    576  O   PRO A  77       0.970  15.675  -3.423  1.00  0.94           O  \\nATOM    577  CG  PRO A  77       3.683  14.398  -6.996  1.00  0.92           C  \\nATOM    578  CD  PRO A  77       2.584  13.482  -7.452  1.00  0.90           C  \\nATOM    579  N   ASN A  78       0.143  15.741  -5.464  1.00  0.96           N  \\nATOM    580  CA  ASN A  78      -0.997  16.540  -5.028  1.00  0.96           C  \\nATOM    581  C   ASN A  78      -1.991  15.707  -4.223  1.00  0.96           C  \\nATOM    582  CB  ASN A  78      -1.694  17.183  -6.229  1.00  0.94           C  \\nATOM    583  O   ASN A  78      -2.474  16.147  -3.178  1.00  0.96           O  \\nATOM    584  CG  ASN A  78      -2.795  18.142  -5.822  1.00  0.77           C  \\nATOM    585  ND2 ASN A  78      -3.942  18.044  -6.484  1.00  0.64           N  \\nATOM    586  OD1 ASN A  78      -2.616  18.964  -4.920  1.00  0.65           O  \\nATOM    587  N   ALA A  79      -2.274  14.566  -4.662  1.00  0.96           N  \\nATOM    588  CA  ALA A  79      -3.262  13.690  -4.039  1.00  0.96           C  \\nATOM    589  C   ALA A  79      -2.823  13.277  -2.637  1.00  0.96           C  \\nATOM    590  CB  ALA A  79      -3.498  12.454  -4.904  1.00  0.95           C  \\nATOM    591  O   ALA A  79      -3.658  13.074  -1.752  1.00  0.95           O  \\nATOM    592  N   LEU A  80      -1.515  13.168  -2.408  1.00  0.96           N  \\nATOM    593  CA  LEU A  80      -0.999  12.658  -1.142  1.00  0.96           C  \\nATOM    594  C   LEU A  80      -0.334  13.770  -0.338  1.00  0.96           C  \\nATOM    595  CB  LEU A  80      -0.001  11.523  -1.388  1.00  0.96           C  \\nATOM    596  O   LEU A  80       0.399  13.499   0.616  1.00  0.95           O  \\nATOM    597  CG  LEU A  80      -0.539  10.294  -2.123  1.00  0.94           C  \\nATOM    598  CD1 LEU A  80       0.585   9.297  -2.382  1.00  0.91           C  \\nATOM    599  CD2 LEU A  80      -1.664   9.644  -1.324  1.00  0.91           C  \\nATOM    600  N   SER A  81      -0.521  15.039  -0.720  1.00  0.95           N  \\nATOM    601  CA  SER A  81       0.184  16.164  -0.115  1.00  0.95           C  \\nATOM    602  C   SER A  81      -0.058  16.222   1.390  1.00  0.95           C  \\nATOM    603  CB  SER A  81      -0.253  17.480  -0.760  1.00  0.94           C  \\nATOM    604  O   SER A  81       0.883  16.392   2.169  1.00  0.94           O  \\nATOM    605  OG  SER A  81       0.373  18.583  -0.127  1.00  0.86           O  \\nATOM    606  N   ALA A  82      -1.296  16.009   1.805  1.00  0.95           N  \\nATOM    607  CA  ALA A  82      -1.609  16.048   3.232  1.00  0.95           C  \\nATOM    608  C   ALA A  82      -0.918  14.909   3.976  1.00  0.95           C  \\nATOM    609  CB  ALA A  82      -3.119  15.983   3.447  1.00  0.94           C  \\nATOM    610  O   ALA A  82      -0.419  15.098   5.088  1.00  0.95           O  \\nATOM    611  N   LEU A  83      -0.828  13.775   3.396  1.00  0.95           N  \\nATOM    612  CA  LEU A  83      -0.173  12.636   4.030  1.00  0.95           C  \\nATOM    613  C   LEU A  83       1.339  12.833   4.072  1.00  0.95           C  \\nATOM    614  CB  LEU A  83      -0.510  11.341   3.286  1.00  0.95           C  \\nATOM    615  O   LEU A  83       1.993  12.445   5.042  1.00  0.95           O  \\nATOM    616  CG  LEU A  83      -1.931  10.805   3.468  1.00  0.92           C  \\nATOM    617  CD1 LEU A  83      -2.158   9.591   2.573  1.00  0.87           C  \\nATOM    618  CD2 LEU A  83      -2.186  10.454   4.930  1.00  0.86           C  \\nATOM    619  N   SER A  84       1.857  13.417   3.017  1.00  0.94           N  \\nATOM    620  CA  SER A  84       3.281  13.733   3.049  1.00  0.94           C  \\nATOM    621  C   SER A  84       3.617  14.655   4.216  1.00  0.94           C  \\nATOM    622  CB  SER A  84       3.716  14.382   1.734  1.00  0.93           C  \\nATOM    623  O   SER A  84       4.582  14.416   4.945  1.00  0.93           O  \\nATOM    624  OG  SER A  84       5.103  14.673   1.751  1.00  0.75           O  \\nATOM    625  N   ASP A  85       2.766  15.665   4.468  1.00  0.94           N  \\nATOM    626  CA  ASP A  85       2.953  16.582   5.588  1.00  0.94           C  \\nATOM    627  C   ASP A  85       2.849  15.847   6.923  1.00  0.94           C  \\nATOM    628  CB  ASP A  85       1.926  17.716   5.531  1.00  0.93           C  \\nATOM    629  O   ASP A  85       3.652  16.079   7.829  1.00  0.93           O  \\nATOM    630  CG  ASP A  85       2.280  18.785   4.513  1.00  0.89           C  \\nATOM    631  OD1 ASP A  85       3.482  19.071   4.321  1.00  0.85           O  \\nATOM    632  OD2 ASP A  85       1.349  19.349   3.898  1.00  0.86           O  \\nATOM    633  N   LEU A  86       1.866  15.041   7.022  1.00  0.94           N  \\nATOM    634  CA  LEU A  86       1.636  14.290   8.252  1.00  0.94           C  \\nATOM    635  C   LEU A  86       2.847  13.431   8.599  1.00  0.94           C  \\nATOM    636  CB  LEU A  86       0.392  13.409   8.117  1.00  0.93           C  \\nATOM    637  O   LEU A  86       3.327  13.458   9.735  1.00  0.93           O  \\nATOM    638  CG  LEU A  86       0.068  12.506   9.309  1.00  0.90           C  \\nATOM    639  CD1 LEU A  86      -0.255  13.348  10.539  1.00  0.85           C  \\nATOM    640  CD2 LEU A  86      -1.090  11.573   8.973  1.00  0.85           C  \\nATOM    641  N   HIS A  87       3.409  12.697   7.650  1.00  0.93           N  \\nATOM    642  CA  HIS A  87       4.510  11.776   7.910  1.00  0.93           C  \\nATOM    643  C   HIS A  87       5.814  12.530   8.147  1.00  0.92           C  \\nATOM    644  CB  HIS A  87       4.674  10.796   6.747  1.00  0.93           C  \\nATOM    645  O   HIS A  87       6.658  12.087   8.930  1.00  0.91           O  \\nATOM    646  CG  HIS A  87       3.619   9.737   6.702  1.00  0.91           C  \\nATOM    647  CD2 HIS A  87       3.554   8.520   7.293  1.00  0.87           C  \\nATOM    648  ND1 HIS A  87       2.456   9.875   5.975  1.00  0.88           N  \\nATOM    649  CE1 HIS A  87       1.720   8.787   6.123  1.00  0.88           C  \\nATOM    650  NE2 HIS A  87       2.363   7.949   6.917  1.00  0.88           N  \\nATOM    651  N   ALA A  88       5.969  13.634   7.435  1.00  0.92           N  \\nATOM    652  CA  ALA A  88       7.223  14.376   7.533  1.00  0.92           C  \\nATOM    653  C   ALA A  88       7.270  15.206   8.813  1.00  0.91           C  \\nATOM    654  CB  ALA A  88       7.407  15.274   6.313  1.00  0.91           C  \\nATOM    655  O   ALA A  88       8.299  15.254   9.491  1.00  0.90           O  \\nATOM    656  N   HIS A  89       6.182  15.754   9.203  1.00  0.91           N  \\nATOM    657  CA  HIS A  89       6.264  16.795  10.222  1.00  0.91           C  \\nATOM    658  C   HIS A  89       5.613  16.343  11.525  1.00  0.91           C  \\nATOM    659  CB  HIS A  89       5.606  18.084   9.725  1.00  0.90           C  \\nATOM    660  O   HIS A  89       6.060  16.721  12.610  1.00  0.90           O  \\nATOM    661  CG  HIS A  89       6.265  18.667   8.516  1.00  0.87           C  \\nATOM    662  CD2 HIS A  89       5.795  18.888   7.266  1.00  0.83           C  \\nATOM    663  ND1 HIS A  89       7.574  19.098   8.519  1.00  0.82           N  \\nATOM    664  CE1 HIS A  89       7.881  19.561   7.318  1.00  0.82           C  \\nATOM    665  NE2 HIS A  89       6.819  19.444   6.540  1.00  0.80           N  \\nATOM    666  N   LYS A  90       4.573  15.561  11.436  1.00  0.92           N  \\nATOM    667  CA  LYS A  90       3.879  15.169  12.659  1.00  0.91           C  \\nATOM    668  C   LYS A  90       4.341  13.796  13.138  1.00  0.92           C  \\nATOM    669  CB  LYS A  90       2.365  15.164  12.439  1.00  0.90           C  \\nATOM    670  O   LYS A  90       4.723  13.634  14.299  1.00  0.91           O  \\nATOM    671  CG  LYS A  90       1.766  16.547  12.225  1.00  0.81           C  \\nATOM    672  CD  LYS A  90       0.244  16.504  12.234  1.00  0.75           C  \\nATOM    673  CE  LYS A  90      -0.357  17.881  11.983  1.00  0.65           C  \\nATOM    674  NZ  LYS A  90      -1.850  17.844  11.994  1.00  0.56           N  \\nATOM    675  N   LEU A  91       4.357  12.856  12.265  1.00  0.92           N  \\nATOM    676  CA  LEU A  91       4.699  11.490  12.643  1.00  0.92           C  \\nATOM    677  C   LEU A  91       6.207  11.271  12.583  1.00  0.91           C  \\nATOM    678  CB  LEU A  91       3.989  10.487  11.730  1.00  0.91           C  \\nATOM    679  O   LEU A  91       6.751  10.458  13.334  1.00  0.90           O  \\nATOM    680  CG  LEU A  91       2.460  10.528  11.737  1.00  0.89           C  \\nATOM    681  CD1 LEU A  91       1.899   9.530  10.730  1.00  0.86           C  \\nATOM    682  CD2 LEU A  91       1.924  10.242  13.136  1.00  0.86           C  \\nATOM    683  N   ARG A  92       6.891  11.994  11.615  1.00  0.91           N  \\nATOM    684  CA  ARG A  92       8.340  11.942  11.445  1.00  0.91           C  \\nATOM    685  C   ARG A  92       8.821  10.503  11.295  1.00  0.91           C  \\nATOM    686  CB  ARG A  92       9.044  12.609  12.628  1.00  0.89           C  \\nATOM    687  O   ARG A  92       9.755  10.080  11.980  1.00  0.89           O  \\nATOM    688  CG  ARG A  92       8.703  14.081  12.798  1.00  0.84           C  \\nATOM    689  CD  ARG A  92       9.510  14.721  13.919  1.00  0.81           C  \\nATOM    690  NE  ARG A  92       9.105  16.105  14.152  1.00  0.75           N  \\nATOM    691  NH1 ARG A  92      10.521  16.434  15.947  1.00  0.57           N  \\nATOM    692  NH2 ARG A  92       9.156  18.131  15.232  1.00  0.52           N  \\nATOM    693  CZ  ARG A  92       9.595  16.887  15.110  1.00  0.73           C  \\nATOM    694  N   VAL A  93       8.192   9.778  10.462  1.00  0.92           N  \\nATOM    695  CA  VAL A  93       8.518   8.385  10.177  1.00  0.92           C  \\nATOM    696  C   VAL A  93       9.804   8.312   9.357  1.00  0.92           C  \\nATOM    697  CB  VAL A  93       7.368   7.675   9.427  1.00  0.91           C  \\nATOM    698  O   VAL A  93       9.940   8.998   8.341  1.00  0.91           O  \\nATOM    699  CG1 VAL A  93       7.761   6.243   9.068  1.00  0.86           C  \\nATOM    700  CG2 VAL A  93       6.095   7.687  10.271  1.00  0.85           C  \\nATOM    701  N   ASP A  94      10.764   7.507   9.844  1.00  0.92           N  \\nATOM    702  CA  ASP A  94      11.971   7.280   9.055  1.00  0.92           C  \\nATOM    703  C   ASP A  94      11.630   6.701   7.684  1.00  0.93           C  \\nATOM    704  CB  ASP A  94      12.930   6.347   9.797  1.00  0.91           C  \\nATOM    705  O   ASP A  94      10.916   5.701   7.588  1.00  0.92           O  \\nATOM    706  CG  ASP A  94      14.299   6.263   9.145  1.00  0.86           C  \\nATOM    707  OD1 ASP A  94      14.391   6.367   7.903  1.00  0.82           O  \\nATOM    708  OD2 ASP A  94      15.295   6.089   9.880  1.00  0.84           O  \\nATOM    709  N   PRO A  95      12.186   7.238   6.605  1.00  0.91           N  \\nATOM    710  CA  PRO A  95      11.840   6.833   5.241  1.00  0.91           C  \\nATOM    711  C   PRO A  95      12.108   5.352   4.979  1.00  0.91           C  \\nATOM    712  CB  PRO A  95      12.742   7.713   4.372  1.00  0.90           C  \\nATOM    713  O   PRO A  95      11.439   4.738   4.144  1.00  0.91           O  \\nATOM    714  CG  PRO A  95      12.894   8.984   5.144  1.00  0.88           C  \\nATOM    715  CD  PRO A  95      12.954   8.649   6.607  1.00  0.87           C  \\nATOM    716  N   VAL A  96      13.045   4.754   5.652  1.00  0.92           N  \\nATOM    717  CA  VAL A  96      13.426   3.365   5.415  1.00  0.92           C  \\nATOM    718  C   VAL A  96      12.255   2.444   5.747  1.00  0.92           C  \\nATOM    719  CB  VAL A  96      14.669   2.969   6.244  1.00  0.89           C  \\nATOM    720  O   VAL A  96      12.139   1.350   5.188  1.00  0.91           O  \\nATOM    721  CG1 VAL A  96      14.322   2.882   7.729  1.00  0.74           C  \\nATOM    722  CG2 VAL A  96      15.239   1.642   5.747  1.00  0.74           C  \\nATOM    723  N   ASN A  97      11.415   2.838   6.604  1.00  0.94           N  \\nATOM    724  CA  ASN A  97      10.312   1.998   7.059  1.00  0.94           C  \\nATOM    725  C   ASN A  97       9.233   1.862   5.988  1.00  0.94           C  \\nATOM    726  CB  ASN A  97       9.711   2.555   8.351  1.00  0.93           C  \\nATOM    727  O   ASN A  97       8.460   0.902   5.998  1.00  0.94           O  \\nATOM    728  CG  ASN A  97      10.613   2.349   9.551  1.00  0.88           C  \\nATOM    729  ND2 ASN A  97      11.263   3.418   9.996  1.00  0.84           N  \\nATOM    730  OD1 ASN A  97      10.726   1.237  10.074  1.00  0.84           O  \\nATOM    731  N   PHE A  98       9.159   2.835   5.094  1.00  0.95           N  \\nATOM    732  CA  PHE A  98       8.192   2.723   4.008  1.00  0.95           C  \\nATOM    733  C   PHE A  98       8.507   1.522   3.125  1.00  0.95           C  \\nATOM    734  CB  PHE A  98       8.175   4.003   3.167  1.00  0.94           C  \\nATOM    735  O   PHE A  98       7.597   0.874   2.602  1.00  0.94           O  \\nATOM    736  CG  PHE A  98       7.532   5.175   3.858  1.00  0.92           C  \\nATOM    737  CD1 PHE A  98       6.153   5.343   3.832  1.00  0.88           C  \\nATOM    738  CD2 PHE A  98       8.307   6.109   4.533  1.00  0.88           C  \\nATOM    739  CE1 PHE A  98       5.554   6.427   4.470  1.00  0.88           C  \\nATOM    740  CE2 PHE A  98       7.716   7.195   5.174  1.00  0.87           C  \\nATOM    741  CZ  PHE A  98       6.340   7.352   5.140  1.00  0.86           C  \\nATOM    742  N   LYS A  99       9.772   1.202   3.002  1.00  0.95           N  \\nATOM    743  CA  LYS A  99      10.157   0.030   2.221  1.00  0.95           C  \\nATOM    744  C   LYS A  99       9.680  -1.257   2.889  1.00  0.96           C  \\nATOM    745  CB  LYS A  99      11.674  -0.011   2.028  1.00  0.93           C  \\nATOM    746  O   LYS A  99       9.287  -2.206   2.208  1.00  0.95           O  \\nATOM    747  CG  LYS A  99      12.212   1.084   1.119  1.00  0.77           C  \\nATOM    748  CD  LYS A  99      13.716   0.953   0.916  1.00  0.71           C  \\nATOM    749  CE  LYS A  99      14.259   2.061   0.023  1.00  0.63           C  \\nATOM    750  NZ  LYS A  99      15.745   1.983  -0.114  1.00  0.57           N  \\nATOM    751  N   LEU A 100       9.738  -1.265   4.220  1.00  0.96           N  \\nATOM    752  CA  LEU A 100       9.306  -2.444   4.962  1.00  0.96           C  \\nATOM    753  C   LEU A 100       7.814  -2.692   4.768  1.00  0.96           C  \\nATOM    754  CB  LEU A 100       9.618  -2.283   6.453  1.00  0.96           C  \\nATOM    755  O   LEU A 100       7.397  -3.823   4.506  1.00  0.96           O  \\nATOM    756  CG  LEU A 100      11.090  -2.089   6.820  1.00  0.94           C  \\nATOM    757  CD1 LEU A 100      11.244  -1.937   8.329  1.00  0.91           C  \\nATOM    758  CD2 LEU A 100      11.928  -3.255   6.306  1.00  0.90           C  \\nATOM    759  N   LEU A 101       6.991  -1.653   4.853  1.00  0.97           N  \\nATOM    760  CA  LEU A 101       5.558  -1.802   4.628  1.00  0.97           C  \\nATOM    761  C   LEU A 101       5.271  -2.169   3.175  1.00  0.97           C  \\nATOM    762  CB  LEU A 101       4.820  -0.513   4.996  1.00  0.96           C  \\nATOM    763  O   LEU A 101       4.405  -3.002   2.900  1.00  0.97           O  \\nATOM    764  CG  LEU A 101       3.304  -0.515   4.787  1.00  0.93           C  \\nATOM    765  CD1 LEU A 101       2.664  -1.662   5.561  1.00  0.87           C  \\nATOM    766  CD2 LEU A 101       2.704   0.822   5.207  1.00  0.86           C  \\nATOM    767  N   SER A 102       5.964  -1.564   2.232  1.00  0.97           N  \\nATOM    768  CA  SER A 102       5.804  -1.892   0.819  1.00  0.97           C  \\nATOM    769  C   SER A 102       6.021  -3.381   0.570  1.00  0.97           C  \\nATOM    770  CB  SER A 102       6.776  -1.077  -0.034  1.00  0.96           C  \\nATOM    771  O   SER A 102       5.250  -4.013  -0.155  1.00  0.97           O  \\nATOM    772  OG  SER A 102       6.426   0.296  -0.024  1.00  0.83           O  \\nATOM    773  N   HIS A 103       7.066  -3.848   1.198  1.00  0.97           N  \\nATOM    774  CA  HIS A 103       7.349  -5.273   1.070  1.00  0.97           C  \\nATOM    775  C   HIS A 103       6.184  -6.114   1.583  1.00  0.97           C  \\nATOM    776  CB  HIS A 103       8.630  -5.635   1.825  1.00  0.96           C  \\nATOM    777  O   HIS A 103       5.757  -7.061   0.919  1.00  0.97           O  \\nATOM    778  CG  HIS A 103       8.953  -7.095   1.791  1.00  0.92           C  \\nATOM    779  CD2 HIS A 103       8.857  -8.049   2.746  1.00  0.83           C  \\nATOM    780  ND1 HIS A 103       9.446  -7.722   0.667  1.00  0.83           N  \\nATOM    781  CE1 HIS A 103       9.638  -9.003   0.934  1.00  0.83           C  \\nATOM    782  NE2 HIS A 103       9.288  -9.228   2.189  1.00  0.85           N  \\nATOM    783  N   CYS A 104       5.640  -5.781   2.698  1.00  0.97           N  \\nATOM    784  CA  CYS A 104       4.558  -6.553   3.298  1.00  0.97           C  \\nATOM    785  C   CYS A 104       3.273  -6.409   2.493  1.00  0.97           C  \\nATOM    786  CB  CYS A 104       4.320  -6.110   4.741  1.00  0.97           C  \\nATOM    787  O   CYS A 104       2.489  -7.354   2.393  1.00  0.97           O  \\nATOM    788  SG  CYS A 104       5.656  -6.557   5.871  1.00  0.95           S  \\nATOM    789  N   LEU A 105       3.042  -5.250   1.901  1.00  0.97           N  \\nATOM    790  CA  LEU A 105       1.913  -5.066   0.995  1.00  0.97           C  \\nATOM    791  C   LEU A 105       2.033  -5.986  -0.216  1.00  0.97           C  \\nATOM    792  CB  LEU A 105       1.826  -3.608   0.535  1.00  0.97           C  \\nATOM    793  O   LEU A 105       1.057  -6.630  -0.610  1.00  0.97           O  \\nATOM    794  CG  LEU A 105       1.249  -2.613   1.543  1.00  0.94           C  \\nATOM    795  CD1 LEU A 105       1.566  -1.183   1.118  1.00  0.87           C  \\nATOM    796  CD2 LEU A 105      -0.256  -2.812   1.690  1.00  0.86           C  \\nATOM    797  N   LEU A 106       3.199  -6.059  -0.798  1.00  0.97           N  \\nATOM    798  CA  LEU A 106       3.408  -6.918  -1.958  1.00  0.97           C  \\nATOM    799  C   LEU A 106       3.178  -8.382  -1.599  1.00  0.97           C  \\nATOM    800  CB  LEU A 106       4.822  -6.733  -2.515  1.00  0.97           C  \\nATOM    801  O   LEU A 106       2.559  -9.124  -2.365  1.00  0.97           O  \\nATOM    802  CG  LEU A 106       5.114  -5.393  -3.193  1.00  0.96           C  \\nATOM    803  CD1 LEU A 106       6.602  -5.268  -3.502  1.00  0.93           C  \\nATOM    804  CD2 LEU A 106       4.284  -5.244  -4.463  1.00  0.92           C  \\nATOM    805  N   VAL A 107       3.675  -8.806  -0.435  1.00  0.97           N  \\nATOM    806  CA  VAL A 107       3.480 -10.178   0.022  1.00  0.97           C  \\nATOM    807  C   VAL A 107       1.989 -10.457   0.199  1.00  0.97           C  \\nATOM    808  CB  VAL A 107       4.234 -10.448   1.344  1.00  0.96           C  \\nATOM    809  O   VAL A 107       1.496 -11.511  -0.210  1.00  0.96           O  \\nATOM    810  CG1 VAL A 107       3.842 -11.806   1.922  1.00  0.89           C  \\nATOM    811  CG2 VAL A 107       5.743 -10.373   1.122  1.00  0.89           C  \\nATOM    812  N   THR A 108       1.285  -9.521   0.774  1.00  0.97           N  \\nATOM    813  CA  THR A 108      -0.149  -9.681   0.991  1.00  0.97           C  \\nATOM    814  C   THR A 108      -0.888  -9.789  -0.340  1.00  0.96           C  \\nATOM    815  CB  THR A 108      -0.725  -8.508   1.807  1.00  0.96           C  \\nATOM    816  O   THR A 108      -1.757 -10.648  -0.505  1.00  0.96           O  \\nATOM    817  CG2 THR A 108      -2.235  -8.643   1.973  1.00  0.93           C  \\nATOM    818  OG1 THR A 108      -0.112  -8.489   3.102  1.00  0.93           O  \\nATOM    819  N   LEU A 109      -0.573  -8.919  -1.283  1.00  0.97           N  \\nATOM    820  CA  LEU A 109      -1.225  -8.956  -2.588  1.00  0.97           C  \\nATOM    821  C   LEU A 109      -0.914 -10.260  -3.314  1.00  0.97           C  \\nATOM    822  CB  LEU A 109      -0.783  -7.765  -3.442  1.00  0.96           C  \\nATOM    823  O   LEU A 109      -1.798 -10.856  -3.935  1.00  0.96           O  \\nATOM    824  CG  LEU A 109      -1.270  -6.387  -2.988  1.00  0.94           C  \\nATOM    825  CD1 LEU A 109      -0.632  -5.293  -3.837  1.00  0.90           C  \\nATOM    826  CD2 LEU A 109      -2.791  -6.308  -3.059  1.00  0.89           C  \\nATOM    827  N   ALA A 110       0.313 -10.708  -3.180  1.00  0.97           N  \\nATOM    828  CA  ALA A 110       0.703 -11.966  -3.812  1.00  0.97           C  \\nATOM    829  C   ALA A 110      -0.087 -13.138  -3.234  1.00  0.96           C  \\nATOM    830  CB  ALA A 110       2.201 -12.205  -3.644  1.00  0.96           C  \\nATOM    831  O   ALA A 110      -0.476 -14.052  -3.965  1.00  0.96           O  \\nATOM    832  N   ALA A 111      -0.305 -13.134  -1.962  1.00  0.96           N  \\nATOM    833  CA  ALA A 111      -1.012 -14.209  -1.272  1.00  0.96           C  \\nATOM    834  C   ALA A 111      -2.483 -14.246  -1.675  1.00  0.96           C  \\nATOM    835  CB  ALA A 111      -0.882 -14.047   0.241  1.00  0.96           C  \\nATOM    836  O   ALA A 111      -3.079 -15.321  -1.778  1.00  0.95           O  \\nATOM    837  N   HIS A 112      -3.104 -13.076  -1.971  1.00  0.96           N  \\nATOM    838  CA  HIS A 112      -4.547 -13.021  -2.177  1.00  0.96           C  \\nATOM    839  C   HIS A 112      -4.889 -12.960  -3.662  1.00  0.96           C  \\nATOM    840  CB  HIS A 112      -5.147 -11.817  -1.450  1.00  0.96           C  \\nATOM    841  O   HIS A 112      -6.034 -13.202  -4.050  1.00  0.94           O  \\nATOM    842  CG  HIS A 112      -5.249 -12.000   0.031  1.00  0.95           C  \\nATOM    843  CD2 HIS A 112      -4.488 -11.516   1.041  1.00  0.92           C  \\nATOM    844  ND1 HIS A 112      -6.229 -12.769   0.620  1.00  0.92           N  \\nATOM    845  CE1 HIS A 112      -6.068 -12.747   1.933  1.00  0.91           C  \\nATOM    846  NE2 HIS A 112      -5.018 -11.994   2.214  1.00  0.92           N  \\nATOM    847  N   LEU A 113      -3.935 -12.591  -4.479  1.00  0.96           N  \\nATOM    848  CA  LEU A 113      -4.163 -12.478  -5.916  1.00  0.96           C  \\nATOM    849  C   LEU A 113      -3.168 -13.333  -6.693  1.00  0.95           C  \\nATOM    850  CB  LEU A 113      -4.057 -11.017  -6.362  1.00  0.94           C  \\nATOM    851  O   LEU A 113      -2.469 -12.831  -7.576  1.00  0.94           O  \\nATOM    852  CG  LEU A 113      -4.960 -10.019  -5.635  1.00  0.89           C  \\nATOM    853  CD1 LEU A 113      -4.652  -8.597  -6.091  1.00  0.81           C  \\nATOM    854  CD2 LEU A 113      -6.428 -10.356  -5.871  1.00  0.80           C  \\nATOM    855  N   PRO A 114      -3.087 -14.615  -6.375  1.00  0.95           N  \\nATOM    856  CA  PRO A 114      -2.041 -15.449  -6.973  1.00  0.95           C  \\nATOM    857  C   PRO A 114      -2.122 -15.494  -8.497  1.00  0.94           C  \\nATOM    858  CB  PRO A 114      -2.306 -16.832  -6.371  1.00  0.93           C  \\nATOM    859  O   PRO A 114      -1.091 -15.560  -9.171  1.00  0.92           O  \\nATOM    860  CG  PRO A 114      -3.739 -16.795  -5.947  1.00  0.91           C  \\nATOM    861  CD  PRO A 114      -4.067 -15.403  -5.490  1.00  0.91           C  \\nATOM    862  N   ALA A 115      -3.324 -15.443  -9.102  1.00  0.95           N  \\nATOM    863  CA  ALA A 115      -3.485 -15.517 -10.552  1.00  0.95           C  \\nATOM    864  C   ALA A 115      -3.121 -14.190 -11.213  1.00  0.94           C  \\nATOM    865  CB  ALA A 115      -4.916 -15.910 -10.909  1.00  0.93           C  \\nATOM    866  O   ALA A 115      -2.602 -14.168 -12.331  1.00  0.93           O  \\nATOM    867  N   GLU A 116      -3.324 -13.126 -10.589  1.00  0.95           N  \\nATOM    868  CA  GLU A 116      -3.128 -11.796 -11.157  1.00  0.94           C  \\nATOM    869  C   GLU A 116      -1.714 -11.286 -10.893  1.00  0.95           C  \\nATOM    870  CB  GLU A 116      -4.156 -10.813 -10.592  1.00  0.92           C  \\nATOM    871  O   GLU A 116      -1.235 -10.382 -11.581  1.00  0.94           O  \\nATOM    872  CG  GLU A 116      -5.588 -11.113 -11.009  1.00  0.80           C  \\nATOM    873  CD  GLU A 116      -6.229 -12.224 -10.192  1.00  0.75           C  \\nATOM    874  OE1 GLU A 116      -5.710 -12.554  -9.102  1.00  0.74           O  \\nATOM    875  OE2 GLU A 116      -7.260 -12.770 -10.647  1.00  0.72           O  \\nATOM    876  N   PHE A 117      -1.070 -11.860  -9.920  1.00  0.96           N  \\nATOM    877  CA  PHE A 117       0.205 -11.320  -9.461  1.00  0.96           C  \\nATOM    878  C   PHE A 117       1.362 -11.912 -10.258  1.00  0.96           C  \\nATOM    879  CB  PHE A 117       0.401 -11.596  -7.967  1.00  0.95           C  \\nATOM    880  O   PHE A 117       2.272 -12.516  -9.686  1.00  0.94           O  \\nATOM    881  CG  PHE A 117       1.238 -10.563  -7.264  1.00  0.92           C  \\nATOM    882  CD1 PHE A 117       2.594 -10.776  -7.047  1.00  0.87           C  \\nATOM    883  CD2 PHE A 117       0.669  -9.377  -6.818  1.00  0.86           C  \\nATOM    884  CE1 PHE A 117       3.372  -9.821  -6.396  1.00  0.86           C  \\nATOM    885  CE2 PHE A 117       1.440  -8.418  -6.167  1.00  0.86           C  \\nATOM    886  CZ  PHE A 117       2.790  -8.642  -5.956  1.00  0.85           C  \\nATOM    887  N   THR A 118       1.432 -11.619 -11.543  1.00  0.96           N  \\nATOM    888  CA  THR A 118       2.512 -12.027 -12.435  1.00  0.96           C  \\nATOM    889  C   THR A 118       3.778 -11.223 -12.155  1.00  0.97           C  \\nATOM    890  CB  THR A 118       2.110 -11.859 -13.912  1.00  0.96           C  \\nATOM    891  O   THR A 118       3.732 -10.205 -11.461  1.00  0.96           O  \\nATOM    892  CG2 THR A 118       0.776 -12.540 -14.199  1.00  0.92           C  \\nATOM    893  OG1 THR A 118       1.996 -10.463 -14.215  1.00  0.92           O  \\nATOM    894  N   PRO A 119       4.853 -11.622 -12.654  1.00  0.96           N  \\nATOM    895  CA  PRO A 119       6.099 -10.873 -12.472  1.00  0.96           C  \\nATOM    896  C   PRO A 119       5.998  -9.431 -12.963  1.00  0.96           C  \\nATOM    897  CB  PRO A 119       7.110 -11.664 -13.306  1.00  0.96           C  \\nATOM    898  O   PRO A 119       6.541  -8.521 -12.331  1.00  0.96           O  \\nATOM    899  CG  PRO A 119       6.638 -13.081 -13.236  1.00  0.93           C  \\nATOM    900  CD  PRO A 119       5.136 -13.085 -13.273  1.00  0.91           C  \\nATOM    901  N   ALA A 120       5.302  -9.215 -14.116  1.00  0.96           N  \\nATOM    902  CA  ALA A 120       5.124  -7.861 -14.633  1.00  0.96           C  \\nATOM    903  C   ALA A 120       4.297  -7.011 -13.672  1.00  0.97           C  \\nATOM    904  CB  ALA A 120       4.461  -7.900 -16.008  1.00  0.96           C  \\nATOM    905  O   ALA A 120       4.623  -5.847 -13.426  1.00  0.97           O  \\nATOM    906  N   VAL A 121       3.238  -7.599 -13.131  1.00  0.97           N  \\nATOM    907  CA  VAL A 121       2.391  -6.885 -12.181  1.00  0.97           C  \\nATOM    908  C   VAL A 121       3.172  -6.607 -10.898  1.00  0.97           C  \\nATOM    909  CB  VAL A 121       1.105  -7.680 -11.860  1.00  0.96           C  \\nATOM    910  O   VAL A 121       3.096  -5.509 -10.344  1.00  0.97           O  \\nATOM    911  CG1 VAL A 121       0.370  -7.060 -10.672  1.00  0.94           C  \\nATOM    912  CG2 VAL A 121       0.194  -7.737 -13.085  1.00  0.94           C  \\nATOM    913  N   HIS A 122       3.921  -7.599 -10.441  1.00  0.97           N  \\nATOM    914  CA  HIS A 122       4.789  -7.396  -9.287  1.00  0.97           C  \\nATOM    915  C   HIS A 122       5.711  -6.199  -9.495  1.00  0.97           C  \\nATOM    916  CB  HIS A 122       5.616  -8.653  -9.011  1.00  0.96           C  \\nATOM    917  O   HIS A 122       5.822  -5.336  -8.621  1.00  0.96           O  \\nATOM    918  CG  HIS A 122       6.414  -8.580  -7.748  1.00  0.93           C  \\nATOM    919  CD2 HIS A 122       6.530  -7.597  -6.825  1.00  0.88           C  \\nATOM    920  ND1 HIS A 122       7.222  -9.611  -7.318  1.00  0.87           N  \\nATOM    921  CE1 HIS A 122       7.801  -9.264  -6.181  1.00  0.87           C  \\nATOM    922  NE2 HIS A 122       7.397  -8.046  -5.860  1.00  0.89           N  \\nATOM    923  N   ALA A 123       6.325  -6.152 -10.608  1.00  0.97           N  \\nATOM    924  CA  ALA A 123       7.263  -5.073 -10.909  1.00  0.97           C  \\nATOM    925  C   ALA A 123       6.554  -3.722 -10.935  1.00  0.97           C  \\nATOM    926  CB  ALA A 123       7.960  -5.330 -12.242  1.00  0.96           C  \\nATOM    927  O   ALA A 123       7.062  -2.736 -10.396  1.00  0.96           O  \\nATOM    928  N   SER A 124       5.403  -3.686 -11.556  1.00  0.97           N  \\nATOM    929  CA  SER A 124       4.625  -2.454 -11.637  1.00  0.97           C  \\nATOM    930  C   SER A 124       4.188  -1.984 -10.254  1.00  0.97           C  \\nATOM    931  CB  SER A 124       3.398  -2.653 -12.528  1.00  0.97           C  \\nATOM    932  O   SER A 124       4.278  -0.795  -9.940  1.00  0.97           O  \\nATOM    933  OG  SER A 124       3.785  -2.917 -13.866  1.00  0.92           O  \\nATOM    934  N   LEU A 125       3.732  -2.907  -9.418  1.00  0.97           N  \\nATOM    935  CA  LEU A 125       3.305  -2.562  -8.066  1.00  0.97           C  \\nATOM    936  C   LEU A 125       4.488  -2.096  -7.224  1.00  0.97           C  \\nATOM    937  CB  LEU A 125       2.626  -3.759  -7.396  1.00  0.97           C  \\nATOM    938  O   LEU A 125       4.350  -1.188  -6.402  1.00  0.97           O  \\nATOM    939  CG  LEU A 125       1.239  -4.133  -7.920  1.00  0.95           C  \\nATOM    940  CD1 LEU A 125       0.814  -5.491  -7.371  1.00  0.92           C  \\nATOM    941  CD2 LEU A 125       0.221  -3.059  -7.551  1.00  0.92           C  \\nATOM    942  N   ASP A 126       5.592  -2.739  -7.401  1.00  0.97           N  \\nATOM    943  CA  ASP A 126       6.801  -2.316  -6.701  1.00  0.96           C  \\nATOM    944  C   ASP A 126       7.169  -0.878  -7.058  1.00  0.96           C  \\nATOM    945  CB  ASP A 126       7.966  -3.253  -7.028  1.00  0.96           C  \\nATOM    946  O   ASP A 126       7.447  -0.064  -6.174  1.00  0.96           O  \\nATOM    947  CG  ASP A 126       9.161  -3.055  -6.112  1.00  0.88           C  \\nATOM    948  OD1 ASP A 126       8.974  -2.943  -4.881  1.00  0.78           O  \\nATOM    949  OD2 ASP A 126      10.300  -3.009  -6.626  1.00  0.85           O  \\nATOM    950  N   LYS A 127       7.135  -0.518  -8.337  1.00  0.97           N  \\nATOM    951  CA  LYS A 127       7.427   0.843  -8.778  1.00  0.97           C  \\nATOM    952  C   LYS A 127       6.400   1.831  -8.231  1.00  0.97           C  \\nATOM    953  CB  LYS A 127       7.463   0.916 -10.305  1.00  0.96           C  \\nATOM    954  O   LYS A 127       6.755   2.935  -7.814  1.00  0.96           O  \\nATOM    955  CG  LYS A 127       8.677   0.243 -10.929  1.00  0.90           C  \\nATOM    956  CD  LYS A 127       8.628   0.300 -12.450  1.00  0.84           C  \\nATOM    957  CE  LYS A 127       9.744  -0.523 -13.078  1.00  0.77           C  \\nATOM    958  NZ  LYS A 127       9.650  -0.538 -14.569  1.00  0.69           N  \\nATOM    959  N   PHE A 128       5.164   1.430  -8.249  1.00  0.97           N  \\nATOM    960  CA  PHE A 128       4.085   2.270  -7.743  1.00  0.97           C  \\nATOM    961  C   PHE A 128       4.289   2.583  -6.265  1.00  0.97           C  \\nATOM    962  CB  PHE A 128       2.728   1.589  -7.953  1.00  0.97           C  \\nATOM    963  O   PHE A 128       4.257   3.747  -5.862  1.00  0.97           O  \\nATOM    964  CG  PHE A 128       1.599   2.244  -7.204  1.00  0.97           C  \\nATOM    965  CD1 PHE A 128       1.058   3.446  -7.645  1.00  0.96           C  \\nATOM    966  CD2 PHE A 128       1.077   1.656  -6.059  1.00  0.95           C  \\nATOM    967  CE1 PHE A 128       0.012   4.053  -6.954  1.00  0.95           C  \\nATOM    968  CE2 PHE A 128       0.032   2.258  -5.363  1.00  0.95           C  \\nATOM    969  CZ  PHE A 128      -0.498   3.457  -5.812  1.00  0.94           C  \\nATOM    970  N   LEU A 129       4.572   1.531  -5.471  1.00  0.97           N  \\nATOM    971  CA  LEU A 129       4.757   1.732  -4.038  1.00  0.97           C  \\nATOM    972  C   LEU A 129       6.014   2.549  -3.762  1.00  0.97           C  \\nATOM    973  CB  LEU A 129       4.839   0.385  -3.314  1.00  0.97           C  \\nATOM    974  O   LEU A 129       6.048   3.346  -2.822  1.00  0.97           O  \\nATOM    975  CG  LEU A 129       3.556  -0.447  -3.282  1.00  0.96           C  \\nATOM    976  CD1 LEU A 129       3.832  -1.823  -2.685  1.00  0.95           C  \\nATOM    977  CD2 LEU A 129       2.469   0.276  -2.494  1.00  0.95           C  \\nATOM    978  N   ALA A 130       7.037   2.377  -4.575  1.00  0.96           N  \\nATOM    979  CA  ALA A 130       8.232   3.210  -4.462  1.00  0.96           C  \\nATOM    980  C   ALA A 130       7.908   4.674  -4.742  1.00  0.96           C  \\nATOM    981  CB  ALA A 130       9.317   2.716  -5.417  1.00  0.95           C  \\nATOM    982  O   ALA A 130       8.403   5.569  -4.053  1.00  0.96           O  \\nATOM    983  N   SER A 131       7.071   4.927  -5.735  1.00  0.96           N  \\nATOM    984  CA  SER A 131       6.656   6.289  -6.057  1.00  0.96           C  \\nATOM    985  C   SER A 131       5.840   6.901  -4.923  1.00  0.96           C  \\nATOM    986  CB  SER A 131       5.840   6.309  -7.350  1.00  0.95           C  \\nATOM    987  O   SER A 131       6.035   8.066  -4.571  1.00  0.96           O  \\nATOM    988  OG  SER A 131       6.632   5.893  -8.450  1.00  0.81           O  \\nATOM    989  N   VAL A 132       4.947   6.118  -4.305  1.00  0.96           N  \\nATOM    990  CA  VAL A 132       4.173   6.563  -3.150  1.00  0.96           C  \\nATOM    991  C   VAL A 132       5.116   6.930  -2.006  1.00  0.96           C  \\nATOM    992  CB  VAL A 132       3.171   5.482  -2.688  1.00  0.96           C  \\nATOM    993  O   VAL A 132       4.969   7.987  -1.387  1.00  0.96           O  \\nATOM    994  CG1 VAL A 132       2.579   5.840  -1.326  1.00  0.93           C  \\nATOM    995  CG2 VAL A 132       2.063   5.304  -3.725  1.00  0.93           C  \\nATOM    996  N   SER A 133       6.101   6.108  -1.756  1.00  0.95           N  \\nATOM    997  CA  SER A 133       7.069   6.364  -0.695  1.00  0.95           C  \\nATOM    998  C   SER A 133       7.829   7.663  -0.941  1.00  0.95           C  \\nATOM    999  CB  SER A 133       8.056   5.201  -0.577  1.00  0.94           C  \\nATOM   1000  O   SER A 133       8.049   8.446  -0.015  1.00  0.94           O  \\nATOM   1001  OG  SER A 133       7.405   4.037  -0.098  1.00  0.78           O  \\nATOM   1002  N   THR A 134       8.211   7.833  -2.191  1.00  0.94           N  \\nATOM   1003  CA  THR A 134       8.931   9.044  -2.567  1.00  0.94           C  \\nATOM   1004  C   THR A 134       8.084  10.284  -2.294  1.00  0.94           C  \\nATOM   1005  CB  THR A 134       9.338   9.014  -4.052  1.00  0.93           C  \\nATOM   1006  O   THR A 134       8.576  11.270  -1.741  1.00  0.94           O  \\nATOM   1007  CG2 THR A 134      10.003  10.323  -4.467  1.00  0.88           C  \\nATOM   1008  OG1 THR A 134      10.256   7.935  -4.268  1.00  0.89           O  \\nATOM   1009  N   VAL A 135       6.835  10.225  -2.612  1.00  0.95           N  \\nATOM   1010  CA  VAL A 135       5.943  11.359  -2.394  1.00  0.95           C  \\nATOM   1011  C   VAL A 135       5.762  11.594  -0.896  1.00  0.95           C  \\nATOM   1012  CB  VAL A 135       4.570  11.139  -3.068  1.00  0.94           C  \\nATOM   1013  O   VAL A 135       5.837  12.732  -0.426  1.00  0.94           O  \\nATOM   1014  CG1 VAL A 135       3.569  12.199  -2.612  1.00  0.92           C  \\nATOM   1015  CG2 VAL A 135       4.714  11.154  -4.588  1.00  0.91           C  \\nATOM   1016  N   LEU A 136       5.595  10.544  -0.138  1.00  0.95           N  \\nATOM   1017  CA  LEU A 136       5.316  10.650   1.290  1.00  0.95           C  \\nATOM   1018  C   LEU A 136       6.527  11.192   2.042  1.00  0.94           C  \\nATOM   1019  CB  LEU A 136       4.914   9.288   1.861  1.00  0.94           C  \\nATOM   1020  O   LEU A 136       6.380  11.835   3.084  1.00  0.93           O  \\nATOM   1021  CG  LEU A 136       3.544   8.754   1.439  1.00  0.93           C  \\nATOM   1022  CD1 LEU A 136       3.278   7.402   2.093  1.00  0.90           C  \\nATOM   1023  CD2 LEU A 136       2.447   9.751   1.796  1.00  0.90           C  \\nATOM   1024  N   THR A 137       7.716  11.025   1.471  1.00  0.94           N  \\nATOM   1025  CA  THR A 137       8.921  11.447   2.177  1.00  0.93           C  \\nATOM   1026  C   THR A 137       9.515  12.697   1.534  1.00  0.93           C  \\nATOM   1027  CB  THR A 137       9.978  10.327   2.198  1.00  0.92           C  \\nATOM   1028  O   THR A 137      10.617  13.120   1.888  1.00  0.91           O  \\nATOM   1029  CG2 THR A 137       9.463   9.099   2.943  1.00  0.85           C  \\nATOM   1030  OG1 THR A 137      10.298   9.955   0.851  1.00  0.88           O  \\nATOM   1031  N   SER A 138       8.899  13.226   0.660  1.00  0.93           N  \\nATOM   1032  CA  SER A 138       9.431  14.330  -0.133  1.00  0.93           C  \\nATOM   1033  C   SER A 138       9.574  15.594   0.707  1.00  0.92           C  \\nATOM   1034  CB  SER A 138       8.532  14.607  -1.338  1.00  0.92           C  \\nATOM   1035  O   SER A 138      10.308  16.513   0.336  1.00  0.90           O  \\nATOM   1036  OG  SER A 138       7.225  14.961  -0.919  1.00  0.88           O  \\nATOM   1037  N   LYS A 139       8.882  15.632   1.759  1.00  0.92           N  \\nATOM   1038  CA  LYS A 139       8.876  16.872   2.530  1.00  0.92           C  \\nATOM   1039  C   LYS A 139       9.664  16.716   3.828  1.00  0.91           C  \\nATOM   1040  CB  LYS A 139       7.442  17.306   2.836  1.00  0.91           C  \\nATOM   1041  O   LYS A 139       9.591  17.573   4.711  1.00  0.88           O  \\nATOM   1042  CG  LYS A 139       6.635  17.688   1.605  1.00  0.90           C  \\nATOM   1043  CD  LYS A 139       5.257  18.219   1.981  1.00  0.88           C  \\nATOM   1044  CE  LYS A 139       4.462  18.635   0.752  1.00  0.85           C  \\nATOM   1045  NZ  LYS A 139       3.076  19.063   1.107  1.00  0.81           N  \\nATOM   1046  N   TYR A 140      10.462  15.638   3.953  1.00  0.89           N  \\nATOM   1047  CA  TYR A 140      11.336  15.456   5.106  1.00  0.88           C  \\nATOM   1048  C   TYR A 140      12.450  16.496   5.115  1.00  0.84           C  \\nATOM   1049  CB  TYR A 140      11.936  14.047   5.109  1.00  0.86           C  \\nATOM   1050  O   TYR A 140      12.917  16.923   4.057  1.00  0.79           O  \\nATOM   1051  CG  TYR A 140      10.959  12.972   5.519  1.00  0.84           C  \\nATOM   1052  CD1 TYR A 140      11.119  12.277   6.716  1.00  0.80           C  \\nATOM   1053  CD2 TYR A 140       9.876  12.647   4.710  1.00  0.80           C  \\nATOM   1054  CE1 TYR A 140      10.223  11.284   7.096  1.00  0.80           C  \\nATOM   1055  CE2 TYR A 140       8.973  11.656   5.080  1.00  0.79           C  \\nATOM   1056  OH  TYR A 140       8.264   9.999   6.645  1.00  0.71           O  \\nATOM   1057  CZ  TYR A 140       9.154  10.981   6.273  1.00  0.79           C  \\nATOM   1058  N   ARG A 141      12.557  17.156   6.259  1.00  0.78           N  \\nATOM   1059  CA  ARG A 141      13.618  18.145   6.419  1.00  0.77           C  \\nATOM   1060  C   ARG A 141      14.957  17.471   6.695  1.00  0.75           C  \\nATOM   1061  CB  ARG A 141      13.279  19.119   7.549  1.00  0.72           C  \\nATOM   1062  O   ARG A 141      15.008  16.411   7.322  1.00  0.69           O  \\nATOM   1063  CG  ARG A 141      12.110  20.041   7.239  1.00  0.65           C  \\nATOM   1064  CD  ARG A 141      11.922  21.097   8.319  1.00  0.64           C  \\nATOM   1065  NE  ARG A 141      10.799  21.980   8.019  1.00  0.53           N  \\nATOM   1066  NH1 ARG A 141      11.030  23.262   9.927  1.00  0.42           N  \\nATOM   1067  NH2 ARG A 141       9.365  23.729   8.422  1.00  0.38           N  \\nATOM   1068  CZ  ARG A 141      10.401  22.989   8.790  1.00  0.57           C  \",\"pdb\");\n\tviewer_1744327365365795.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"green\"}});\n\tviewer_1744327365365795.zoomTo();\nviewer_1744327365365795.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1744327365365795\"  style=\"position: relative; width: 640px; height: 480px;\">\n",
              "        <p id=\"3dmolwarning_1744327365365795\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_1744327365365795 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1744327365365795\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1744327365365795 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1744327365365795\"),{backgroundColor:\"white\"});\n",
              "viewer_1744327365365795.zoomTo();\n",
              "\tviewer_1744327365365795.addModel(\"HEADER                                            18-OCT-22                     \\nTITLE     ESMFOLD V1 PREDICTION FOR INPUT\\nREMARK   1                                                                      \\nREMARK   1 REFERENCE 1                                                          \\nREMARK   1  AUTH   ZEMING LIN, HALIL AKIN, ROSHAN RAO, BRIAN HIE, ZHONGKAI ZHU,\\nREMARK   1  AUTH 2 WENTING LU, NIKITA SMETANIN, ROBERT VERKUIL, ORI KABELI,\\nREMARK   1  AUTH 3 YANIV SHMUELI, ALLAN DOS SANTOS COSTA,\\nREMARK   1  AUTH 4 MARYAM FAZEL-ZARANDI, TOM SERCU, SALVATORE CANDIDO,\\nREMARK   1  AUTH 5 ALEXANDER RIVES\\nREMARK   1  TITL   EVOLUTIONARY-SCALE PREDICTION OF ATOMIC LEVEL PROTEIN\\nREMARK   1  TITL 2 STRUCTURE WITH A LANGUAGE MODEL\\nREMARK   1  REF                                                                 \\nREMARK   1  REFN                                                                \\nREMARK   1  PMID                                                                \\nREMARK   1  DOI    10.1101/2022.07.20.500902                                    \\nREMARK   1                                                                      \\nREMARK   1 LICENSE AND DISCLAIMERS                \\nREMARK   1 ESM METAGENOMIC ATLAS DATA IS AVAILABLE UNDER\\nREMARK   1 A CC-BY-4.0 LICENSE FOR ACADEMIC AND COMMERCIAL USE.\\nREMARK   1 COPYRIGHT (C) META PLATFORMS, INC. ALL RIGHTS RESERVED.\\nREMARK   1 USE OF THE ESM METAGENOMIC ATLAS DATA IS SUBJECT\\nREMARK   1 TO THE META OPEN SOURCE TERMS OF USE AND PRIVACY POLICY.\\nATOM      1  N   VAL A   1       9.646   6.431 -12.355  1.00  0.83           N  \\nATOM      2  CA  VAL A   1       8.465   6.885 -13.083  1.00  0.85           C  \\nATOM      3  C   VAL A   1       7.678   5.680 -13.595  1.00  0.90           C  \\nATOM      4  CB  VAL A   1       8.845   7.812 -14.259  1.00  0.79           C  \\nATOM      5  O   VAL A   1       8.263   4.713 -14.088  1.00  0.89           O  \\nATOM      6  CG1 VAL A   1       7.602   8.222 -15.046  1.00  0.65           C  \\nATOM      7  CG2 VAL A   1       9.589   9.044 -13.747  1.00  0.65           C  \\nATOM      8  N   LEU A   2       6.407   5.746 -13.541  1.00  0.95           N  \\nATOM      9  CA  LEU A   2       5.512   4.684 -13.986  1.00  0.95           C  \\nATOM     10  C   LEU A   2       5.221   4.807 -15.477  1.00  0.94           C  \\nATOM     11  CB  LEU A   2       4.202   4.720 -13.193  1.00  0.93           C  \\nATOM     12  O   LEU A   2       4.777   5.859 -15.943  1.00  0.93           O  \\nATOM     13  CG  LEU A   2       4.310   4.439 -11.694  1.00  0.88           C  \\nATOM     14  CD1 LEU A   2       2.998   4.784 -10.995  1.00  0.81           C  \\nATOM     15  CD2 LEU A   2       4.686   2.982 -11.447  1.00  0.81           C  \\nATOM     16  N   SER A   3       5.533   3.858 -16.212  1.00  0.96           N  \\nATOM     17  CA  SER A   3       5.147   3.805 -17.619  1.00  0.96           C  \\nATOM     18  C   SER A   3       3.643   3.604 -17.773  1.00  0.96           C  \\nATOM     19  CB  SER A   3       5.896   2.682 -18.338  1.00  0.95           C  \\nATOM     20  O   SER A   3       2.957   3.257 -16.809  1.00  0.96           O  \\nATOM     21  OG  SER A   3       5.422   1.413 -17.921  1.00  0.86           O  \\nATOM     22  N   PRO A   4       3.134   3.858 -18.947  1.00  0.96           N  \\nATOM     23  CA  PRO A   4       1.715   3.585 -19.183  1.00  0.96           C  \\nATOM     24  C   PRO A   4       1.333   2.139 -18.872  1.00  0.96           C  \\nATOM     25  CB  PRO A   4       1.540   3.887 -20.673  1.00  0.95           C  \\nATOM     26  O   PRO A   4       0.262   1.886 -18.314  1.00  0.96           O  \\nATOM     27  CG  PRO A   4       2.577   4.919 -20.978  1.00  0.92           C  \\nATOM     28  CD  PRO A   4       3.808   4.609 -20.175  1.00  0.90           C  \\nATOM     29  N   ALA A   5       2.245   1.221 -19.202  1.00  0.96           N  \\nATOM     30  CA  ALA A   5       1.996  -0.186 -18.898  1.00  0.96           C  \\nATOM     31  C   ALA A   5       1.939  -0.419 -17.391  1.00  0.97           C  \\nATOM     32  CB  ALA A   5       3.072  -1.066 -19.529  1.00  0.96           C  \\nATOM     33  O   ALA A   5       1.097  -1.178 -16.905  1.00  0.97           O  \\nATOM     34  N   ASP A   6       2.814   0.196 -16.642  1.00  0.97           N  \\nATOM     35  CA  ASP A   6       2.795   0.095 -15.186  1.00  0.97           C  \\nATOM     36  C   ASP A   6       1.472   0.605 -14.617  1.00  0.97           C  \\nATOM     37  CB  ASP A   6       3.963   0.874 -14.579  1.00  0.97           C  \\nATOM     38  O   ASP A   6       0.876  -0.036 -13.749  1.00  0.97           O  \\nATOM     39  CG  ASP A   6       5.314   0.250 -14.884  1.00  0.95           C  \\nATOM     40  OD1 ASP A   6       5.405  -0.993 -14.978  1.00  0.94           O  \\nATOM     41  OD2 ASP A   6       6.297   1.009 -15.029  1.00  0.93           O  \\nATOM     42  N   LYS A   7       1.050   1.711 -15.118  1.00  0.97           N  \\nATOM     43  CA  LYS A   7      -0.194   2.296 -14.626  1.00  0.97           C  \\nATOM     44  C   LYS A   7      -1.379   1.373 -14.894  1.00  0.97           C  \\nATOM     45  CB  LYS A   7      -0.437   3.662 -15.271  1.00  0.97           C  \\nATOM     46  O   LYS A   7      -2.240   1.193 -14.030  1.00  0.97           O  \\nATOM     47  CG  LYS A   7       0.608   4.709 -14.916  1.00  0.94           C  \\nATOM     48  CD  LYS A   7       0.359   6.019 -15.652  1.00  0.92           C  \\nATOM     49  CE  LYS A   7       1.456   7.037 -15.370  1.00  0.89           C  \\nATOM     50  NZ  LYS A   7       1.208   8.329 -16.077  1.00  0.85           N  \\nATOM     51  N   THR A   8      -1.411   0.813 -16.028  1.00  0.97           N  \\nATOM     52  CA  THR A   8      -2.460  -0.131 -16.394  1.00  0.97           C  \\nATOM     53  C   THR A   8      -2.438  -1.348 -15.474  1.00  0.97           C  \\nATOM     54  CB  THR A   8      -2.316  -0.588 -17.858  1.00  0.96           C  \\nATOM     55  O   THR A   8      -3.481  -1.773 -14.972  1.00  0.96           O  \\nATOM     56  CG2 THR A   8      -3.359  -1.643 -18.210  1.00  0.90           C  \\nATOM     57  OG1 THR A   8      -2.483   0.541 -18.724  1.00  0.90           O  \\nATOM     58  N   ASN A   9      -1.287  -1.887 -15.265  1.00  0.97           N  \\nATOM     59  CA  ASN A   9      -1.134  -3.041 -14.386  1.00  0.97           C  \\nATOM     60  C   ASN A   9      -1.584  -2.724 -12.962  1.00  0.97           C  \\nATOM     61  CB  ASN A   9       0.316  -3.530 -14.388  1.00  0.96           C  \\nATOM     62  O   ASN A   9      -2.280  -3.524 -12.334  1.00  0.97           O  \\nATOM     63  CG  ASN A   9       0.669  -4.305 -15.642  1.00  0.93           C  \\nATOM     64  ND2 ASN A   9       1.960  -4.395 -15.940  1.00  0.89           N  \\nATOM     65  OD1 ASN A   9      -0.211  -4.818 -16.338  1.00  0.88           O  \\nATOM     66  N   VAL A  10      -1.174  -1.563 -12.462  1.00  0.97           N  \\nATOM     67  CA  VAL A  10      -1.507  -1.159 -11.100  1.00  0.97           C  \\nATOM     68  C   VAL A  10      -3.019  -0.995 -10.965  1.00  0.97           C  \\nATOM     69  CB  VAL A  10      -0.791   0.152 -10.705  1.00  0.97           C  \\nATOM     70  O   VAL A  10      -3.623  -1.501 -10.016  1.00  0.97           O  \\nATOM     71  CG1 VAL A  10      -1.389   0.729  -9.423  1.00  0.94           C  \\nATOM     72  CG2 VAL A  10       0.708  -0.089 -10.538  1.00  0.94           C  \\nATOM     73  N   LYS A  11      -3.631  -0.324 -11.906  1.00  0.97           N  \\nATOM     74  CA  LYS A  11      -5.075  -0.113 -11.853  1.00  0.97           C  \\nATOM     75  C   LYS A  11      -5.828  -1.437 -11.945  1.00  0.97           C  \\nATOM     76  CB  LYS A  11      -5.522   0.824 -12.976  1.00  0.96           C  \\nATOM     77  O   LYS A  11      -6.808  -1.651 -11.229  1.00  0.96           O  \\nATOM     78  CG  LYS A  11      -5.118   2.276 -12.768  1.00  0.92           C  \\nATOM     79  CD  LYS A  11      -5.546   3.149 -13.942  1.00  0.89           C  \\nATOM     80  CE  LYS A  11      -5.094   4.591 -13.761  1.00  0.83           C  \\nATOM     81  NZ  LYS A  11      -5.461   5.438 -14.935  1.00  0.79           N  \\nATOM     82  N   ALA A  12      -5.348  -2.276 -12.806  1.00  0.97           N  \\nATOM     83  CA  ALA A  12      -5.997  -3.574 -12.971  1.00  0.97           C  \\nATOM     84  C   ALA A  12      -5.892  -4.406 -11.696  1.00  0.97           C  \\nATOM     85  CB  ALA A  12      -5.384  -4.330 -14.148  1.00  0.96           C  \\nATOM     86  O   ALA A  12      -6.889  -4.956 -11.223  1.00  0.96           O  \\nATOM     87  N   ALA A  13      -4.705  -4.494 -11.137  1.00  0.97           N  \\nATOM     88  CA  ALA A  13      -4.499  -5.267  -9.915  1.00  0.97           C  \\nATOM     89  C   ALA A  13      -5.304  -4.686  -8.756  1.00  0.97           C  \\nATOM     90  CB  ALA A  13      -3.015  -5.313  -9.559  1.00  0.96           C  \\nATOM     91  O   ALA A  13      -5.935  -5.427  -7.998  1.00  0.96           O  \\nATOM     92  N   TRP A  14      -5.271  -3.390  -8.654  1.00  0.97           N  \\nATOM     93  CA  TRP A  14      -5.993  -2.749  -7.560  1.00  0.97           C  \\nATOM     94  C   TRP A  14      -7.498  -2.949  -7.711  1.00  0.96           C  \\nATOM     95  CB  TRP A  14      -5.667  -1.255  -7.503  1.00  0.96           C  \\nATOM     96  O   TRP A  14      -8.219  -3.054  -6.716  1.00  0.95           O  \\nATOM     97  CG  TRP A  14      -6.024  -0.603  -6.201  1.00  0.89           C  \\nATOM     98  CD1 TRP A  14      -7.078   0.233  -5.960  1.00  0.79           C  \\nATOM     99  CD2 TRP A  14      -5.322  -0.731  -4.961  1.00  0.80           C  \\nATOM    100  CE2 TRP A  14      -6.007   0.056  -4.009  1.00  0.79           C  \\nATOM    101  CE3 TRP A  14      -4.178  -1.436  -4.562  1.00  0.74           C  \\nATOM    102  NE1 TRP A  14      -7.074   0.633  -4.643  1.00  0.84           N  \\nATOM    103  CH2 TRP A  14      -4.464  -0.543  -2.317  1.00  0.81           C  \\nATOM    104  CZ2 TRP A  14      -5.586   0.157  -2.681  1.00  0.83           C  \\nATOM    105  CZ3 TRP A  14      -3.760  -1.334  -3.240  1.00  0.80           C  \\nATOM    106  N   GLY A  15      -7.971  -2.975  -8.950  1.00  0.96           N  \\nATOM    107  CA  GLY A  15      -9.366  -3.298  -9.207  1.00  0.96           C  \\nATOM    108  C   GLY A  15      -9.775  -4.652  -8.660  1.00  0.95           C  \\nATOM    109  O   GLY A  15     -10.908  -4.827  -8.205  1.00  0.94           O  \\nATOM    110  N   LYS A  16      -8.793  -5.598  -8.617  1.00  0.96           N  \\nATOM    111  CA  LYS A  16      -9.058  -6.939  -8.103  1.00  0.96           C  \\nATOM    112  C   LYS A  16      -9.075  -6.950  -6.577  1.00  0.95           C  \\nATOM    113  CB  LYS A  16      -8.015  -7.929  -8.623  1.00  0.94           C  \\nATOM    114  O   LYS A  16      -9.733  -7.794  -5.964  1.00  0.95           O  \\nATOM    115  CG  LYS A  16      -8.019  -8.096 -10.135  1.00  0.87           C  \\nATOM    116  CD  LYS A  16      -9.309  -8.743 -10.623  1.00  0.83           C  \\nATOM    117  CE  LYS A  16      -9.282  -8.981 -12.127  1.00  0.77           C  \\nATOM    118  NZ  LYS A  16     -10.562  -9.573 -12.617  1.00  0.74           N  \\nATOM    119  N   VAL A  17      -8.330  -6.052  -5.973  1.00  0.95           N  \\nATOM    120  CA  VAL A  17      -8.340  -5.913  -4.521  1.00  0.95           C  \\nATOM    121  C   VAL A  17      -9.734  -5.502  -4.051  1.00  0.93           C  \\nATOM    122  CB  VAL A  17      -7.291  -4.885  -4.041  1.00  0.93           C  \\nATOM    123  O   VAL A  17     -10.261  -6.061  -3.087  1.00  0.91           O  \\nATOM    124  CG1 VAL A  17      -7.479  -4.574  -2.557  1.00  0.88           C  \\nATOM    125  CG2 VAL A  17      -5.879  -5.402  -4.307  1.00  0.88           C  \\nATOM    126  N   GLY A  18     -10.306  -4.545  -4.797  1.00  0.93           N  \\nATOM    127  CA  GLY A  18     -11.685  -4.146  -4.567  1.00  0.93           C  \\nATOM    128  C   GLY A  18     -11.990  -3.866  -3.107  1.00  0.93           C  \\nATOM    129  O   GLY A  18     -11.256  -3.128  -2.445  1.00  0.89           O  \\nATOM    130  N   ALA A  19     -13.072  -4.451  -2.666  1.00  0.92           N  \\nATOM    131  CA  ALA A  19     -13.642  -4.215  -1.342  1.00  0.91           C  \\nATOM    132  C   ALA A  19     -12.757  -4.807  -0.249  1.00  0.92           C  \\nATOM    133  CB  ALA A  19     -15.050  -4.800  -1.256  1.00  0.88           C  \\nATOM    134  O   ALA A  19     -12.992  -4.580   0.940  1.00  0.90           O  \\nATOM    135  N   HIS A  20     -11.810  -5.596  -0.593  1.00  0.95           N  \\nATOM    136  CA  HIS A  20     -10.943  -6.252   0.379  1.00  0.95           C  \\nATOM    137  C   HIS A  20      -9.755  -5.368   0.743  1.00  0.95           C  \\nATOM    138  CB  HIS A  20     -10.451  -7.595  -0.164  1.00  0.93           C  \\nATOM    139  O   HIS A  20      -8.888  -5.776   1.519  1.00  0.95           O  \\nATOM    140  CG  HIS A  20     -11.552  -8.567  -0.449  1.00  0.87           C  \\nATOM    141  CD2 HIS A  20     -12.054  -9.022  -1.621  1.00  0.79           C  \\nATOM    142  ND1 HIS A  20     -12.274  -9.186   0.547  1.00  0.81           N  \\nATOM    143  CE1 HIS A  20     -13.175  -9.984  -0.001  1.00  0.78           C  \\nATOM    144  NE2 HIS A  20     -13.063  -9.903  -1.316  1.00  0.80           N  \\nATOM    145  N   ALA A  21      -9.733  -4.179   0.206  1.00  0.95           N  \\nATOM    146  CA  ALA A  21      -8.559  -3.325   0.366  1.00  0.95           C  \\nATOM    147  C   ALA A  21      -8.251  -3.088   1.842  1.00  0.96           C  \\nATOM    148  CB  ALA A  21      -8.768  -1.993  -0.351  1.00  0.94           C  \\nATOM    149  O   ALA A  21      -7.095  -3.170   2.262  1.00  0.95           O  \\nATOM    150  N   GLY A  22      -9.287  -2.789   2.643  1.00  0.95           N  \\nATOM    151  CA  GLY A  22      -9.083  -2.554   4.064  1.00  0.95           C  \\nATOM    152  C   GLY A  22      -8.518  -3.759   4.793  1.00  0.95           C  \\nATOM    153  O   GLY A  22      -7.618  -3.622   5.624  1.00  0.95           O  \\nATOM    154  N   GLU A  23      -9.010  -4.925   4.464  1.00  0.96           N  \\nATOM    155  CA  GLU A  23      -8.541  -6.173   5.060  1.00  0.96           C  \\nATOM    156  C   GLU A  23      -7.091  -6.457   4.676  1.00  0.96           C  \\nATOM    157  CB  GLU A  23      -9.436  -7.340   4.636  1.00  0.93           C  \\nATOM    158  O   GLU A  23      -6.295  -6.888   5.513  1.00  0.95           O  \\nATOM    159  CG  GLU A  23      -9.026  -8.678   5.237  1.00  0.76           C  \\nATOM    160  CD  GLU A  23      -9.837  -9.848   4.704  1.00  0.70           C  \\nATOM    161  OE1 GLU A  23     -10.789  -9.621   3.922  1.00  0.66           O  \\nATOM    162  OE2 GLU A  23      -9.519 -11.001   5.071  1.00  0.64           O  \\nATOM    163  N   TYR A  24      -6.771  -6.225   3.410  1.00  0.96           N  \\nATOM    164  CA  TYR A  24      -5.407  -6.442   2.941  1.00  0.96           C  \\nATOM    165  C   TYR A  24      -4.447  -5.444   3.577  1.00  0.96           C  \\nATOM    166  CB  TYR A  24      -5.340  -6.331   1.414  1.00  0.96           C  \\nATOM    167  O   TYR A  24      -3.313  -5.792   3.915  1.00  0.96           O  \\nATOM    168  CG  TYR A  24      -6.081  -7.430   0.693  1.00  0.94           C  \\nATOM    169  CD1 TYR A  24      -6.660  -8.484   1.396  1.00  0.91           C  \\nATOM    170  CD2 TYR A  24      -6.203  -7.418  -0.692  1.00  0.91           C  \\nATOM    171  CE1 TYR A  24      -7.342  -9.501   0.735  1.00  0.90           C  \\nATOM    172  CE2 TYR A  24      -6.883  -8.429  -1.364  1.00  0.90           C  \\nATOM    173  OH  TYR A  24      -8.123 -10.467  -1.302  1.00  0.86           O  \\nATOM    174  CZ  TYR A  24      -7.449  -9.464  -0.642  1.00  0.89           C  \\nATOM    175  N   GLY A  25      -4.934  -4.213   3.746  1.00  0.96           N  \\nATOM    176  CA  GLY A  25      -4.145  -3.230   4.472  1.00  0.96           C  \\nATOM    177  C   GLY A  25      -3.849  -3.641   5.901  1.00  0.97           C  \\nATOM    178  O   GLY A  25      -2.713  -3.518   6.365  1.00  0.96           O  \\nATOM    179  N   ALA A  26      -4.842  -4.125   6.589  1.00  0.96           N  \\nATOM    180  CA  ALA A  26      -4.669  -4.571   7.969  1.00  0.96           C  \\nATOM    181  C   ALA A  26      -3.692  -5.741   8.049  1.00  0.96           C  \\nATOM    182  CB  ALA A  26      -6.014  -4.963   8.575  1.00  0.96           C  \\nATOM    183  O   ALA A  26      -2.831  -5.780   8.931  1.00  0.96           O  \\nATOM    184  N   GLU A  27      -3.808  -6.672   7.135  1.00  0.96           N  \\nATOM    185  CA  GLU A  27      -2.908  -7.821   7.118  1.00  0.96           C  \\nATOM    186  C   GLU A  27      -1.469  -7.393   6.845  1.00  0.96           C  \\nATOM    187  CB  GLU A  27      -3.360  -8.842   6.070  1.00  0.96           C  \\nATOM    188  O   GLU A  27      -0.535  -7.905   7.465  1.00  0.96           O  \\nATOM    189  CG  GLU A  27      -2.507 -10.101   6.030  1.00  0.93           C  \\nATOM    190  CD  GLU A  27      -3.001 -11.129   5.025  1.00  0.90           C  \\nATOM    191  OE1 GLU A  27      -4.206 -11.117   4.686  1.00  0.88           O  \\nATOM    192  OE2 GLU A  27      -2.176 -11.954   4.572  1.00  0.88           O  \\nATOM    193  N   ALA A  28      -1.278  -6.482   5.928  1.00  0.97           N  \\nATOM    194  CA  ALA A  28       0.063  -5.984   5.633  1.00  0.97           C  \\nATOM    195  C   ALA A  28       0.702  -5.363   6.872  1.00  0.97           C  \\nATOM    196  CB  ALA A  28       0.014  -4.966   4.496  1.00  0.96           C  \\nATOM    197  O   ALA A  28       1.887  -5.578   7.141  1.00  0.96           O  \\nATOM    198  N   LEU A  29      -0.067  -4.567   7.648  1.00  0.97           N  \\nATOM    199  CA  LEU A  29       0.445  -3.977   8.881  1.00  0.97           C  \\nATOM    200  C   LEU A  29       0.800  -5.060   9.894  1.00  0.96           C  \\nATOM    201  CB  LEU A  29      -0.584  -3.017   9.482  1.00  0.96           C  \\nATOM    202  O   LEU A  29       1.838  -4.980  10.555  1.00  0.96           O  \\nATOM    203  CG  LEU A  29      -0.788  -1.692   8.744  1.00  0.94           C  \\nATOM    204  CD1 LEU A  29      -2.007  -0.961   9.296  1.00  0.90           C  \\nATOM    205  CD2 LEU A  29       0.459  -0.822   8.854  1.00  0.90           C  \\nATOM    206  N   GLU A  30      -0.045  -6.019  10.005  1.00  0.96           N  \\nATOM    207  CA  GLU A  30       0.246  -7.104  10.936  1.00  0.96           C  \\nATOM    208  C   GLU A  30       1.520  -7.845  10.540  1.00  0.96           C  \\nATOM    209  CB  GLU A  30      -0.930  -8.082  11.008  1.00  0.96           C  \\nATOM    210  O   GLU A  30       2.349  -8.166  11.394  1.00  0.96           O  \\nATOM    211  CG  GLU A  30      -2.146  -7.529  11.737  1.00  0.92           C  \\nATOM    212  CD  GLU A  30      -3.287  -8.528  11.844  1.00  0.89           C  \\nATOM    213  OE1 GLU A  30      -3.140  -9.672  11.356  1.00  0.87           O  \\nATOM    214  OE2 GLU A  30      -4.336  -8.164  12.420  1.00  0.87           O  \\nATOM    215  N   ARG A  31       1.720  -8.158   9.264  1.00  0.96           N  \\nATOM    216  CA  ARG A  31       2.954  -8.771   8.781  1.00  0.96           C  \\nATOM    217  C   ARG A  31       4.162  -7.901   9.110  1.00  0.96           C  \\nATOM    218  CB  ARG A  31       2.879  -9.015   7.272  1.00  0.96           C  \\nATOM    219  O   ARG A  31       5.213  -8.412   9.503  1.00  0.96           O  \\nATOM    220  CG  ARG A  31       1.935 -10.139   6.876  1.00  0.93           C  \\nATOM    221  CD  ARG A  31       1.893 -10.334   5.367  1.00  0.91           C  \\nATOM    222  NE  ARG A  31       0.905 -11.338   4.981  1.00  0.90           N  \\nATOM    223  NH1 ARG A  31       2.286 -13.148   5.370  1.00  0.83           N  \\nATOM    224  NH2 ARG A  31       0.145 -13.475   4.620  1.00  0.82           N  \\nATOM    225  CZ  ARG A  31       1.114 -12.651   4.991  1.00  0.87           C  \\nATOM    226  N   MET A  32       3.987  -6.637   8.929  1.00  0.96           N  \\nATOM    227  CA  MET A  32       5.097  -5.726   9.191  1.00  0.96           C  \\nATOM    228  C   MET A  32       5.483  -5.747  10.666  1.00  0.96           C  \\nATOM    229  CB  MET A  32       4.736  -4.301   8.765  1.00  0.96           C  \\nATOM    230  O   MET A  32       6.667  -5.821  11.001  1.00  0.95           O  \\nATOM    231  CG  MET A  32       5.866  -3.301   8.949  1.00  0.94           C  \\nATOM    232  SD  MET A  32       5.343  -1.578   8.598  1.00  0.94           S  \\nATOM    233  CE  MET A  32       4.260  -1.275  10.022  1.00  0.88           C  \\nATOM    234  N   PHE A  33       4.507  -5.705  11.540  1.00  0.96           N  \\nATOM    235  CA  PHE A  33       4.784  -5.706  12.972  1.00  0.96           C  \\nATOM    236  C   PHE A  33       5.475  -6.999  13.388  1.00  0.95           C  \\nATOM    237  CB  PHE A  33       3.490  -5.522  13.771  1.00  0.95           C  \\nATOM    238  O   PHE A  33       6.334  -6.992  14.273  1.00  0.94           O  \\nATOM    239  CG  PHE A  33       2.852  -4.171  13.591  1.00  0.95           C  \\nATOM    240  CD1 PHE A  33       3.632  -3.023  13.516  1.00  0.93           C  \\nATOM    241  CD2 PHE A  33       1.472  -4.049  13.495  1.00  0.92           C  \\nATOM    242  CE1 PHE A  33       3.044  -1.771  13.349  1.00  0.92           C  \\nATOM    243  CE2 PHE A  33       0.878  -2.801  13.329  1.00  0.92           C  \\nATOM    244  CZ  PHE A  33       1.665  -1.664  13.257  1.00  0.91           C  \\nATOM    245  N   LEU A  34       5.068  -8.096  12.787  1.00  0.96           N  \\nATOM    246  CA  LEU A  34       5.637  -9.398  13.117  1.00  0.95           C  \\nATOM    247  C   LEU A  34       7.032  -9.549  12.521  1.00  0.95           C  \\nATOM    248  CB  LEU A  34       4.730 -10.523  12.612  1.00  0.94           C  \\nATOM    249  O   LEU A  34       7.944 -10.050  13.183  1.00  0.95           O  \\nATOM    250  CG  LEU A  34       3.410 -10.718  13.360  1.00  0.86           C  \\nATOM    251  CD1 LEU A  34       2.531 -11.728  12.631  1.00  0.77           C  \\nATOM    252  CD2 LEU A  34       3.669 -11.165  14.795  1.00  0.76           C  \\nATOM    253  N   SER A  35       7.230  -9.071  11.302  1.00  0.96           N  \\nATOM    254  CA  SER A  35       8.483  -9.287  10.586  1.00  0.96           C  \\nATOM    255  C   SER A  35       9.517  -8.226  10.948  1.00  0.96           C  \\nATOM    256  CB  SER A  35       8.245  -9.283   9.075  1.00  0.95           C  \\nATOM    257  O   SER A  35      10.722  -8.478  10.882  1.00  0.95           O  \\nATOM    258  OG  SER A  35       7.385 -10.345   8.698  1.00  0.82           O  \\nATOM    259  N   PHE A  36       9.007  -7.040  11.259  1.00  0.95           N  \\nATOM    260  CA  PHE A  36       9.873  -5.909  11.569  1.00  0.96           C  \\nATOM    261  C   PHE A  36       9.412  -5.208  12.842  1.00  0.96           C  \\nATOM    262  CB  PHE A  36       9.899  -4.915  10.403  1.00  0.95           C  \\nATOM    263  O   PHE A  36       8.856  -4.109  12.785  1.00  0.95           O  \\nATOM    264  CG  PHE A  36      10.149  -5.557   9.065  1.00  0.93           C  \\nATOM    265  CD1 PHE A  36      11.403  -6.061   8.743  1.00  0.89           C  \\nATOM    266  CD2 PHE A  36       9.128  -5.656   8.128  1.00  0.89           C  \\nATOM    267  CE1 PHE A  36      11.637  -6.656   7.505  1.00  0.89           C  \\nATOM    268  CE2 PHE A  36       9.355  -6.248   6.889  1.00  0.89           C  \\nATOM    269  CZ  PHE A  36      10.610  -6.747   6.579  1.00  0.87           C  \\nATOM    270  N   PRO A  37       9.723  -5.759  13.973  1.00  0.95           N  \\nATOM    271  CA  PRO A  37       9.161  -5.268  15.233  1.00  0.95           C  \\nATOM    272  C   PRO A  37       9.576  -3.832  15.544  1.00  0.95           C  \\nATOM    273  CB  PRO A  37       9.727  -6.237  16.273  1.00  0.94           C  \\nATOM    274  O   PRO A  37       8.892  -3.138  16.302  1.00  0.94           O  \\nATOM    275  CG  PRO A  37       9.966  -7.507  15.522  1.00  0.91           C  \\nATOM    276  CD  PRO A  37      10.349  -7.162  14.112  1.00  0.90           C  \\nATOM    277  N   THR A  38      10.682  -3.379  15.001  1.00  0.95           N  \\nATOM    278  CA  THR A  38      11.122  -2.008  15.235  1.00  0.94           C  \\nATOM    279  C   THR A  38      10.071  -1.013  14.753  1.00  0.94           C  \\nATOM    280  CB  THR A  38      12.463  -1.725  14.533  1.00  0.93           C  \\nATOM    281  O   THR A  38      10.000   0.113  15.251  1.00  0.94           O  \\nATOM    282  CG2 THR A  38      13.596  -2.524  15.168  1.00  0.80           C  \\nATOM    283  OG1 THR A  38      12.356  -2.084  13.150  1.00  0.81           O  \\nATOM    284  N   THR A  39       9.270  -1.412  13.855  1.00  0.95           N  \\nATOM    285  CA  THR A  39       8.261  -0.507  13.316  1.00  0.96           C  \\nATOM    286  C   THR A  39       7.176  -0.228  14.352  1.00  0.96           C  \\nATOM    287  CB  THR A  39       7.620  -1.082  12.039  1.00  0.95           C  \\nATOM    288  O   THR A  39       6.422   0.738  14.224  1.00  0.95           O  \\nATOM    289  CG2 THR A  39       8.653  -1.242  10.929  1.00  0.92           C  \\nATOM    290  OG1 THR A  39       7.049  -2.363  12.333  1.00  0.92           O  \\nATOM    291  N   LYS A  40       7.037  -1.036  15.335  1.00  0.95           N  \\nATOM    292  CA  LYS A  40       6.016  -0.870  16.365  1.00  0.95           C  \\nATOM    293  C   LYS A  40       6.226   0.425  17.144  1.00  0.95           C  \\nATOM    294  CB  LYS A  40       6.020  -2.064  17.322  1.00  0.95           C  \\nATOM    295  O   LYS A  40       5.293   0.942  17.761  1.00  0.94           O  \\nATOM    296  CG  LYS A  40       5.542  -3.364  16.692  1.00  0.93           C  \\nATOM    297  CD  LYS A  40       5.507  -4.498  17.709  1.00  0.91           C  \\nATOM    298  CE  LYS A  40       4.987  -5.788  17.091  1.00  0.88           C  \\nATOM    299  NZ  LYS A  40       4.901  -6.889  18.096  1.00  0.86           N  \\nATOM    300  N   THR A  41       7.435   0.940  17.200  1.00  0.95           N  \\nATOM    301  CA  THR A  41       7.766   2.148  17.947  1.00  0.94           C  \\nATOM    302  C   THR A  41       6.949   3.335  17.446  1.00  0.94           C  \\nATOM    303  CB  THR A  41       9.268   2.475  17.845  1.00  0.93           C  \\nATOM    304  O   THR A  41       6.727   4.298  18.183  1.00  0.93           O  \\nATOM    305  CG2 THR A  41      10.118   1.330  18.387  1.00  0.77           C  \\nATOM    306  OG1 THR A  41       9.609   2.699  16.472  1.00  0.79           O  \\nATOM    307  N   TYR A  42       6.403   3.249  16.213  1.00  0.94           N  \\nATOM    308  CA  TYR A  42       5.636   4.353  15.647  1.00  0.94           C  \\nATOM    309  C   TYR A  42       4.178   4.285  16.085  1.00  0.94           C  \\nATOM    310  CB  TYR A  42       5.723   4.340  14.118  1.00  0.94           C  \\nATOM    311  O   TYR A  42       3.385   5.173  15.762  1.00  0.93           O  \\nATOM    312  CG  TYR A  42       7.093   4.682  13.585  1.00  0.93           C  \\nATOM    313  CD1 TYR A  42       7.557   5.996  13.593  1.00  0.91           C  \\nATOM    314  CD2 TYR A  42       7.926   3.694  13.071  1.00  0.91           C  \\nATOM    315  CE1 TYR A  42       8.818   6.317  13.101  1.00  0.90           C  \\nATOM    316  CE2 TYR A  42       9.189   4.003  12.577  1.00  0.90           C  \\nATOM    317  OH  TYR A  42      10.875   5.627  12.109  1.00  0.88           O  \\nATOM    318  CZ  TYR A  42       9.626   5.315  12.597  1.00  0.90           C  \\nATOM    319  N   PHE A  43       3.764   3.287  16.751  1.00  0.94           N  \\nATOM    320  CA  PHE A  43       2.390   3.067  17.184  1.00  0.94           C  \\nATOM    321  C   PHE A  43       2.329   2.812  18.686  1.00  0.93           C  \\nATOM    322  CB  PHE A  43       1.768   1.889  16.427  1.00  0.93           C  \\nATOM    323  O   PHE A  43       1.762   1.812  19.129  1.00  0.91           O  \\nATOM    324  CG  PHE A  43       1.790   2.049  14.931  1.00  0.93           C  \\nATOM    325  CD1 PHE A  43       0.700   2.590  14.260  1.00  0.90           C  \\nATOM    326  CD2 PHE A  43       2.902   1.660  14.195  1.00  0.90           C  \\nATOM    327  CE1 PHE A  43       0.717   2.740  12.875  1.00  0.90           C  \\nATOM    328  CE2 PHE A  43       2.926   1.807  12.811  1.00  0.90           C  \\nATOM    329  CZ  PHE A  43       1.833   2.346  12.153  1.00  0.89           C  \\nATOM    330  N   PRO A  44       2.887   3.714  19.463  1.00  0.92           N  \\nATOM    331  CA  PRO A  44       3.074   3.479  20.897  1.00  0.92           C  \\nATOM    332  C   PRO A  44       1.753   3.316  21.646  1.00  0.91           C  \\nATOM    333  CB  PRO A  44       3.817   4.732  21.366  1.00  0.91           C  \\nATOM    334  O   PRO A  44       1.711   2.677  22.700  1.00  0.88           O  \\nATOM    335  CG  PRO A  44       3.470   5.781  20.361  1.00  0.88           C  \\nATOM    336  CD  PRO A  44       3.337   5.128  19.015  1.00  0.87           C  \\nATOM    337  N   HIS A  45       0.632   3.802  21.087  1.00  0.93           N  \\nATOM    338  CA  HIS A  45      -0.645   3.773  21.791  1.00  0.94           C  \\nATOM    339  C   HIS A  45      -1.604   2.774  21.153  1.00  0.94           C  \\nATOM    340  CB  HIS A  45      -1.276   5.167  21.812  1.00  0.91           C  \\nATOM    341  O   HIS A  45      -2.790   2.745  21.488  1.00  0.93           O  \\nATOM    342  CG  HIS A  45      -0.388   6.219  22.396  1.00  0.82           C  \\nATOM    343  CD2 HIS A  45       0.142   7.339  21.849  1.00  0.75           C  \\nATOM    344  ND1 HIS A  45       0.054   6.180  23.700  1.00  0.72           N  \\nATOM    345  CE1 HIS A  45       0.819   7.233  23.932  1.00  0.73           C  \\nATOM    346  NE2 HIS A  45       0.889   7.952  22.824  1.00  0.71           N  \\nATOM    347  N   PHE A  46      -1.096   2.003  20.265  1.00  0.93           N  \\nATOM    348  CA  PHE A  46      -1.936   1.036  19.568  1.00  0.94           C  \\nATOM    349  C   PHE A  46      -1.901  -0.317  20.269  1.00  0.93           C  \\nATOM    350  CB  PHE A  46      -1.487   0.884  18.111  1.00  0.92           C  \\nATOM    351  O   PHE A  46      -0.850  -0.744  20.752  1.00  0.92           O  \\nATOM    352  CG  PHE A  46      -2.090   1.902  17.181  1.00  0.89           C  \\nATOM    353  CD1 PHE A  46      -2.151   3.243  17.542  1.00  0.83           C  \\nATOM    354  CD2 PHE A  46      -2.594   1.519  15.945  1.00  0.83           C  \\nATOM    355  CE1 PHE A  46      -2.708   4.188  16.683  1.00  0.84           C  \\nATOM    356  CE2 PHE A  46      -3.152   2.457  15.082  1.00  0.81           C  \\nATOM    357  CZ  PHE A  46      -3.209   3.791  15.453  1.00  0.82           C  \\nATOM    358  N   ASP A  47      -3.019  -0.880  20.473  1.00  0.95           N  \\nATOM    359  CA  ASP A  47      -3.085  -2.330  20.623  1.00  0.95           C  \\nATOM    360  C   ASP A  47      -2.726  -3.035  19.317  1.00  0.95           C  \\nATOM    361  CB  ASP A  47      -4.479  -2.760  21.087  1.00  0.94           C  \\nATOM    362  O   ASP A  47      -3.495  -2.998  18.354  1.00  0.94           O  \\nATOM    363  CG  ASP A  47      -4.616  -4.264  21.239  1.00  0.89           C  \\nATOM    364  OD1 ASP A  47      -3.602  -4.985  21.115  1.00  0.83           O  \\nATOM    365  OD2 ASP A  47      -5.749  -4.734  21.481  1.00  0.86           O  \\nATOM    366  N   LEU A  48      -1.587  -3.738  19.331  1.00  0.93           N  \\nATOM    367  CA  LEU A  48      -1.062  -4.282  18.083  1.00  0.94           C  \\nATOM    368  C   LEU A  48      -1.283  -5.789  18.011  1.00  0.92           C  \\nATOM    369  CB  LEU A  48       0.429  -3.965  17.946  1.00  0.92           C  \\nATOM    370  O   LEU A  48      -0.667  -6.472  17.190  1.00  0.88           O  \\nATOM    371  CG  LEU A  48       0.813  -2.485  17.961  1.00  0.90           C  \\nATOM    372  CD1 LEU A  48       2.326  -2.328  17.860  1.00  0.85           C  \\nATOM    373  CD2 LEU A  48       0.116  -1.740  16.828  1.00  0.85           C  \\nATOM    374  N   SER A  49      -2.119  -6.347  18.814  1.00  0.93           N  \\nATOM    375  CA  SER A  49      -2.439  -7.771  18.788  1.00  0.92           C  \\nATOM    376  C   SER A  49      -3.190  -8.145  17.514  1.00  0.92           C  \\nATOM    377  CB  SER A  49      -3.270  -8.157  20.012  1.00  0.90           C  \\nATOM    378  O   SER A  49      -3.760  -7.279  16.846  1.00  0.92           O  \\nATOM    379  OG  SER A  49      -4.534  -7.517  19.984  1.00  0.81           O  \\nATOM    380  N   HIS A  50      -3.081  -9.380  17.246  1.00  0.90           N  \\nATOM    381  CA  HIS A  50      -3.778  -9.888  16.070  1.00  0.91           C  \\nATOM    382  C   HIS A  50      -5.273  -9.598  16.148  1.00  0.90           C  \\nATOM    383  CB  HIS A  50      -3.543 -11.392  15.915  1.00  0.87           C  \\nATOM    384  O   HIS A  50      -5.911  -9.873  17.168  1.00  0.87           O  \\nATOM    385  CG  HIS A  50      -4.205 -11.982  14.711  1.00  0.78           C  \\nATOM    386  CD2 HIS A  50      -5.240 -12.848  14.601  1.00  0.72           C  \\nATOM    387  ND1 HIS A  50      -3.805 -11.694  13.425  1.00  0.72           N  \\nATOM    388  CE1 HIS A  50      -4.568 -12.358  12.573  1.00  0.70           C  \\nATOM    389  NE2 HIS A  50      -5.447 -13.066  13.261  1.00  0.70           N  \\nATOM    390  N   GLY A  51      -5.782  -8.921  15.098  1.00  0.90           N  \\nATOM    391  CA  GLY A  51      -7.209  -8.656  15.009  1.00  0.90           C  \\nATOM    392  C   GLY A  51      -7.625  -7.384  15.723  1.00  0.90           C  \\nATOM    393  O   GLY A  51      -8.818  -7.127  15.900  1.00  0.89           O  \\nATOM    394  N   SER A  52      -6.672  -6.631  16.169  1.00  0.93           N  \\nATOM    395  CA  SER A  52      -6.923  -5.376  16.871  1.00  0.93           C  \\nATOM    396  C   SER A  52      -7.752  -4.422  16.018  1.00  0.94           C  \\nATOM    397  CB  SER A  52      -5.605  -4.708  17.264  1.00  0.92           C  \\nATOM    398  O   SER A  52      -7.521  -4.298  14.813  1.00  0.93           O  \\nATOM    399  OG  SER A  52      -5.784  -3.314  17.451  1.00  0.84           O  \\nATOM    400  N   ALA A  53      -8.736  -3.777  16.568  1.00  0.95           N  \\nATOM    401  CA  ALA A  53      -9.566  -2.782  15.894  1.00  0.95           C  \\nATOM    402  C   ALA A  53      -8.728  -1.596  15.424  1.00  0.95           C  \\nATOM    403  CB  ALA A  53     -10.683  -2.306  16.820  1.00  0.94           C  \\nATOM    404  O   ALA A  53      -8.997  -1.017  14.369  1.00  0.95           O  \\nATOM    405  N   GLN A  54      -7.734  -1.225  16.209  1.00  0.95           N  \\nATOM    406  CA  GLN A  54      -6.867  -0.104  15.862  1.00  0.96           C  \\nATOM    407  C   GLN A  54      -6.048  -0.409  14.611  1.00  0.96           C  \\nATOM    408  CB  GLN A  54      -5.938   0.237  17.028  1.00  0.95           C  \\nATOM    409  O   GLN A  54      -5.913   0.441  13.728  1.00  0.95           O  \\nATOM    410  CG  GLN A  54      -6.659   0.805  18.243  1.00  0.91           C  \\nATOM    411  CD  GLN A  54      -5.743   0.981  19.440  1.00  0.87           C  \\nATOM    412  NE2 GLN A  54      -5.650   2.209  19.940  1.00  0.86           N  \\nATOM    413  OE1 GLN A  54      -5.125   0.021  19.912  1.00  0.87           O  \\nATOM    414  N   VAL A  55      -5.518  -1.666  14.550  1.00  0.96           N  \\nATOM    415  CA  VAL A  55      -4.739  -2.049  13.377  1.00  0.96           C  \\nATOM    416  C   VAL A  55      -5.658  -2.171  12.164  1.00  0.96           C  \\nATOM    417  CB  VAL A  55      -3.981  -3.375  13.607  1.00  0.95           C  \\nATOM    418  O   VAL A  55      -5.316  -1.717  11.069  1.00  0.96           O  \\nATOM    419  CG1 VAL A  55      -3.321  -3.851  12.315  1.00  0.92           C  \\nATOM    420  CG2 VAL A  55      -2.939  -3.208  14.712  1.00  0.92           C  \\nATOM    421  N   LYS A  56      -6.816  -2.680  12.354  1.00  0.96           N  \\nATOM    422  CA  LYS A  56      -7.795  -2.797  11.277  1.00  0.96           C  \\nATOM    423  C   LYS A  56      -8.205  -1.424  10.754  1.00  0.96           C  \\nATOM    424  CB  LYS A  56      -9.028  -3.565  11.755  1.00  0.95           C  \\nATOM    425  O   LYS A  56      -8.253  -1.205   9.542  1.00  0.95           O  \\nATOM    426  CG  LYS A  56      -8.806  -5.063  11.902  1.00  0.87           C  \\nATOM    427  CD  LYS A  56     -10.075  -5.775  12.354  1.00  0.82           C  \\nATOM    428  CE  LYS A  56      -9.852  -7.273  12.509  1.00  0.77           C  \\nATOM    429  NZ  LYS A  56     -11.071  -7.964  13.025  1.00  0.71           N  \\nATOM    430  N   GLY A  57      -8.476  -0.553  11.663  1.00  0.96           N  \\nATOM    431  CA  GLY A  57      -8.820   0.802  11.265  1.00  0.95           C  \\nATOM    432  C   GLY A  57      -7.697   1.513  10.535  1.00  0.96           C  \\nATOM    433  O   GLY A  57      -7.933   2.197   9.536  1.00  0.95           O  \\nATOM    434  N   HIS A  58      -6.488   1.389  11.001  1.00  0.95           N  \\nATOM    435  CA  HIS A  58      -5.339   1.998  10.341  1.00  0.96           C  \\nATOM    436  C   HIS A  58      -5.075   1.353   8.985  1.00  0.96           C  \\nATOM    437  CB  HIS A  58      -4.095   1.890  11.225  1.00  0.95           C  \\nATOM    438  O   HIS A  58      -4.689   2.035   8.033  1.00  0.95           O  \\nATOM    439  CG  HIS A  58      -2.973   2.780  10.794  1.00  0.90           C  \\nATOM    440  CD2 HIS A  58      -1.790   2.495  10.199  1.00  0.83           C  \\nATOM    441  ND1 HIS A  58      -2.999   4.147  10.966  1.00  0.83           N  \\nATOM    442  CE1 HIS A  58      -1.878   4.665  10.494  1.00  0.84           C  \\nATOM    443  NE2 HIS A  58      -1.127   3.684  10.023  1.00  0.85           N  \\nATOM    444  N   GLY A  59      -5.275   0.006   8.901  1.00  0.96           N  \\nATOM    445  CA  GLY A  59      -5.169  -0.677   7.622  1.00  0.96           C  \\nATOM    446  C   GLY A  59      -6.124  -0.136   6.575  1.00  0.96           C  \\nATOM    447  O   GLY A  59      -5.752   0.022   5.410  1.00  0.96           O  \\nATOM    448  N   LYS A  60      -7.290   0.163   7.018  1.00  0.96           N  \\nATOM    449  CA  LYS A  60      -8.264   0.776   6.120  1.00  0.96           C  \\nATOM    450  C   LYS A  60      -7.790   2.148   5.648  1.00  0.96           C  \\nATOM    451  CB  LYS A  60      -9.624   0.901   6.809  1.00  0.95           C  \\nATOM    452  O   LYS A  60      -7.938   2.491   4.473  1.00  0.96           O  \\nATOM    453  CG  LYS A  60     -10.704   1.528   5.939  1.00  0.80           C  \\nATOM    454  CD  LYS A  60     -12.071   1.459   6.607  1.00  0.73           C  \\nATOM    455  CE  LYS A  60     -13.148   2.107   5.748  1.00  0.66           C  \\nATOM    456  NZ  LYS A  60     -12.861   1.952   4.290  1.00  0.61           N  \\nATOM    457  N   LYS A  61      -7.188   2.908   6.496  1.00  0.96           N  \\nATOM    458  CA  LYS A  61      -6.665   4.218   6.119  1.00  0.96           C  \\nATOM    459  C   LYS A  61      -5.542   4.088   5.094  1.00  0.96           C  \\nATOM    460  CB  LYS A  61      -6.164   4.971   7.352  1.00  0.95           C  \\nATOM    461  O   LYS A  61      -5.468   4.869   4.143  1.00  0.96           O  \\nATOM    462  CG  LYS A  61      -7.271   5.425   8.293  1.00  0.90           C  \\nATOM    463  CD  LYS A  61      -6.711   6.175   9.495  1.00  0.86           C  \\nATOM    464  CE  LYS A  61      -7.805   6.524  10.496  1.00  0.81           C  \\nATOM    465  NZ  LYS A  61      -7.245   7.145  11.733  1.00  0.76           N  \\nATOM    466  N   VAL A  62      -4.675   3.097   5.340  1.00  0.97           N  \\nATOM    467  CA  VAL A  62      -3.597   2.840   4.391  1.00  0.97           C  \\nATOM    468  C   VAL A  62      -4.185   2.459   3.034  1.00  0.97           C  \\nATOM    469  CB  VAL A  62      -2.650   1.728   4.894  1.00  0.96           C  \\nATOM    470  O   VAL A  62      -3.782   3.000   2.001  1.00  0.97           O  \\nATOM    471  CG1 VAL A  62      -1.669   1.318   3.797  1.00  0.92           C  \\nATOM    472  CG2 VAL A  62      -1.900   2.191   6.141  1.00  0.92           C  \\nATOM    473  N   ALA A  63      -5.181   1.581   3.020  1.00  0.97           N  \\nATOM    474  CA  ALA A  63      -5.821   1.133   1.786  1.00  0.97           C  \\nATOM    475  C   ALA A  63      -6.506   2.293   1.070  1.00  0.97           C  \\nATOM    476  CB  ALA A  63      -6.828   0.024   2.082  1.00  0.96           C  \\nATOM    477  O   ALA A  63      -6.430   2.407  -0.155  1.00  0.96           O  \\nATOM    478  N   ASP A  64      -7.160   3.151   1.835  1.00  0.96           N  \\nATOM    479  CA  ASP A  64      -7.818   4.318   1.257  1.00  0.96           C  \\nATOM    480  C   ASP A  64      -6.804   5.252   0.601  1.00  0.96           C  \\nATOM    481  CB  ASP A  64      -8.612   5.072   2.327  1.00  0.96           C  \\nATOM    482  O   ASP A  64      -7.052   5.783  -0.483  1.00  0.96           O  \\nATOM    483  CG  ASP A  64      -9.869   4.337   2.760  1.00  0.91           C  \\nATOM    484  OD1 ASP A  64     -10.299   3.397   2.057  1.00  0.86           O  \\nATOM    485  OD2 ASP A  64     -10.436   4.704   3.812  1.00  0.87           O  \\nATOM    486  N   ALA A  65      -5.701   5.440   1.256  1.00  0.97           N  \\nATOM    487  CA  ALA A  65      -4.641   6.284   0.711  1.00  0.97           C  \\nATOM    488  C   ALA A  65      -4.103   5.712  -0.597  1.00  0.97           C  \\nATOM    489  CB  ALA A  65      -3.511   6.441   1.726  1.00  0.96           C  \\nATOM    490  O   ALA A  65      -3.892   6.448  -1.564  1.00  0.97           O  \\nATOM    491  N   LEU A  66      -3.890   4.393  -0.631  1.00  0.97           N  \\nATOM    492  CA  LEU A  66      -3.388   3.754  -1.842  1.00  0.97           C  \\nATOM    493  C   LEU A  66      -4.426   3.812  -2.958  1.00  0.97           C  \\nATOM    494  CB  LEU A  66      -3.005   2.298  -1.561  1.00  0.97           C  \\nATOM    495  O   LEU A  66      -4.081   4.017  -4.124  1.00  0.97           O  \\nATOM    496  CG  LEU A  66      -1.782   2.079  -0.670  1.00  0.96           C  \\nATOM    497  CD1 LEU A  66      -1.620   0.598  -0.346  1.00  0.93           C  \\nATOM    498  CD2 LEU A  66      -0.526   2.626  -1.340  1.00  0.92           C  \\nATOM    499  N   THR A  67      -5.673   3.631  -2.600  1.00  0.97           N  \\nATOM    500  CA  THR A  67      -6.747   3.771  -3.578  1.00  0.97           C  \\nATOM    501  C   THR A  67      -6.742   5.168  -4.190  1.00  0.97           C  \\nATOM    502  CB  THR A  67      -8.121   3.491  -2.942  1.00  0.96           C  \\nATOM    503  O   THR A  67      -6.863   5.319  -5.408  1.00  0.96           O  \\nATOM    504  CG2 THR A  67      -9.245   3.686  -3.954  1.00  0.94           C  \\nATOM    505  OG1 THR A  67      -8.153   2.141  -2.462  1.00  0.94           O  \\nATOM    506  N   ASN A  68      -6.595   6.123  -3.329  1.00  0.97           N  \\nATOM    507  CA  ASN A  68      -6.501   7.501  -3.801  1.00  0.97           C  \\nATOM    508  C   ASN A  68      -5.282   7.705  -4.696  1.00  0.97           C  \\nATOM    509  CB  ASN A  68      -6.460   8.471  -2.618  1.00  0.96           C  \\nATOM    510  O   ASN A  68      -5.366   8.382  -5.722  1.00  0.96           O  \\nATOM    511  CG  ASN A  68      -6.597   9.919  -3.046  1.00  0.91           C  \\nATOM    512  ND2 ASN A  68      -5.770  10.787  -2.476  1.00  0.84           N  \\nATOM    513  OD1 ASN A  68      -7.441  10.255  -3.882  1.00  0.82           O  \\nATOM    514  N   ALA A  69      -4.179   7.127  -4.352  1.00  0.97           N  \\nATOM    515  CA  ALA A  69      -2.964   7.205  -5.158  1.00  0.97           C  \\nATOM    516  C   ALA A  69      -3.171   6.562  -6.526  1.00  0.97           C  \\nATOM    517  CB  ALA A  69      -1.799   6.538  -4.431  1.00  0.97           C  \\nATOM    518  O   ALA A  69      -2.761   7.116  -7.549  1.00  0.97           O  \\nATOM    519  N   VAL A  70      -3.835   5.399  -6.558  1.00  0.97           N  \\nATOM    520  CA  VAL A  70      -4.101   4.699  -7.810  1.00  0.97           C  \\nATOM    521  C   VAL A  70      -5.022   5.543  -8.688  1.00  0.97           C  \\nATOM    522  CB  VAL A  70      -4.727   3.309  -7.561  1.00  0.97           C  \\nATOM    523  O   VAL A  70      -4.806   5.655  -9.897  1.00  0.96           O  \\nATOM    524  CG1 VAL A  70      -5.225   2.699  -8.870  1.00  0.92           C  \\nATOM    525  CG2 VAL A  70      -3.718   2.382  -6.886  1.00  0.93           C  \\nATOM    526  N   ALA A  71      -6.028   6.193  -8.059  1.00  0.97           N  \\nATOM    527  CA  ALA A  71      -6.995   7.014  -8.783  1.00  0.96           C  \\nATOM    528  C   ALA A  71      -6.331   8.260  -9.363  1.00  0.96           C  \\nATOM    529  CB  ALA A  71      -8.150   7.408  -7.866  1.00  0.96           C  \\nATOM    530  O   ALA A  71      -6.775   8.789 -10.384  1.00  0.95           O  \\nATOM    531  N   HIS A  72      -5.292   8.731  -8.745  1.00  0.97           N  \\nATOM    532  CA  HIS A  72      -4.610   9.949  -9.167  1.00  0.97           C  \\nATOM    533  C   HIS A  72      -3.140   9.679  -9.473  1.00  0.96           C  \\nATOM    534  CB  HIS A  72      -4.732  11.031  -8.093  1.00  0.96           C  \\nATOM    535  O   HIS A  72      -2.270  10.474  -9.110  1.00  0.95           O  \\nATOM    536  CG  HIS A  72      -6.145  11.408  -7.780  1.00  0.92           C  \\nATOM    537  CD2 HIS A  72      -6.940  12.379  -8.289  1.00  0.83           C  \\nATOM    538  ND1 HIS A  72      -6.899  10.750  -6.832  1.00  0.81           N  \\nATOM    539  CE1 HIS A  72      -8.099  11.302  -6.772  1.00  0.82           C  \\nATOM    540  NE2 HIS A  72      -8.150  12.292  -7.647  1.00  0.84           N  \\nATOM    541  N   VAL A  73      -2.864   8.544 -10.158  1.00  0.96           N  \\nATOM    542  CA  VAL A  73      -1.518   8.016 -10.361  1.00  0.96           C  \\nATOM    543  C   VAL A  73      -0.661   9.051 -11.085  1.00  0.95           C  \\nATOM    544  CB  VAL A  73      -1.542   6.692 -11.157  1.00  0.95           C  \\nATOM    545  O   VAL A  73       0.552   9.119 -10.874  1.00  0.94           O  \\nATOM    546  CG1 VAL A  73      -2.044   6.927 -12.581  1.00  0.82           C  \\nATOM    547  CG2 VAL A  73      -0.153   6.056 -11.175  1.00  0.82           C  \\nATOM    548  N   ASP A  74      -1.252   9.948 -11.861  1.00  0.95           N  \\nATOM    549  CA  ASP A  74      -0.519  10.960 -12.617  1.00  0.95           C  \\nATOM    550  C   ASP A  74      -0.307  12.222 -11.784  1.00  0.95           C  \\nATOM    551  CB  ASP A  74      -1.258  11.302 -13.912  1.00  0.94           C  \\nATOM    552  O   ASP A  74       0.398  13.140 -12.207  1.00  0.94           O  \\nATOM    553  CG  ASP A  74      -1.211  10.182 -14.936  1.00  0.90           C  \\nATOM    554  OD1 ASP A  74      -0.120   9.621 -15.177  1.00  0.86           O  \\nATOM    555  OD2 ASP A  74      -2.274   9.860 -15.510  1.00  0.86           O  \\nATOM    556  N   ASP A  75      -0.930  12.292 -10.662  1.00  0.96           N  \\nATOM    557  CA  ASP A  75      -0.791  13.468  -9.810  1.00  0.96           C  \\nATOM    558  C   ASP A  75      -0.704  13.073  -8.338  1.00  0.96           C  \\nATOM    559  CB  ASP A  75      -1.962  14.430 -10.028  1.00  0.95           C  \\nATOM    560  O   ASP A  75      -1.386  13.656  -7.492  1.00  0.95           O  \\nATOM    561  CG  ASP A  75      -1.733  15.794  -9.400  1.00  0.90           C  \\nATOM    562  OD1 ASP A  75      -0.569  16.148  -9.117  1.00  0.83           O  \\nATOM    563  OD2 ASP A  75      -2.728  16.520  -9.184  1.00  0.86           O  \\nATOM    564  N   MET A  76       0.091  12.121  -7.998  1.00  0.96           N  \\nATOM    565  CA  MET A  76       0.188  11.578  -6.646  1.00  0.96           C  \\nATOM    566  C   MET A  76       0.702  12.633  -5.671  1.00  0.96           C  \\nATOM    567  CB  MET A  76       1.103  10.353  -6.622  1.00  0.95           C  \\nATOM    568  O   MET A  76       0.236  12.712  -4.534  1.00  0.95           O  \\nATOM    569  CG  MET A  76       0.459   9.096  -7.184  1.00  0.90           C  \\nATOM    570  SD  MET A  76       1.449   7.588  -6.847  1.00  0.91           S  \\nATOM    571  CE  MET A  76       2.791   7.812  -8.048  1.00  0.83           C  \\nATOM    572  N   PRO A  77       1.676  13.502  -6.114  1.00  0.95           N  \\nATOM    573  CA  PRO A  77       2.143  14.514  -5.163  1.00  0.95           C  \\nATOM    574  C   PRO A  77       1.009  15.381  -4.621  1.00  0.95           C  \\nATOM    575  CB  PRO A  77       3.121  15.351  -5.991  1.00  0.94           C  \\nATOM    576  O   PRO A  77       0.970  15.675  -3.423  1.00  0.94           O  \\nATOM    577  CG  PRO A  77       3.683  14.398  -6.996  1.00  0.92           C  \\nATOM    578  CD  PRO A  77       2.584  13.482  -7.452  1.00  0.90           C  \\nATOM    579  N   ASN A  78       0.143  15.741  -5.464  1.00  0.96           N  \\nATOM    580  CA  ASN A  78      -0.997  16.540  -5.028  1.00  0.96           C  \\nATOM    581  C   ASN A  78      -1.991  15.707  -4.223  1.00  0.96           C  \\nATOM    582  CB  ASN A  78      -1.694  17.183  -6.229  1.00  0.94           C  \\nATOM    583  O   ASN A  78      -2.474  16.147  -3.178  1.00  0.96           O  \\nATOM    584  CG  ASN A  78      -2.795  18.142  -5.822  1.00  0.77           C  \\nATOM    585  ND2 ASN A  78      -3.942  18.044  -6.484  1.00  0.64           N  \\nATOM    586  OD1 ASN A  78      -2.616  18.964  -4.920  1.00  0.65           O  \\nATOM    587  N   ALA A  79      -2.274  14.566  -4.662  1.00  0.96           N  \\nATOM    588  CA  ALA A  79      -3.262  13.690  -4.039  1.00  0.96           C  \\nATOM    589  C   ALA A  79      -2.823  13.277  -2.637  1.00  0.96           C  \\nATOM    590  CB  ALA A  79      -3.498  12.454  -4.904  1.00  0.95           C  \\nATOM    591  O   ALA A  79      -3.658  13.074  -1.752  1.00  0.95           O  \\nATOM    592  N   LEU A  80      -1.515  13.168  -2.408  1.00  0.96           N  \\nATOM    593  CA  LEU A  80      -0.999  12.658  -1.142  1.00  0.96           C  \\nATOM    594  C   LEU A  80      -0.334  13.770  -0.338  1.00  0.96           C  \\nATOM    595  CB  LEU A  80      -0.001  11.523  -1.388  1.00  0.96           C  \\nATOM    596  O   LEU A  80       0.399  13.499   0.616  1.00  0.95           O  \\nATOM    597  CG  LEU A  80      -0.539  10.294  -2.123  1.00  0.94           C  \\nATOM    598  CD1 LEU A  80       0.585   9.297  -2.382  1.00  0.91           C  \\nATOM    599  CD2 LEU A  80      -1.664   9.644  -1.324  1.00  0.91           C  \\nATOM    600  N   SER A  81      -0.521  15.039  -0.720  1.00  0.95           N  \\nATOM    601  CA  SER A  81       0.184  16.164  -0.115  1.00  0.95           C  \\nATOM    602  C   SER A  81      -0.058  16.222   1.390  1.00  0.95           C  \\nATOM    603  CB  SER A  81      -0.253  17.480  -0.760  1.00  0.94           C  \\nATOM    604  O   SER A  81       0.883  16.392   2.169  1.00  0.94           O  \\nATOM    605  OG  SER A  81       0.373  18.583  -0.127  1.00  0.86           O  \\nATOM    606  N   ALA A  82      -1.296  16.009   1.805  1.00  0.95           N  \\nATOM    607  CA  ALA A  82      -1.609  16.048   3.232  1.00  0.95           C  \\nATOM    608  C   ALA A  82      -0.918  14.909   3.976  1.00  0.95           C  \\nATOM    609  CB  ALA A  82      -3.119  15.983   3.447  1.00  0.94           C  \\nATOM    610  O   ALA A  82      -0.419  15.098   5.088  1.00  0.95           O  \\nATOM    611  N   LEU A  83      -0.828  13.775   3.396  1.00  0.95           N  \\nATOM    612  CA  LEU A  83      -0.173  12.636   4.030  1.00  0.95           C  \\nATOM    613  C   LEU A  83       1.339  12.833   4.072  1.00  0.95           C  \\nATOM    614  CB  LEU A  83      -0.510  11.341   3.286  1.00  0.95           C  \\nATOM    615  O   LEU A  83       1.993  12.445   5.042  1.00  0.95           O  \\nATOM    616  CG  LEU A  83      -1.931  10.805   3.468  1.00  0.92           C  \\nATOM    617  CD1 LEU A  83      -2.158   9.591   2.573  1.00  0.87           C  \\nATOM    618  CD2 LEU A  83      -2.186  10.454   4.930  1.00  0.86           C  \\nATOM    619  N   SER A  84       1.857  13.417   3.017  1.00  0.94           N  \\nATOM    620  CA  SER A  84       3.281  13.733   3.049  1.00  0.94           C  \\nATOM    621  C   SER A  84       3.617  14.655   4.216  1.00  0.94           C  \\nATOM    622  CB  SER A  84       3.716  14.382   1.734  1.00  0.93           C  \\nATOM    623  O   SER A  84       4.582  14.416   4.945  1.00  0.93           O  \\nATOM    624  OG  SER A  84       5.103  14.673   1.751  1.00  0.75           O  \\nATOM    625  N   ASP A  85       2.766  15.665   4.468  1.00  0.94           N  \\nATOM    626  CA  ASP A  85       2.953  16.582   5.588  1.00  0.94           C  \\nATOM    627  C   ASP A  85       2.849  15.847   6.923  1.00  0.94           C  \\nATOM    628  CB  ASP A  85       1.926  17.716   5.531  1.00  0.93           C  \\nATOM    629  O   ASP A  85       3.652  16.079   7.829  1.00  0.93           O  \\nATOM    630  CG  ASP A  85       2.280  18.785   4.513  1.00  0.89           C  \\nATOM    631  OD1 ASP A  85       3.482  19.071   4.321  1.00  0.85           O  \\nATOM    632  OD2 ASP A  85       1.349  19.349   3.898  1.00  0.86           O  \\nATOM    633  N   LEU A  86       1.866  15.041   7.022  1.00  0.94           N  \\nATOM    634  CA  LEU A  86       1.636  14.290   8.252  1.00  0.94           C  \\nATOM    635  C   LEU A  86       2.847  13.431   8.599  1.00  0.94           C  \\nATOM    636  CB  LEU A  86       0.392  13.409   8.117  1.00  0.93           C  \\nATOM    637  O   LEU A  86       3.327  13.458   9.735  1.00  0.93           O  \\nATOM    638  CG  LEU A  86       0.068  12.506   9.309  1.00  0.90           C  \\nATOM    639  CD1 LEU A  86      -0.255  13.348  10.539  1.00  0.85           C  \\nATOM    640  CD2 LEU A  86      -1.090  11.573   8.973  1.00  0.85           C  \\nATOM    641  N   HIS A  87       3.409  12.697   7.650  1.00  0.93           N  \\nATOM    642  CA  HIS A  87       4.510  11.776   7.910  1.00  0.93           C  \\nATOM    643  C   HIS A  87       5.814  12.530   8.147  1.00  0.92           C  \\nATOM    644  CB  HIS A  87       4.674  10.796   6.747  1.00  0.93           C  \\nATOM    645  O   HIS A  87       6.658  12.087   8.930  1.00  0.91           O  \\nATOM    646  CG  HIS A  87       3.619   9.737   6.702  1.00  0.91           C  \\nATOM    647  CD2 HIS A  87       3.554   8.520   7.293  1.00  0.87           C  \\nATOM    648  ND1 HIS A  87       2.456   9.875   5.975  1.00  0.88           N  \\nATOM    649  CE1 HIS A  87       1.720   8.787   6.123  1.00  0.88           C  \\nATOM    650  NE2 HIS A  87       2.363   7.949   6.917  1.00  0.88           N  \\nATOM    651  N   ALA A  88       5.969  13.634   7.435  1.00  0.92           N  \\nATOM    652  CA  ALA A  88       7.223  14.376   7.533  1.00  0.92           C  \\nATOM    653  C   ALA A  88       7.270  15.206   8.813  1.00  0.91           C  \\nATOM    654  CB  ALA A  88       7.407  15.274   6.313  1.00  0.91           C  \\nATOM    655  O   ALA A  88       8.299  15.254   9.491  1.00  0.90           O  \\nATOM    656  N   HIS A  89       6.182  15.754   9.203  1.00  0.91           N  \\nATOM    657  CA  HIS A  89       6.264  16.795  10.222  1.00  0.91           C  \\nATOM    658  C   HIS A  89       5.613  16.343  11.525  1.00  0.91           C  \\nATOM    659  CB  HIS A  89       5.606  18.084   9.725  1.00  0.90           C  \\nATOM    660  O   HIS A  89       6.060  16.721  12.610  1.00  0.90           O  \\nATOM    661  CG  HIS A  89       6.265  18.667   8.516  1.00  0.87           C  \\nATOM    662  CD2 HIS A  89       5.795  18.888   7.266  1.00  0.83           C  \\nATOM    663  ND1 HIS A  89       7.574  19.098   8.519  1.00  0.82           N  \\nATOM    664  CE1 HIS A  89       7.881  19.561   7.318  1.00  0.82           C  \\nATOM    665  NE2 HIS A  89       6.819  19.444   6.540  1.00  0.80           N  \\nATOM    666  N   LYS A  90       4.573  15.561  11.436  1.00  0.92           N  \\nATOM    667  CA  LYS A  90       3.879  15.169  12.659  1.00  0.91           C  \\nATOM    668  C   LYS A  90       4.341  13.796  13.138  1.00  0.92           C  \\nATOM    669  CB  LYS A  90       2.365  15.164  12.439  1.00  0.90           C  \\nATOM    670  O   LYS A  90       4.723  13.634  14.299  1.00  0.91           O  \\nATOM    671  CG  LYS A  90       1.766  16.547  12.225  1.00  0.81           C  \\nATOM    672  CD  LYS A  90       0.244  16.504  12.234  1.00  0.75           C  \\nATOM    673  CE  LYS A  90      -0.357  17.881  11.983  1.00  0.65           C  \\nATOM    674  NZ  LYS A  90      -1.850  17.844  11.994  1.00  0.56           N  \\nATOM    675  N   LEU A  91       4.357  12.856  12.265  1.00  0.92           N  \\nATOM    676  CA  LEU A  91       4.699  11.490  12.643  1.00  0.92           C  \\nATOM    677  C   LEU A  91       6.207  11.271  12.583  1.00  0.91           C  \\nATOM    678  CB  LEU A  91       3.989  10.487  11.730  1.00  0.91           C  \\nATOM    679  O   LEU A  91       6.751  10.458  13.334  1.00  0.90           O  \\nATOM    680  CG  LEU A  91       2.460  10.528  11.737  1.00  0.89           C  \\nATOM    681  CD1 LEU A  91       1.899   9.530  10.730  1.00  0.86           C  \\nATOM    682  CD2 LEU A  91       1.924  10.242  13.136  1.00  0.86           C  \\nATOM    683  N   ARG A  92       6.891  11.994  11.615  1.00  0.91           N  \\nATOM    684  CA  ARG A  92       8.340  11.942  11.445  1.00  0.91           C  \\nATOM    685  C   ARG A  92       8.821  10.503  11.295  1.00  0.91           C  \\nATOM    686  CB  ARG A  92       9.044  12.609  12.628  1.00  0.89           C  \\nATOM    687  O   ARG A  92       9.755  10.080  11.980  1.00  0.89           O  \\nATOM    688  CG  ARG A  92       8.703  14.081  12.798  1.00  0.84           C  \\nATOM    689  CD  ARG A  92       9.510  14.721  13.919  1.00  0.81           C  \\nATOM    690  NE  ARG A  92       9.105  16.105  14.152  1.00  0.75           N  \\nATOM    691  NH1 ARG A  92      10.521  16.434  15.947  1.00  0.57           N  \\nATOM    692  NH2 ARG A  92       9.156  18.131  15.232  1.00  0.52           N  \\nATOM    693  CZ  ARG A  92       9.595  16.887  15.110  1.00  0.73           C  \\nATOM    694  N   VAL A  93       8.192   9.778  10.462  1.00  0.92           N  \\nATOM    695  CA  VAL A  93       8.518   8.385  10.177  1.00  0.92           C  \\nATOM    696  C   VAL A  93       9.804   8.312   9.357  1.00  0.92           C  \\nATOM    697  CB  VAL A  93       7.368   7.675   9.427  1.00  0.91           C  \\nATOM    698  O   VAL A  93       9.940   8.998   8.341  1.00  0.91           O  \\nATOM    699  CG1 VAL A  93       7.761   6.243   9.068  1.00  0.86           C  \\nATOM    700  CG2 VAL A  93       6.095   7.687  10.271  1.00  0.85           C  \\nATOM    701  N   ASP A  94      10.764   7.507   9.844  1.00  0.92           N  \\nATOM    702  CA  ASP A  94      11.971   7.280   9.055  1.00  0.92           C  \\nATOM    703  C   ASP A  94      11.630   6.701   7.684  1.00  0.93           C  \\nATOM    704  CB  ASP A  94      12.930   6.347   9.797  1.00  0.91           C  \\nATOM    705  O   ASP A  94      10.916   5.701   7.588  1.00  0.92           O  \\nATOM    706  CG  ASP A  94      14.299   6.263   9.145  1.00  0.86           C  \\nATOM    707  OD1 ASP A  94      14.391   6.367   7.903  1.00  0.82           O  \\nATOM    708  OD2 ASP A  94      15.295   6.089   9.880  1.00  0.84           O  \\nATOM    709  N   PRO A  95      12.186   7.238   6.605  1.00  0.91           N  \\nATOM    710  CA  PRO A  95      11.840   6.833   5.241  1.00  0.91           C  \\nATOM    711  C   PRO A  95      12.108   5.352   4.979  1.00  0.91           C  \\nATOM    712  CB  PRO A  95      12.742   7.713   4.372  1.00  0.90           C  \\nATOM    713  O   PRO A  95      11.439   4.738   4.144  1.00  0.91           O  \\nATOM    714  CG  PRO A  95      12.894   8.984   5.144  1.00  0.88           C  \\nATOM    715  CD  PRO A  95      12.954   8.649   6.607  1.00  0.87           C  \\nATOM    716  N   VAL A  96      13.045   4.754   5.652  1.00  0.92           N  \\nATOM    717  CA  VAL A  96      13.426   3.365   5.415  1.00  0.92           C  \\nATOM    718  C   VAL A  96      12.255   2.444   5.747  1.00  0.92           C  \\nATOM    719  CB  VAL A  96      14.669   2.969   6.244  1.00  0.89           C  \\nATOM    720  O   VAL A  96      12.139   1.350   5.188  1.00  0.91           O  \\nATOM    721  CG1 VAL A  96      14.322   2.882   7.729  1.00  0.74           C  \\nATOM    722  CG2 VAL A  96      15.239   1.642   5.747  1.00  0.74           C  \\nATOM    723  N   ASN A  97      11.415   2.838   6.604  1.00  0.94           N  \\nATOM    724  CA  ASN A  97      10.312   1.998   7.059  1.00  0.94           C  \\nATOM    725  C   ASN A  97       9.233   1.862   5.988  1.00  0.94           C  \\nATOM    726  CB  ASN A  97       9.711   2.555   8.351  1.00  0.93           C  \\nATOM    727  O   ASN A  97       8.460   0.902   5.998  1.00  0.94           O  \\nATOM    728  CG  ASN A  97      10.613   2.349   9.551  1.00  0.88           C  \\nATOM    729  ND2 ASN A  97      11.263   3.418   9.996  1.00  0.84           N  \\nATOM    730  OD1 ASN A  97      10.726   1.237  10.074  1.00  0.84           O  \\nATOM    731  N   PHE A  98       9.159   2.835   5.094  1.00  0.95           N  \\nATOM    732  CA  PHE A  98       8.192   2.723   4.008  1.00  0.95           C  \\nATOM    733  C   PHE A  98       8.507   1.522   3.125  1.00  0.95           C  \\nATOM    734  CB  PHE A  98       8.175   4.003   3.167  1.00  0.94           C  \\nATOM    735  O   PHE A  98       7.597   0.874   2.602  1.00  0.94           O  \\nATOM    736  CG  PHE A  98       7.532   5.175   3.858  1.00  0.92           C  \\nATOM    737  CD1 PHE A  98       6.153   5.343   3.832  1.00  0.88           C  \\nATOM    738  CD2 PHE A  98       8.307   6.109   4.533  1.00  0.88           C  \\nATOM    739  CE1 PHE A  98       5.554   6.427   4.470  1.00  0.88           C  \\nATOM    740  CE2 PHE A  98       7.716   7.195   5.174  1.00  0.87           C  \\nATOM    741  CZ  PHE A  98       6.340   7.352   5.140  1.00  0.86           C  \\nATOM    742  N   LYS A  99       9.772   1.202   3.002  1.00  0.95           N  \\nATOM    743  CA  LYS A  99      10.157   0.030   2.221  1.00  0.95           C  \\nATOM    744  C   LYS A  99       9.680  -1.257   2.889  1.00  0.96           C  \\nATOM    745  CB  LYS A  99      11.674  -0.011   2.028  1.00  0.93           C  \\nATOM    746  O   LYS A  99       9.287  -2.206   2.208  1.00  0.95           O  \\nATOM    747  CG  LYS A  99      12.212   1.084   1.119  1.00  0.77           C  \\nATOM    748  CD  LYS A  99      13.716   0.953   0.916  1.00  0.71           C  \\nATOM    749  CE  LYS A  99      14.259   2.061   0.023  1.00  0.63           C  \\nATOM    750  NZ  LYS A  99      15.745   1.983  -0.114  1.00  0.57           N  \\nATOM    751  N   LEU A 100       9.738  -1.265   4.220  1.00  0.96           N  \\nATOM    752  CA  LEU A 100       9.306  -2.444   4.962  1.00  0.96           C  \\nATOM    753  C   LEU A 100       7.814  -2.692   4.768  1.00  0.96           C  \\nATOM    754  CB  LEU A 100       9.618  -2.283   6.453  1.00  0.96           C  \\nATOM    755  O   LEU A 100       7.397  -3.823   4.506  1.00  0.96           O  \\nATOM    756  CG  LEU A 100      11.090  -2.089   6.820  1.00  0.94           C  \\nATOM    757  CD1 LEU A 100      11.244  -1.937   8.329  1.00  0.91           C  \\nATOM    758  CD2 LEU A 100      11.928  -3.255   6.306  1.00  0.90           C  \\nATOM    759  N   LEU A 101       6.991  -1.653   4.853  1.00  0.97           N  \\nATOM    760  CA  LEU A 101       5.558  -1.802   4.628  1.00  0.97           C  \\nATOM    761  C   LEU A 101       5.271  -2.169   3.175  1.00  0.97           C  \\nATOM    762  CB  LEU A 101       4.820  -0.513   4.996  1.00  0.96           C  \\nATOM    763  O   LEU A 101       4.405  -3.002   2.900  1.00  0.97           O  \\nATOM    764  CG  LEU A 101       3.304  -0.515   4.787  1.00  0.93           C  \\nATOM    765  CD1 LEU A 101       2.664  -1.662   5.561  1.00  0.87           C  \\nATOM    766  CD2 LEU A 101       2.704   0.822   5.207  1.00  0.86           C  \\nATOM    767  N   SER A 102       5.964  -1.564   2.232  1.00  0.97           N  \\nATOM    768  CA  SER A 102       5.804  -1.892   0.819  1.00  0.97           C  \\nATOM    769  C   SER A 102       6.021  -3.381   0.570  1.00  0.97           C  \\nATOM    770  CB  SER A 102       6.776  -1.077  -0.034  1.00  0.96           C  \\nATOM    771  O   SER A 102       5.250  -4.013  -0.155  1.00  0.97           O  \\nATOM    772  OG  SER A 102       6.426   0.296  -0.024  1.00  0.83           O  \\nATOM    773  N   HIS A 103       7.066  -3.848   1.198  1.00  0.97           N  \\nATOM    774  CA  HIS A 103       7.349  -5.273   1.070  1.00  0.97           C  \\nATOM    775  C   HIS A 103       6.184  -6.114   1.583  1.00  0.97           C  \\nATOM    776  CB  HIS A 103       8.630  -5.635   1.825  1.00  0.96           C  \\nATOM    777  O   HIS A 103       5.757  -7.061   0.919  1.00  0.97           O  \\nATOM    778  CG  HIS A 103       8.953  -7.095   1.791  1.00  0.92           C  \\nATOM    779  CD2 HIS A 103       8.857  -8.049   2.746  1.00  0.83           C  \\nATOM    780  ND1 HIS A 103       9.446  -7.722   0.667  1.00  0.83           N  \\nATOM    781  CE1 HIS A 103       9.638  -9.003   0.934  1.00  0.83           C  \\nATOM    782  NE2 HIS A 103       9.288  -9.228   2.189  1.00  0.85           N  \\nATOM    783  N   CYS A 104       5.640  -5.781   2.698  1.00  0.97           N  \\nATOM    784  CA  CYS A 104       4.558  -6.553   3.298  1.00  0.97           C  \\nATOM    785  C   CYS A 104       3.273  -6.409   2.493  1.00  0.97           C  \\nATOM    786  CB  CYS A 104       4.320  -6.110   4.741  1.00  0.97           C  \\nATOM    787  O   CYS A 104       2.489  -7.354   2.393  1.00  0.97           O  \\nATOM    788  SG  CYS A 104       5.656  -6.557   5.871  1.00  0.95           S  \\nATOM    789  N   LEU A 105       3.042  -5.250   1.901  1.00  0.97           N  \\nATOM    790  CA  LEU A 105       1.913  -5.066   0.995  1.00  0.97           C  \\nATOM    791  C   LEU A 105       2.033  -5.986  -0.216  1.00  0.97           C  \\nATOM    792  CB  LEU A 105       1.826  -3.608   0.535  1.00  0.97           C  \\nATOM    793  O   LEU A 105       1.057  -6.630  -0.610  1.00  0.97           O  \\nATOM    794  CG  LEU A 105       1.249  -2.613   1.543  1.00  0.94           C  \\nATOM    795  CD1 LEU A 105       1.566  -1.183   1.118  1.00  0.87           C  \\nATOM    796  CD2 LEU A 105      -0.256  -2.812   1.690  1.00  0.86           C  \\nATOM    797  N   LEU A 106       3.199  -6.059  -0.798  1.00  0.97           N  \\nATOM    798  CA  LEU A 106       3.408  -6.918  -1.958  1.00  0.97           C  \\nATOM    799  C   LEU A 106       3.178  -8.382  -1.599  1.00  0.97           C  \\nATOM    800  CB  LEU A 106       4.822  -6.733  -2.515  1.00  0.97           C  \\nATOM    801  O   LEU A 106       2.559  -9.124  -2.365  1.00  0.97           O  \\nATOM    802  CG  LEU A 106       5.114  -5.393  -3.193  1.00  0.96           C  \\nATOM    803  CD1 LEU A 106       6.602  -5.268  -3.502  1.00  0.93           C  \\nATOM    804  CD2 LEU A 106       4.284  -5.244  -4.463  1.00  0.92           C  \\nATOM    805  N   VAL A 107       3.675  -8.806  -0.435  1.00  0.97           N  \\nATOM    806  CA  VAL A 107       3.480 -10.178   0.022  1.00  0.97           C  \\nATOM    807  C   VAL A 107       1.989 -10.457   0.199  1.00  0.97           C  \\nATOM    808  CB  VAL A 107       4.234 -10.448   1.344  1.00  0.96           C  \\nATOM    809  O   VAL A 107       1.496 -11.511  -0.210  1.00  0.96           O  \\nATOM    810  CG1 VAL A 107       3.842 -11.806   1.922  1.00  0.89           C  \\nATOM    811  CG2 VAL A 107       5.743 -10.373   1.122  1.00  0.89           C  \\nATOM    812  N   THR A 108       1.285  -9.521   0.774  1.00  0.97           N  \\nATOM    813  CA  THR A 108      -0.149  -9.681   0.991  1.00  0.97           C  \\nATOM    814  C   THR A 108      -0.888  -9.789  -0.340  1.00  0.96           C  \\nATOM    815  CB  THR A 108      -0.725  -8.508   1.807  1.00  0.96           C  \\nATOM    816  O   THR A 108      -1.757 -10.648  -0.505  1.00  0.96           O  \\nATOM    817  CG2 THR A 108      -2.235  -8.643   1.973  1.00  0.93           C  \\nATOM    818  OG1 THR A 108      -0.112  -8.489   3.102  1.00  0.93           O  \\nATOM    819  N   LEU A 109      -0.573  -8.919  -1.283  1.00  0.97           N  \\nATOM    820  CA  LEU A 109      -1.225  -8.956  -2.588  1.00  0.97           C  \\nATOM    821  C   LEU A 109      -0.914 -10.260  -3.314  1.00  0.97           C  \\nATOM    822  CB  LEU A 109      -0.783  -7.765  -3.442  1.00  0.96           C  \\nATOM    823  O   LEU A 109      -1.798 -10.856  -3.935  1.00  0.96           O  \\nATOM    824  CG  LEU A 109      -1.270  -6.387  -2.988  1.00  0.94           C  \\nATOM    825  CD1 LEU A 109      -0.632  -5.293  -3.837  1.00  0.90           C  \\nATOM    826  CD2 LEU A 109      -2.791  -6.308  -3.059  1.00  0.89           C  \\nATOM    827  N   ALA A 110       0.313 -10.708  -3.180  1.00  0.97           N  \\nATOM    828  CA  ALA A 110       0.703 -11.966  -3.812  1.00  0.97           C  \\nATOM    829  C   ALA A 110      -0.087 -13.138  -3.234  1.00  0.96           C  \\nATOM    830  CB  ALA A 110       2.201 -12.205  -3.644  1.00  0.96           C  \\nATOM    831  O   ALA A 110      -0.476 -14.052  -3.965  1.00  0.96           O  \\nATOM    832  N   ALA A 111      -0.305 -13.134  -1.962  1.00  0.96           N  \\nATOM    833  CA  ALA A 111      -1.012 -14.209  -1.272  1.00  0.96           C  \\nATOM    834  C   ALA A 111      -2.483 -14.246  -1.675  1.00  0.96           C  \\nATOM    835  CB  ALA A 111      -0.882 -14.047   0.241  1.00  0.96           C  \\nATOM    836  O   ALA A 111      -3.079 -15.321  -1.778  1.00  0.95           O  \\nATOM    837  N   HIS A 112      -3.104 -13.076  -1.971  1.00  0.96           N  \\nATOM    838  CA  HIS A 112      -4.547 -13.021  -2.177  1.00  0.96           C  \\nATOM    839  C   HIS A 112      -4.889 -12.960  -3.662  1.00  0.96           C  \\nATOM    840  CB  HIS A 112      -5.147 -11.817  -1.450  1.00  0.96           C  \\nATOM    841  O   HIS A 112      -6.034 -13.202  -4.050  1.00  0.94           O  \\nATOM    842  CG  HIS A 112      -5.249 -12.000   0.031  1.00  0.95           C  \\nATOM    843  CD2 HIS A 112      -4.488 -11.516   1.041  1.00  0.92           C  \\nATOM    844  ND1 HIS A 112      -6.229 -12.769   0.620  1.00  0.92           N  \\nATOM    845  CE1 HIS A 112      -6.068 -12.747   1.933  1.00  0.91           C  \\nATOM    846  NE2 HIS A 112      -5.018 -11.994   2.214  1.00  0.92           N  \\nATOM    847  N   LEU A 113      -3.935 -12.591  -4.479  1.00  0.96           N  \\nATOM    848  CA  LEU A 113      -4.163 -12.478  -5.916  1.00  0.96           C  \\nATOM    849  C   LEU A 113      -3.168 -13.333  -6.693  1.00  0.95           C  \\nATOM    850  CB  LEU A 113      -4.057 -11.017  -6.362  1.00  0.94           C  \\nATOM    851  O   LEU A 113      -2.469 -12.831  -7.576  1.00  0.94           O  \\nATOM    852  CG  LEU A 113      -4.960 -10.019  -5.635  1.00  0.89           C  \\nATOM    853  CD1 LEU A 113      -4.652  -8.597  -6.091  1.00  0.81           C  \\nATOM    854  CD2 LEU A 113      -6.428 -10.356  -5.871  1.00  0.80           C  \\nATOM    855  N   PRO A 114      -3.087 -14.615  -6.375  1.00  0.95           N  \\nATOM    856  CA  PRO A 114      -2.041 -15.449  -6.973  1.00  0.95           C  \\nATOM    857  C   PRO A 114      -2.122 -15.494  -8.497  1.00  0.94           C  \\nATOM    858  CB  PRO A 114      -2.306 -16.832  -6.371  1.00  0.93           C  \\nATOM    859  O   PRO A 114      -1.091 -15.560  -9.171  1.00  0.92           O  \\nATOM    860  CG  PRO A 114      -3.739 -16.795  -5.947  1.00  0.91           C  \\nATOM    861  CD  PRO A 114      -4.067 -15.403  -5.490  1.00  0.91           C  \\nATOM    862  N   ALA A 115      -3.324 -15.443  -9.102  1.00  0.95           N  \\nATOM    863  CA  ALA A 115      -3.485 -15.517 -10.552  1.00  0.95           C  \\nATOM    864  C   ALA A 115      -3.121 -14.190 -11.213  1.00  0.94           C  \\nATOM    865  CB  ALA A 115      -4.916 -15.910 -10.909  1.00  0.93           C  \\nATOM    866  O   ALA A 115      -2.602 -14.168 -12.331  1.00  0.93           O  \\nATOM    867  N   GLU A 116      -3.324 -13.126 -10.589  1.00  0.95           N  \\nATOM    868  CA  GLU A 116      -3.128 -11.796 -11.157  1.00  0.94           C  \\nATOM    869  C   GLU A 116      -1.714 -11.286 -10.893  1.00  0.95           C  \\nATOM    870  CB  GLU A 116      -4.156 -10.813 -10.592  1.00  0.92           C  \\nATOM    871  O   GLU A 116      -1.235 -10.382 -11.581  1.00  0.94           O  \\nATOM    872  CG  GLU A 116      -5.588 -11.113 -11.009  1.00  0.80           C  \\nATOM    873  CD  GLU A 116      -6.229 -12.224 -10.192  1.00  0.75           C  \\nATOM    874  OE1 GLU A 116      -5.710 -12.554  -9.102  1.00  0.74           O  \\nATOM    875  OE2 GLU A 116      -7.260 -12.770 -10.647  1.00  0.72           O  \\nATOM    876  N   PHE A 117      -1.070 -11.860  -9.920  1.00  0.96           N  \\nATOM    877  CA  PHE A 117       0.205 -11.320  -9.461  1.00  0.96           C  \\nATOM    878  C   PHE A 117       1.362 -11.912 -10.258  1.00  0.96           C  \\nATOM    879  CB  PHE A 117       0.401 -11.596  -7.967  1.00  0.95           C  \\nATOM    880  O   PHE A 117       2.272 -12.516  -9.686  1.00  0.94           O  \\nATOM    881  CG  PHE A 117       1.238 -10.563  -7.264  1.00  0.92           C  \\nATOM    882  CD1 PHE A 117       2.594 -10.776  -7.047  1.00  0.87           C  \\nATOM    883  CD2 PHE A 117       0.669  -9.377  -6.818  1.00  0.86           C  \\nATOM    884  CE1 PHE A 117       3.372  -9.821  -6.396  1.00  0.86           C  \\nATOM    885  CE2 PHE A 117       1.440  -8.418  -6.167  1.00  0.86           C  \\nATOM    886  CZ  PHE A 117       2.790  -8.642  -5.956  1.00  0.85           C  \\nATOM    887  N   THR A 118       1.432 -11.619 -11.543  1.00  0.96           N  \\nATOM    888  CA  THR A 118       2.512 -12.027 -12.435  1.00  0.96           C  \\nATOM    889  C   THR A 118       3.778 -11.223 -12.155  1.00  0.97           C  \\nATOM    890  CB  THR A 118       2.110 -11.859 -13.912  1.00  0.96           C  \\nATOM    891  O   THR A 118       3.732 -10.205 -11.461  1.00  0.96           O  \\nATOM    892  CG2 THR A 118       0.776 -12.540 -14.199  1.00  0.92           C  \\nATOM    893  OG1 THR A 118       1.996 -10.463 -14.215  1.00  0.92           O  \\nATOM    894  N   PRO A 119       4.853 -11.622 -12.654  1.00  0.96           N  \\nATOM    895  CA  PRO A 119       6.099 -10.873 -12.472  1.00  0.96           C  \\nATOM    896  C   PRO A 119       5.998  -9.431 -12.963  1.00  0.96           C  \\nATOM    897  CB  PRO A 119       7.110 -11.664 -13.306  1.00  0.96           C  \\nATOM    898  O   PRO A 119       6.541  -8.521 -12.331  1.00  0.96           O  \\nATOM    899  CG  PRO A 119       6.638 -13.081 -13.236  1.00  0.93           C  \\nATOM    900  CD  PRO A 119       5.136 -13.085 -13.273  1.00  0.91           C  \\nATOM    901  N   ALA A 120       5.302  -9.215 -14.116  1.00  0.96           N  \\nATOM    902  CA  ALA A 120       5.124  -7.861 -14.633  1.00  0.96           C  \\nATOM    903  C   ALA A 120       4.297  -7.011 -13.672  1.00  0.97           C  \\nATOM    904  CB  ALA A 120       4.461  -7.900 -16.008  1.00  0.96           C  \\nATOM    905  O   ALA A 120       4.623  -5.847 -13.426  1.00  0.97           O  \\nATOM    906  N   VAL A 121       3.238  -7.599 -13.131  1.00  0.97           N  \\nATOM    907  CA  VAL A 121       2.391  -6.885 -12.181  1.00  0.97           C  \\nATOM    908  C   VAL A 121       3.172  -6.607 -10.898  1.00  0.97           C  \\nATOM    909  CB  VAL A 121       1.105  -7.680 -11.860  1.00  0.96           C  \\nATOM    910  O   VAL A 121       3.096  -5.509 -10.344  1.00  0.97           O  \\nATOM    911  CG1 VAL A 121       0.370  -7.060 -10.672  1.00  0.94           C  \\nATOM    912  CG2 VAL A 121       0.194  -7.737 -13.085  1.00  0.94           C  \\nATOM    913  N   HIS A 122       3.921  -7.599 -10.441  1.00  0.97           N  \\nATOM    914  CA  HIS A 122       4.789  -7.396  -9.287  1.00  0.97           C  \\nATOM    915  C   HIS A 122       5.711  -6.199  -9.495  1.00  0.97           C  \\nATOM    916  CB  HIS A 122       5.616  -8.653  -9.011  1.00  0.96           C  \\nATOM    917  O   HIS A 122       5.822  -5.336  -8.621  1.00  0.96           O  \\nATOM    918  CG  HIS A 122       6.414  -8.580  -7.748  1.00  0.93           C  \\nATOM    919  CD2 HIS A 122       6.530  -7.597  -6.825  1.00  0.88           C  \\nATOM    920  ND1 HIS A 122       7.222  -9.611  -7.318  1.00  0.87           N  \\nATOM    921  CE1 HIS A 122       7.801  -9.264  -6.181  1.00  0.87           C  \\nATOM    922  NE2 HIS A 122       7.397  -8.046  -5.860  1.00  0.89           N  \\nATOM    923  N   ALA A 123       6.325  -6.152 -10.608  1.00  0.97           N  \\nATOM    924  CA  ALA A 123       7.263  -5.073 -10.909  1.00  0.97           C  \\nATOM    925  C   ALA A 123       6.554  -3.722 -10.935  1.00  0.97           C  \\nATOM    926  CB  ALA A 123       7.960  -5.330 -12.242  1.00  0.96           C  \\nATOM    927  O   ALA A 123       7.062  -2.736 -10.396  1.00  0.96           O  \\nATOM    928  N   SER A 124       5.403  -3.686 -11.556  1.00  0.97           N  \\nATOM    929  CA  SER A 124       4.625  -2.454 -11.637  1.00  0.97           C  \\nATOM    930  C   SER A 124       4.188  -1.984 -10.254  1.00  0.97           C  \\nATOM    931  CB  SER A 124       3.398  -2.653 -12.528  1.00  0.97           C  \\nATOM    932  O   SER A 124       4.278  -0.795  -9.940  1.00  0.97           O  \\nATOM    933  OG  SER A 124       3.785  -2.917 -13.866  1.00  0.92           O  \\nATOM    934  N   LEU A 125       3.732  -2.907  -9.418  1.00  0.97           N  \\nATOM    935  CA  LEU A 125       3.305  -2.562  -8.066  1.00  0.97           C  \\nATOM    936  C   LEU A 125       4.488  -2.096  -7.224  1.00  0.97           C  \\nATOM    937  CB  LEU A 125       2.626  -3.759  -7.396  1.00  0.97           C  \\nATOM    938  O   LEU A 125       4.350  -1.188  -6.402  1.00  0.97           O  \\nATOM    939  CG  LEU A 125       1.239  -4.133  -7.920  1.00  0.95           C  \\nATOM    940  CD1 LEU A 125       0.814  -5.491  -7.371  1.00  0.92           C  \\nATOM    941  CD2 LEU A 125       0.221  -3.059  -7.551  1.00  0.92           C  \\nATOM    942  N   ASP A 126       5.592  -2.739  -7.401  1.00  0.97           N  \\nATOM    943  CA  ASP A 126       6.801  -2.316  -6.701  1.00  0.96           C  \\nATOM    944  C   ASP A 126       7.169  -0.878  -7.058  1.00  0.96           C  \\nATOM    945  CB  ASP A 126       7.966  -3.253  -7.028  1.00  0.96           C  \\nATOM    946  O   ASP A 126       7.447  -0.064  -6.174  1.00  0.96           O  \\nATOM    947  CG  ASP A 126       9.161  -3.055  -6.112  1.00  0.88           C  \\nATOM    948  OD1 ASP A 126       8.974  -2.943  -4.881  1.00  0.78           O  \\nATOM    949  OD2 ASP A 126      10.300  -3.009  -6.626  1.00  0.85           O  \\nATOM    950  N   LYS A 127       7.135  -0.518  -8.337  1.00  0.97           N  \\nATOM    951  CA  LYS A 127       7.427   0.843  -8.778  1.00  0.97           C  \\nATOM    952  C   LYS A 127       6.400   1.831  -8.231  1.00  0.97           C  \\nATOM    953  CB  LYS A 127       7.463   0.916 -10.305  1.00  0.96           C  \\nATOM    954  O   LYS A 127       6.755   2.935  -7.814  1.00  0.96           O  \\nATOM    955  CG  LYS A 127       8.677   0.243 -10.929  1.00  0.90           C  \\nATOM    956  CD  LYS A 127       8.628   0.300 -12.450  1.00  0.84           C  \\nATOM    957  CE  LYS A 127       9.744  -0.523 -13.078  1.00  0.77           C  \\nATOM    958  NZ  LYS A 127       9.650  -0.538 -14.569  1.00  0.69           N  \\nATOM    959  N   PHE A 128       5.164   1.430  -8.249  1.00  0.97           N  \\nATOM    960  CA  PHE A 128       4.085   2.270  -7.743  1.00  0.97           C  \\nATOM    961  C   PHE A 128       4.289   2.583  -6.265  1.00  0.97           C  \\nATOM    962  CB  PHE A 128       2.728   1.589  -7.953  1.00  0.97           C  \\nATOM    963  O   PHE A 128       4.257   3.747  -5.862  1.00  0.97           O  \\nATOM    964  CG  PHE A 128       1.599   2.244  -7.204  1.00  0.97           C  \\nATOM    965  CD1 PHE A 128       1.058   3.446  -7.645  1.00  0.96           C  \\nATOM    966  CD2 PHE A 128       1.077   1.656  -6.059  1.00  0.95           C  \\nATOM    967  CE1 PHE A 128       0.012   4.053  -6.954  1.00  0.95           C  \\nATOM    968  CE2 PHE A 128       0.032   2.258  -5.363  1.00  0.95           C  \\nATOM    969  CZ  PHE A 128      -0.498   3.457  -5.812  1.00  0.94           C  \\nATOM    970  N   LEU A 129       4.572   1.531  -5.471  1.00  0.97           N  \\nATOM    971  CA  LEU A 129       4.757   1.732  -4.038  1.00  0.97           C  \\nATOM    972  C   LEU A 129       6.014   2.549  -3.762  1.00  0.97           C  \\nATOM    973  CB  LEU A 129       4.839   0.385  -3.314  1.00  0.97           C  \\nATOM    974  O   LEU A 129       6.048   3.346  -2.822  1.00  0.97           O  \\nATOM    975  CG  LEU A 129       3.556  -0.447  -3.282  1.00  0.96           C  \\nATOM    976  CD1 LEU A 129       3.832  -1.823  -2.685  1.00  0.95           C  \\nATOM    977  CD2 LEU A 129       2.469   0.276  -2.494  1.00  0.95           C  \\nATOM    978  N   ALA A 130       7.037   2.377  -4.575  1.00  0.96           N  \\nATOM    979  CA  ALA A 130       8.232   3.210  -4.462  1.00  0.96           C  \\nATOM    980  C   ALA A 130       7.908   4.674  -4.742  1.00  0.96           C  \\nATOM    981  CB  ALA A 130       9.317   2.716  -5.417  1.00  0.95           C  \\nATOM    982  O   ALA A 130       8.403   5.569  -4.053  1.00  0.96           O  \\nATOM    983  N   SER A 131       7.071   4.927  -5.735  1.00  0.96           N  \\nATOM    984  CA  SER A 131       6.656   6.289  -6.057  1.00  0.96           C  \\nATOM    985  C   SER A 131       5.840   6.901  -4.923  1.00  0.96           C  \\nATOM    986  CB  SER A 131       5.840   6.309  -7.350  1.00  0.95           C  \\nATOM    987  O   SER A 131       6.035   8.066  -4.571  1.00  0.96           O  \\nATOM    988  OG  SER A 131       6.632   5.893  -8.450  1.00  0.81           O  \\nATOM    989  N   VAL A 132       4.947   6.118  -4.305  1.00  0.96           N  \\nATOM    990  CA  VAL A 132       4.173   6.563  -3.150  1.00  0.96           C  \\nATOM    991  C   VAL A 132       5.116   6.930  -2.006  1.00  0.96           C  \\nATOM    992  CB  VAL A 132       3.171   5.482  -2.688  1.00  0.96           C  \\nATOM    993  O   VAL A 132       4.969   7.987  -1.387  1.00  0.96           O  \\nATOM    994  CG1 VAL A 132       2.579   5.840  -1.326  1.00  0.93           C  \\nATOM    995  CG2 VAL A 132       2.063   5.304  -3.725  1.00  0.93           C  \\nATOM    996  N   SER A 133       6.101   6.108  -1.756  1.00  0.95           N  \\nATOM    997  CA  SER A 133       7.069   6.364  -0.695  1.00  0.95           C  \\nATOM    998  C   SER A 133       7.829   7.663  -0.941  1.00  0.95           C  \\nATOM    999  CB  SER A 133       8.056   5.201  -0.577  1.00  0.94           C  \\nATOM   1000  O   SER A 133       8.049   8.446  -0.015  1.00  0.94           O  \\nATOM   1001  OG  SER A 133       7.405   4.037  -0.098  1.00  0.78           O  \\nATOM   1002  N   THR A 134       8.211   7.833  -2.191  1.00  0.94           N  \\nATOM   1003  CA  THR A 134       8.931   9.044  -2.567  1.00  0.94           C  \\nATOM   1004  C   THR A 134       8.084  10.284  -2.294  1.00  0.94           C  \\nATOM   1005  CB  THR A 134       9.338   9.014  -4.052  1.00  0.93           C  \\nATOM   1006  O   THR A 134       8.576  11.270  -1.741  1.00  0.94           O  \\nATOM   1007  CG2 THR A 134      10.003  10.323  -4.467  1.00  0.88           C  \\nATOM   1008  OG1 THR A 134      10.256   7.935  -4.268  1.00  0.89           O  \\nATOM   1009  N   VAL A 135       6.835  10.225  -2.612  1.00  0.95           N  \\nATOM   1010  CA  VAL A 135       5.943  11.359  -2.394  1.00  0.95           C  \\nATOM   1011  C   VAL A 135       5.762  11.594  -0.896  1.00  0.95           C  \\nATOM   1012  CB  VAL A 135       4.570  11.139  -3.068  1.00  0.94           C  \\nATOM   1013  O   VAL A 135       5.837  12.732  -0.426  1.00  0.94           O  \\nATOM   1014  CG1 VAL A 135       3.569  12.199  -2.612  1.00  0.92           C  \\nATOM   1015  CG2 VAL A 135       4.714  11.154  -4.588  1.00  0.91           C  \\nATOM   1016  N   LEU A 136       5.595  10.544  -0.138  1.00  0.95           N  \\nATOM   1017  CA  LEU A 136       5.316  10.650   1.290  1.00  0.95           C  \\nATOM   1018  C   LEU A 136       6.527  11.192   2.042  1.00  0.94           C  \\nATOM   1019  CB  LEU A 136       4.914   9.288   1.861  1.00  0.94           C  \\nATOM   1020  O   LEU A 136       6.380  11.835   3.084  1.00  0.93           O  \\nATOM   1021  CG  LEU A 136       3.544   8.754   1.439  1.00  0.93           C  \\nATOM   1022  CD1 LEU A 136       3.278   7.402   2.093  1.00  0.90           C  \\nATOM   1023  CD2 LEU A 136       2.447   9.751   1.796  1.00  0.90           C  \\nATOM   1024  N   THR A 137       7.716  11.025   1.471  1.00  0.94           N  \\nATOM   1025  CA  THR A 137       8.921  11.447   2.177  1.00  0.93           C  \\nATOM   1026  C   THR A 137       9.515  12.697   1.534  1.00  0.93           C  \\nATOM   1027  CB  THR A 137       9.978  10.327   2.198  1.00  0.92           C  \\nATOM   1028  O   THR A 137      10.617  13.120   1.888  1.00  0.91           O  \\nATOM   1029  CG2 THR A 137       9.463   9.099   2.943  1.00  0.85           C  \\nATOM   1030  OG1 THR A 137      10.298   9.955   0.851  1.00  0.88           O  \\nATOM   1031  N   SER A 138       8.899  13.226   0.660  1.00  0.93           N  \\nATOM   1032  CA  SER A 138       9.431  14.330  -0.133  1.00  0.93           C  \\nATOM   1033  C   SER A 138       9.574  15.594   0.707  1.00  0.92           C  \\nATOM   1034  CB  SER A 138       8.532  14.607  -1.338  1.00  0.92           C  \\nATOM   1035  O   SER A 138      10.308  16.513   0.336  1.00  0.90           O  \\nATOM   1036  OG  SER A 138       7.225  14.961  -0.919  1.00  0.88           O  \\nATOM   1037  N   LYS A 139       8.882  15.632   1.759  1.00  0.92           N  \\nATOM   1038  CA  LYS A 139       8.876  16.872   2.530  1.00  0.92           C  \\nATOM   1039  C   LYS A 139       9.664  16.716   3.828  1.00  0.91           C  \\nATOM   1040  CB  LYS A 139       7.442  17.306   2.836  1.00  0.91           C  \\nATOM   1041  O   LYS A 139       9.591  17.573   4.711  1.00  0.88           O  \\nATOM   1042  CG  LYS A 139       6.635  17.688   1.605  1.00  0.90           C  \\nATOM   1043  CD  LYS A 139       5.257  18.219   1.981  1.00  0.88           C  \\nATOM   1044  CE  LYS A 139       4.462  18.635   0.752  1.00  0.85           C  \\nATOM   1045  NZ  LYS A 139       3.076  19.063   1.107  1.00  0.81           N  \\nATOM   1046  N   TYR A 140      10.462  15.638   3.953  1.00  0.89           N  \\nATOM   1047  CA  TYR A 140      11.336  15.456   5.106  1.00  0.88           C  \\nATOM   1048  C   TYR A 140      12.450  16.496   5.115  1.00  0.84           C  \\nATOM   1049  CB  TYR A 140      11.936  14.047   5.109  1.00  0.86           C  \\nATOM   1050  O   TYR A 140      12.917  16.923   4.057  1.00  0.79           O  \\nATOM   1051  CG  TYR A 140      10.959  12.972   5.519  1.00  0.84           C  \\nATOM   1052  CD1 TYR A 140      11.119  12.277   6.716  1.00  0.80           C  \\nATOM   1053  CD2 TYR A 140       9.876  12.647   4.710  1.00  0.80           C  \\nATOM   1054  CE1 TYR A 140      10.223  11.284   7.096  1.00  0.80           C  \\nATOM   1055  CE2 TYR A 140       8.973  11.656   5.080  1.00  0.79           C  \\nATOM   1056  OH  TYR A 140       8.264   9.999   6.645  1.00  0.71           O  \\nATOM   1057  CZ  TYR A 140       9.154  10.981   6.273  1.00  0.79           C  \\nATOM   1058  N   ARG A 141      12.557  17.156   6.259  1.00  0.78           N  \\nATOM   1059  CA  ARG A 141      13.618  18.145   6.419  1.00  0.77           C  \\nATOM   1060  C   ARG A 141      14.957  17.471   6.695  1.00  0.75           C  \\nATOM   1061  CB  ARG A 141      13.279  19.119   7.549  1.00  0.72           C  \\nATOM   1062  O   ARG A 141      15.008  16.411   7.322  1.00  0.69           O  \\nATOM   1063  CG  ARG A 141      12.110  20.041   7.239  1.00  0.65           C  \\nATOM   1064  CD  ARG A 141      11.922  21.097   8.319  1.00  0.64           C  \\nATOM   1065  NE  ARG A 141      10.799  21.980   8.019  1.00  0.53           N  \\nATOM   1066  NH1 ARG A 141      11.030  23.262   9.927  1.00  0.42           N  \\nATOM   1067  NH2 ARG A 141       9.365  23.729   8.422  1.00  0.38           N  \\nATOM   1068  CZ  ARG A 141      10.401  22.989   8.790  1.00  0.57           C  \",\"pdb\");\n",
              "\tviewer_1744327365365795.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"green\"}});\n",
              "\tviewer_1744327365365795.zoomTo();\n",
              "viewer_1744327365365795.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "The target structure is in cartoon and the target positions (pocket) in stick format.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import os #@markdown #Get the parameters (if not downloaded)\n",
        "if not os.path.exists('/content/params40000.npy'):\n",
        "  print('Getting Umol-pocket network parameters.')\n",
        "  #!wget https://zenodo.org/records/10048543/files/params40000.npy\n",
        "  !wget https://zenodo.org/records/10397462/files/params40000.npy\n",
        "\n",
        "if not os.path.exists('/content/params60000.npy'):\n",
        "  print('Getting Umol network parameters.')\n",
        "  !wget https://zenodo.org/records/10489242/files/params60000.npy\n",
        "\n",
        "#@markdown #Make the input features for the network\n",
        "from make_msa_seq_feats_colab import process\n",
        "import pickle\n",
        "#Make MSA feats\n",
        "feature_dict = process(FASTA_FILE, [MSA])\n",
        "#Write out features as a pickled dictionary.\n",
        "features_output_path = os.path.join(OUTDIR, 'msa_features.pkl')\n",
        "with open(features_output_path, 'wb') as f:\n",
        "    pickle.dump(feature_dict, f, protocol=4)\n",
        "print('Saved MSA features to',features_output_path)\n",
        "\n",
        "\n",
        "#Ligand feats\n",
        "from make_ligand_feats_colab import bonds_from_smiles\n",
        "#Atom encoding - no hydrogens\n",
        "atom_encoding = {'B':0, 'C':1, 'F':2, 'I':3, 'N':4, 'O':5, 'P':6, 'S':7,'Br':8, 'Cl':9, #Individual encoding\n",
        "                'As':10, 'Co':10, 'Fe':10, 'Mg':10, 'Pt':10, 'Rh':10, 'Ru':10, 'Se':10, 'Si':10, 'Te':10, 'V':10, 'Zn':10 #Joint (rare)\n",
        "                 }\n",
        "\n",
        "#Get the atom types and bonds\n",
        "atom_types, atoms, bond_types, bond_lengths, bond_mask = bonds_from_smiles(LIGAND, atom_encoding)\n",
        "\n",
        "ligand_inp_feats = {}\n",
        "ligand_inp_feats['atoms'] = atoms\n",
        "ligand_inp_feats['atom_types'] = atom_types\n",
        "ligand_inp_feats['bond_types'] = bond_types\n",
        "ligand_inp_feats['bond_lengths'] = bond_lengths\n",
        "ligand_inp_feats['bond_mask'] = bond_mask\n",
        "#Write out features as a pickled dictionary.\n",
        "\n",
        "features_output_path = os.path.join(OUTDIR, 'ligand_inp_features.pkl')\n",
        "with open(features_output_path, 'wb') as f:\n",
        "    pickle.dump(ligand_inp_feats, f, protocol=4)\n",
        "print('Saved ligand features to',features_output_path)"
      ],
      "metadata": {
        "id": "WK5jUQ7Hlw2Z",
        "outputId": "8bf11fb6-3d3a-4845-b4d6-ed62663f981d",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 332
        }
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "error",
          "ename": "FileNotFoundError",
          "evalue": "[Errno 2] No such file or directory: '/content/Umol/data/test_case/Hit-1/Hit-1.a3m'",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-5-cf170a2974c1>\u001b[0m in \u001b[0;36m<cell line: 0>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     13\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mpickle\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     14\u001b[0m \u001b[0;31m#Make MSA feats\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 15\u001b[0;31m \u001b[0mfeature_dict\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mprocess\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mFASTA_FILE\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0mMSA\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     16\u001b[0m \u001b[0;31m#Write out features as a pickled dictionary.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     17\u001b[0m \u001b[0mfeatures_output_path\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mos\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mOUTDIR\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m'msa_features.pkl'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/content/Umol/src/make_msa_seq_feats_colab.py\u001b[0m in \u001b[0;36mprocess\u001b[0;34m(input_fasta_path, input_msas)\u001b[0m\n\u001b[1;32m     92\u001b[0m     \u001b[0mparsed_delmat\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     93\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0mcustom_msa\u001b[0m \u001b[0;32min\u001b[0m \u001b[0minput_msas\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 94\u001b[0;31m       \u001b[0mmsa\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m''\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mline\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mline\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mcustom_msa\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     95\u001b[0m       \u001b[0;32mif\u001b[0m \u001b[0mcustom_msa\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m3\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;34m'sto'\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     96\u001b[0m         \u001b[0mparsed_msa\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mparsed_deletion_matrix\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0m_\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mparsers\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mparse_stockholm\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmsa\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: '/content/Umol/data/test_case/Hit-1/Hit-1.a3m'"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#@markdown #Predict the protein-ligand structure (a few minutes)\n",
        "from net.model import config\n",
        "from predict_colab import predict\n",
        "import numpy as np\n",
        "MSA_FEATS='/content/'+ID+'/msa_features.pkl'\n",
        "LIGAND_FEATS='/content/'+ID+'/ligand_inp_features.pkl'\n",
        "if len(TARGET_POSITIONS)>0:\n",
        "  print('Target positions specified. Using pocket version.')\n",
        "  PARAMS=np.load('/content/params40000.npy', allow_pickle=True)\n",
        "else:\n",
        "  print('No target positions specified. Using no-pocket version.')\n",
        "  PARAMS=np.load('/content/params60000.npy', allow_pickle=True)\n",
        "\n",
        "#Predict\n",
        "predict(config.CONFIG,\n",
        "            MSA_FEATS,\n",
        "            LIGAND_FEATS,\n",
        "            ID,\n",
        "            TARGET_POSITIONS,\n",
        "            PARAMS,\n",
        "            NUM_RECYCLES,\n",
        "            outdir=OUTDIR)\n",
        "from relax.align_ligand_conformer_colab import read_pdb, generate_best_conformer, align_coords_transform, write_sdf\n",
        "import pandas as pd\n",
        "RAW_PDB=OUTDIR+'/'+ID+'_pred_raw.pdb'\n",
        "#Get a conformer\n",
        "pred_ligand = read_pdb(RAW_PDB)\n",
        "best_conf, best_conf_pos, best_conf_err, atoms, nonH_inds, mol, best_conf_id  = generate_best_conformer(pred_ligand['chain_coords'], LIGAND)\n",
        "\n",
        "#Align it to the prediction\n",
        "aligned_conf_pos = align_coords_transform(pred_ligand['chain_coords'], best_conf_pos, nonH_inds)\n",
        "\n",
        "#Write sdf\n",
        "write_sdf(mol, best_conf, aligned_conf_pos, best_conf_id, OUTDIR+ID+'_pred_ligand.sdf')\n",
        "!grep ATOM $RAW_PDB > $OUTDIR/$ID'_pred_protein.pdb'\n",
        "\n",
        "#Ligand plDDT\n",
        "!grep HETATM $RAW_PDB| cut -c65-66 > $OUTDIR/ligand_plddt.csv"
      ],
      "metadata": {
        "cellView": "form",
        "id": "IVwnH1ganaVu"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@markdown #Visualise\n",
        "import py3Dmol\n",
        "import pandas as pd\n",
        "import numpy as np\n",
        "\n",
        "ligand_plddt = pd.read_csv(OUTDIR+'ligand_plddt.csv', header=None)\n",
        "print('The average ligand plDDT is:', np.round(ligand_plddt[0].mean(),1))\n",
        "\n",
        "view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "view.addModel(open(OUTDIR+ID+\"_pred_ligand.sdf\",'r').read(),'sdf')\n",
        "view.setStyle({'stick': {'color':'cyan'}})\n",
        "view.addModel(open(OUTDIR+ID+\"_pred_protein.pdb\",'r').read(),'pdb')\n",
        "view.setStyle({'chain':'A'},{'cartoon': {'color':'green'}})\n",
        "view.zoomTo()\n",
        "view.show()"
      ],
      "metadata": {
        "cellView": "form",
        "id": "S_2lYMSYtzXB"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Download unrelaxed results\n",
        "from google.colab import files\n",
        "files.download(OUTDIR+ID+\"_pred_ligand.sdf\")\n",
        "files.download(OUTDIR+ID+\"_pred_protein.pdb\")"
      ],
      "metadata": {
        "cellView": "form",
        "id": "EKdew712yH6o"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@markdown # **Relaxation** - this section requires additional installations *but* it can help fix clashes in the predicted protein\n",
        "\n",
        "#@markdown ###Press play and install conda for relaxation with OpenMM\n",
        "!pip install -q condacolab\n",
        "import condacolab\n",
        "condacolab.install()"
      ],
      "metadata": {
        "cellView": "form",
        "id": "3uvsbkoaCz0S"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@markdown ###Install OpenMM in a conda environment (this can take a while).\n",
        "%%bash\n",
        "conda create -c conda-forge --name openmm openmm\n",
        "source activate openmm\n",
        "conda install -c conda-forge openff-toolkit\n",
        "conda install -c conda-forge pdbfixer\n",
        "conda install -c omnia openmmforcefields\n"
      ],
      "metadata": {
        "cellView": "form",
        "id": "ORhPP9YHDnx-"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@markdown ###**Relax the structure**\n",
        "#@markdown ### You have to double click on this cell and change the ID if you are not running the test case.\n",
        "\n",
        "%%bash\n",
        "source activate openmm\n",
        "\n",
        "python\n",
        "ID='7NB4' #Type the ID in here!\n",
        "import openmm as mm\n",
        "import argparse\n",
        "import sys\n",
        "import numpy as np\n",
        "import openmm.app as mm_app\n",
        "import openmm.unit as mm_unit\n",
        "import pdbfixer\n",
        "import pdb\n",
        "from sys import stdout\n",
        "from openmm.app import PDBFile, Modeller\n",
        "import mdtraj\n",
        "from openmmforcefields.generators import SystemGenerator\n",
        "from openff.toolkit import Molecule\n",
        "from openff.toolkit.utils.exceptions import UndefinedStereochemistryError, RadicalsNotSupportedError\n",
        "from openmm import CustomExternalForce\n",
        "\n",
        "###########Functions##########\n",
        "def fix_pdb(\n",
        "        pdbname,\n",
        "        outdir,\n",
        "        file_name\n",
        "        ):\n",
        "    \"\"\"Add hydrogens to the PDB file\n",
        "    \"\"\"\n",
        "    fixer = pdbfixer.PDBFixer(pdbname)\n",
        "    fixer.findMissingResidues()\n",
        "    fixer.findNonstandardResidues()\n",
        "    fixer.replaceNonstandardResidues()\n",
        "    fixer.findMissingAtoms()\n",
        "    fixer.addMissingAtoms()\n",
        "    fixer.addMissingHydrogens(7.0)\n",
        "    mm_app.PDBFile.writeFile(fixer.topology, fixer.positions, open(f'{outdir}/{file_name}_hydrogen_added.pdb', 'w'))\n",
        "    return fixer.topology, fixer.positions\n",
        "\n",
        "def set_system(topology):\n",
        "    \"\"\"\n",
        "    Set the system using the topology from the pdb file\n",
        "    \"\"\"\n",
        "    #Put it in a force field to skip adding all particles manually\n",
        "    forcefield = mm_app.ForceField('amber14-all.xml', 'amber14/tip3pfb.xml')\n",
        "    system = forcefield.createSystem(topology,\n",
        "                                     removeCMMotion=False,\n",
        "                                     nonbondedMethod=mm_app.NoCutoff,\n",
        "                                     rigidWater=True #Use implicit solvent\n",
        "                                     )\n",
        "    return system\n",
        "\n",
        "def minimize_energy(\n",
        "        topology,\n",
        "        system,\n",
        "        positions,\n",
        "        outdir,\n",
        "        out_title\n",
        "        ):\n",
        "    '''Function that minimizes energy, given topology, OpenMM system, and positions '''\n",
        "    #Use a Brownian Integrator\n",
        "    integrator = mm.BrownianIntegrator(\n",
        "        100 * mm.unit.kelvin,\n",
        "        100. / mm.unit.picoseconds,\n",
        "        2.0 * mm.unit.femtoseconds\n",
        "    )\n",
        "    simulation = mm.app.Simulation(topology, system, integrator)\n",
        "\n",
        "    # Initialize the DCDReporter\n",
        "    reportInterval = 100  # Adjust this value as needed\n",
        "    reporter = mdtraj.reporters.DCDReporter('positions.dcd', reportInterval)\n",
        "\n",
        "    # Add the reporter to the simulation\n",
        "    simulation.reporters.append(reporter)\n",
        "\n",
        "    simulation.context.setPositions(positions)\n",
        "\n",
        "    simulation.minimizeEnergy(1, 1000)\n",
        "    # Save positions\n",
        "    minpositions = simulation.context.getState(getPositions=True).getPositions()\n",
        "    mm_app.PDBFile.writeFile(topology, minpositions, open(outdir+f'{out_title}.pdb','w'))\n",
        "\n",
        "    reporter.close()\n",
        "\n",
        "    return topology, minpositions\n",
        "\n",
        "def add_restraints(\n",
        "        system,\n",
        "        topology,\n",
        "        positions,\n",
        "        restraint_type\n",
        "        ):\n",
        "    '''Function to add restraints to specified group of atoms\n",
        "\n",
        "    Code adapted from https://gist.github.com/peastman/ad8cda653242d731d75e18c836b2a3a5\n",
        "\n",
        "    '''\n",
        "    restraint = CustomExternalForce('k*periodicdistance(x, y, z, x0, y0, z0)^2')\n",
        "    system.addForce(restraint)\n",
        "    restraint.addGlobalParameter('k', 100.0*mm_unit.kilojoules_per_mole/mm_unit.nanometer**2)\n",
        "    restraint.addPerParticleParameter('x0')\n",
        "    restraint.addPerParticleParameter('y0')\n",
        "    restraint.addPerParticleParameter('z0')\n",
        "\n",
        "    for atom in topology.atoms():\n",
        "        if restraint_type == 'protein':\n",
        "            if 'x' not in atom.name:\n",
        "                restraint.addParticle(atom.index, positions[atom.index])\n",
        "        elif restraint_type == 'CA+ligand':\n",
        "            if ('x' in atom.name) or (atom.name == \"CA\"):\n",
        "                restraint.addParticle(atom.index, positions[atom.index])\n",
        "\n",
        "    return system\n",
        "## Read in ligand\n",
        "print('Reading ligand')\n",
        "\n",
        "ligand_mol = Molecule.from_file(\"/content/\"+ID+\"/\"+ID+\"_pred_ligand.sdf\")\n",
        "\n",
        "# Assigning partial charges first because the default method (am1bcc) does not work\n",
        "ligand_mol.assign_partial_charges(partial_charge_method='gasteiger')\n",
        "\n",
        "## Read protein PDB and add hydrogens\n",
        "protein_topology, protein_positions = fix_pdb(\"/content/\"+ID+\"/\"+ID+\"_pred_protein.pdb\", \"/content/\"+ID+\"/\", ID)\n",
        "print('Added all atoms...')\n",
        "\n",
        "# Minimize energy for the protein\n",
        "system = set_system(protein_topology)\n",
        "print('Creating system...')\n",
        "#Relax\n",
        "print('Preparing complex')\n",
        "## Add protein first\n",
        "modeller = Modeller(protein_topology, protein_positions)\n",
        "print('System has %d atoms' % modeller.topology.getNumAtoms())\n",
        "\n",
        "## Then add ligand\n",
        "print('Adding ligand...')\n",
        "lig_top = ligand_mol.to_topology()\n",
        "modeller.add(lig_top.to_openmm(), lig_top.get_positions().to_openmm())\n",
        "print('System has %d atoms' % modeller.topology.getNumAtoms())\n",
        "\n",
        "print('Preparing system')\n",
        "# Initialize a SystemGenerator using the GAFF for the ligand and implicit water.\n",
        "# forcefield_kwargs = {'constraints': mm_app.HBonds, 'rigidWater': True, 'removeCMMotion': False, 'hydrogenMass': 4*mm_unit.amu }\n",
        "system_generator = SystemGenerator(\n",
        "    forcefields=['amber14-all.xml', 'implicit/gbn2.xml'],\n",
        "    small_molecule_forcefield='gaff-2.11',\n",
        "    molecules=[ligand_mol],\n",
        "    # forcefield_kwargs=forcefield_kwargs\n",
        ")\n",
        "\n",
        "## Create system\n",
        "system = system_generator.create_system(modeller.topology, molecules=ligand_mol)\n",
        "\n",
        "print('Adding restraints on protein CAs and ligand atoms')\n",
        "\n",
        "system = add_restraints(system, modeller.topology, modeller.positions, restraint_type='CA+ligand')\n",
        "\n",
        "## Minimize energy\n",
        "minimize_energy(\n",
        "    modeller.topology,\n",
        "    system,\n",
        "    modeller.positions,\n",
        "    \"/content/\"+ID+\"/\",\n",
        "    ID+'_relaxed_complex'\n",
        ")\n",
        "\n",
        "\n"
      ],
      "metadata": {
        "cellView": "form",
        "id": "2bSIbbrCJkLm"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#@title Download relaxed results\n",
        "from google.colab import files\n",
        "import glob\n",
        "for relaxed_complex in glob.glob('/content/*/*_relaxed_complex.pdb'):\n",
        "  files.download(relaxed_complex)"
      ],
      "metadata": {
        "cellView": "form",
        "id": "_SscAKeFKzrp"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}